---
title:  Erosion of somatic tissue identity with loss of the X-linked intellectual disability factor KDM5C
output: pdf_document
geometry: margin = 1in
font-size: 11pt
header-includes:
   - \usepackage{caption}
   - \captionsetup[figure]{font=footnotesize, labelfont=bf}
   - \usepackage{sectsty} 
   - \usepackage{helvet}
   - \renewcommand{\familydefault}{\sfdefault}
   - \usepackage{lineno}
   - \linenumbers
   - \renewcommand\linenumberfont{\normalfont\small}
   - \usepackage{setspace}
   - \usepackage{siunitx}
   - \spacing{1.5}
   - \pagenumbering{arabic}
   - \usepackage{indentfirst}

bibliography: references.bib
csl: cell-reports.csl
indent: true
editor_options:
  chunk_output_type: console
---
```{r, knitrcommas, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#adds commas to large numbers
library(huxtable)

knitr::knit_hooks$set(inline = function(x) {
  prettyNum(x, big.mark=",")
})

```
Katherine M. Bonefas, Ilakkiya Venkatachalam, and Shigeki Iwase.

# Abstract 

Mutations in numerous chromatin-modifying enzymes cause neurodevelopmental disorders (NDDs) with unknown mechanisms. Loss of repressive chromatin regulators can lead to the aberrant transcription of tissue-specific genes outside of their intended context, however the mechanisms and consequences of their dysregulation are largely unknown. Here, we examine how lysine demethylase 5c (KDM5C), an eraser of histone 3 lysine 4 di and tri-methylation (H3K4me2/3) mutated in Claes-Jensen X-linked intellectual disability, contributes to tissue identity. We found male _Kdm5c_ knockout (-KO) mice, which recapitulate key human neurological phenotypes, aberrantly expresses many liver, muscle, ovary, and testis genes within the amygdala and hippocampus. Gonad-enriched genes misexpressed in the _Kdm5c_-KO brain are unique to germ cells, indicating an erosion of the soma-germline boundary. Germline genes are typically decommissioned in somatic lineages in the post-implantation epiblast, yet _Kdm5c_-KO epiblast-like cells (EpiLCs) aberrantly expressed key regulators of germline identity and meiosis, including _Dazl_ and _Stra8_. Germline gene suppression is sexually dimorphic, as female EpiLCs required a higher dose of KDM5C to maintain germline gene suppression. Using a comprehensive list of mouse germline-enriched genes, we found KDM5C is selectively recruited to a subset of germline gene promoters that contain CpG islands (CGIs) to facilitate DNA CpG methylation (CpGme) during ESC to EpiLC differentiation. However, late stage spermatogenesis genes devoid of promoter CGIs can also become activated in _Kdm5c_-KO cells via ectopic activation by RFX transcription factors. Thus, distinct mechanisms govern the misexpression of germline gene classes, including activation by ectopic germline programs that mirror germ cell development within somatic tissues.


# Introduction

A single genome holds the instructions to generate the myriad of cell types found within an organism. This is, in part, accomplished by chromatin regulators that can either promote or impede lineage-specific gene expression through DNA and histone modifications[@strahlLanguageCovalentHistone2000; @jenuweinTranslatingHistoneCode2001; @lewisGeneComplexControlling1978a; @kennisonDosagedependentModifiersPolycomb1988a; @kassisPolycombTrithoraxGroup2017]. Human genetic studies revealed mutations in chromatin regulators are a major cause of neurodevelopmental disorders (NDDs)[@gabrieleChromatinBasisNeurodevelopmental2018] and many studies have identified their importance for regulating brain-specific transcriptional programs. Loss of some chromatin regulators can also result in the ectopic expression of tissue-specific genes outside of their target environment, such as the misexpression of liver-specific genes within adult neurons[@schaeferControlCognitionAdaptive2009]. However, the mechanisms underlying ectopic gene expression and its impact upon neurodevelopment are poorly understood. 

To elucidate the role of tissue identity in chromatin-linked NDDs, it is essential to first characterize the nature of the dysregulated genes and the molecular mechanisms governing their de-repression. Here we focus on lysine demethylase 5C (KDM5C, also known as SMCX or JARID1C), which erases histone 3 lysine 4 di- and trimethylation (H3K4me2/3), a permissive chromatin modification enriched at gene promoters[@iwaseXLinkedMentalRetardation2007]. Pathogenic mutations in _KDM5C_ cause Intellectual Developmental Disorder, X-linked, Syndromic, Claes-Jensen Type (MRXSCJ, OMIM: 300534). MRXSCJ is more common and severe in males and its neurological phenotypes include intellectual disability, seizures, aberrant aggression, and autistic behaviors[@claesNovelSyndromicForm2000; @jensenMutationsJARID1CGene2005; @carmignacFurtherDelineationFemale2020]. Male _Kdm5c_ knockout (-KO) mice recapitulate key MRXSCJ phenotypes, including hyperaggression, increased seizure propensity, and learning impairments[@iwaseMouseModelXlinked2016; @scandagliaLossKdm5cCauses2017]. RNA sequencing (RNA-seq) of the _Kdm5c_-KO hippocampus revealed ectopic expression of some germline genes within the brain[@scandagliaLossKdm5cCauses2017]. However, it is unclear if other tissue-specific genes are aberrantly transcribed with KDM5C loss, at what point in development germline gene misexpression begins, and what mechanisms underlie their dysregulation. 

Distinguishing between germ cells and somatic cells is a key feature of multicellularity[@devlinPanmetazoanViewGermlinesoma2023] that occurs during early embryogenesis in many metazoans[@lehmannGermlineStemCells2012]. In mammals, chromatin regulators are crucial for decommissioning germline genes during the transition from na&#239;ve to primed pluripotency. Initially, germline gene promoters gain repressive histone H2A lysine 119 monoubiquitination (H2AK119ub1)[@endohPCGF6PRC1SuppressesPremature2017] and histone 3 lysine 9 trimethylation (H3K9me3)[@endohPCGF6PRC1SuppressesPremature2017; @mochizukiRepressionGermlineGenes2021] in embryonic stem cells (ESCs) and are then decorated with DNA CpG methylation (CpGme) in the post-implantation embryo[@borgelTargetsDynamicsPromoter2010; @velascoDnmt3bRecruitmentE2F62010; @hackettPromoterDNAMethylation2012; @mochizukiRepressionGermlineGenes2021]. The contribution of KDM5C to this process remains unclear. Furthermore, studies on germline gene repression have primarily been conducted in males and focused on marker genes important for germ cell development rather than germline genes as a whole, given the lack of a curated germline-enriched gene list. Therefore, it is unknown if the mechanism of repression differs between sexes or for certain classes of germline genes, e.g. meiotic genes versus spermatid differentiation genes. 

To illuminate KDM5C's role in tissue identity, here we characterized the aberrant expression of tissue-enriched genes within the _Kdm5c_-KO brain and epiblast-like stem cells (EpiLCs), an _in vitro_ model of the post-implantation embryo. We curated a list of mouse germline-enriched genes, which enabled genome-wide analysis of germline gene silencing mechanisms for the first time. Based on the data presented below, we propose KDM5C plays a fundamental, sexually dimorphic role in the development of tissue identity during early embryogenesis, including the establishment of the soma-germline boundary. 


# Results

## Tissue-enriched genes are aberrantly expressed in the _Kdm5c_-KO brain

```{r, DESeq2_params, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the parameters used repeatedly, like the log2fold change cut off
source("../code/utilities/parameters.R")
```

Previous RNA sequencing (RNA-seq) of the adult male _Kdm5c_-KO hippocampus revealed ectopic expression of some germline genes unique to the testis[@scandagliaLossKdm5cCauses2017]. It is currently unknown if the testis is the only tissue type misexpressed in the _Kdm5c_-KO brain. We thus systematically tested whether other tissue-specific genes are misexpressed in the male brain with constitutive knockout of _Kdm5c_ (_Kdm5c^-/y^_, 5CKO)[@vallianatosMutuallySuppressiveRoles2020] by using a published list of mouse tissue-enriched genes[@liComprehensiveMouseTranscriptomic2017]. 

```{r, tissue_genes, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the number and stats for tissue-enriched gene expression
tissue_res <- read.csv("../data/processed/TissueSpecific_AMYHIP_numberofgenesDEGs.csv", header = TRUE)

#get the value you want to print 
#col is which data column, tiss is which tissue
gettissval <- function(col, tiss){
	val <- subset(tissue_res, tissue == tiss)
	val <- val[,col]
	return(val)
}

```

We found a large proportion of significantly upregulated genes (DESeq2[@loveModeratedEstimationFold2014a], log2 fold change > `r l2fcco`, q < 0.1) within the male _Kdm5c_-KO amygdala and hippocampus are typically enriched within non-brain tissues (Amygdala: 35%, Hippocampus: 24%) (Figure 1A-B). For both the amygdala and hippocampus, the majority of tissue-enriched differentially expressed genes (DEGs) were testis genes (Figure 1A-B). Even though the testis has the largest total number of tissue-biased genes (`r gettissval("count", "Testis")` genes) compared to any other tissue, testis-biased DEGs were significantly enriched for both brain regions (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Testis"), 3)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Testis"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Testis"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Testis"), 3)`, Fisher's Exact Test). An example of a testis-enriched gene misexpressed in the _Kdm5c_-KO brain is _FK506 binding protein 6 (Fkbp6)_, a known regulator of PIWI-interacting RNAs (piRNAs) and meiosis[@crackowerEssentialRoleFkbp62003; @xiolRoleFkbp6Chaperone2012] (Figure 1C). 

Interestingly, we also observed significant enrichment of ovary-biased DEGs in both the amygdala and hippocampus (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Ovary"), 3)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Ovary"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Ovary"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Ovary"), 3)`, Fisher's Exact) (Figure 1A-B). Ovary-enriched DEGs included _Zygotic arrest 1 (Zar1)_, which sequesters mRNAs in oocytes for meiotic maturation[@chengMammalianOocytesStore2022] (Figure 1D). Given that the _Kdm5c_-KO mice we analyzed are male, these data demonstrate that the ectopic expression of gonad-enriched genes is independent of organismal sex.

Although not consistent across brain regions, we also found significant enrichment of DEGs biased towards two non-gonadal tissues - the liver (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Liver"), 2)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Liver"), 3)`, Fisher's Exact Test) and muscles (Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Muscle"), 2)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Muscle"), 3)`, Fisher's Exact Test) (Figure 1A-B). _Apolipoprotein C-I (Apoc1)_ a lipoprotein metabolism and transport gene, is among the liver-biased DEG derepressed in both the hippocampus and amygdala[@roulandRoleApolipoproteinC12022] and its brain overexpression has been implicated in Alzheimer's disease[@leducAPOECholesterolHomeostasis2010] (Figure 1E). 

For all _Kdm5c_-KO tissue-enriched DEGs, aberrantly expressed mRNAs are polyadenylated and spliced into mature transcripts (Figure 1C-E). Of note, we observed little to no dysregulation of brain-enriched genes (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Brain"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Brain"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Brain"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Brain"), 3)`, Fisher's Exact), despite the fact these are brain samples and the brain has the second highest total number of tissue-enriched genes (`r gettissval("count", "Brain")` genes). Altogether, these results suggest the aberrant expression of tissue-enriched genes within the brain is a major effect of KDM5C loss.

```{r, germgenes, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
germgenes <- read.csv("../data/processed/germGENES20.csv", header = TRUE)

#testis gene ontology results
testisGO <- read.csv("../results/GO_Brain_TestisDEGs.csv")

#function to get the GO ID
getGO <- function(x, df, column, cluster){
	df_test <- df
	
	if (cluster == "no") {
		out <- df_test[df_test$Description == x,][column][,1]
	} else {
		out <- df_test[df_test$Description == x & df_test$Cluster == cluster,][column][,1]
	}
	outformat <- ifelse(column == "ID", toString(out), out) #an attempt to remove the commas from GO ID numbers, didn't work
	return(outformat) 
}

```
### Germline genes are misexpressed in the _Kdm5c_-KO brain

_Kdm5c_-KO brain expresses testicular germline genes[@scandagliaLossKdm5cCauses2017], however the testis also contains somatic cells that support hormone production and germline functions. To determine if _Kdm5c_-KO results in ectopic expression of somatic testicular genes, we first evaluated the known functions of testicular DEGs through gene onotology. We found _Kdm5c_-KO testis-enriched DEGs had high enrichment of germline-relevant ontologies, including spermatid development (GO: 0007286, p.adjust = `r signif(getGO("spermatid development", testisGO, 'p.adjust', "no"), 3)`) and sperm axoneme assembly (GO: 0007288, p.adjust = `r signif(getGO("sperm axoneme assembly", testisGO, 'p.adjust', "no"), 3)`) (Figure 2A). 

We then evlauted testicular DEG expression in wild-type testes versus testes with germ cell depletion[@muellerIndependentSpecializationHuman2013], which was accomplished by heterozygous _W_ and _Wv_ mutations in the enzymatic domain of _c-Kit_ (Kit^W/Wv^)[@handelSertoliCellDifferentiation1979]. Almost all _Kdm5c_-KO testis-enriched DEGs lost expression with germ cell depletion (Figure 2B). We then assessed testis-enriched DEG expression in a published single cell RNA-seq dataset that identified cell type-specific markers within the testis[@greenComprehensiveRoadmapMurine2018]. Some _Kdm5c_-KO testis-enriched DEGs were classified as specific markers for different germ cell developmental stages (e.g. spermatogonia, spermatocytes, round spermatids, and elongating spermatids), yet none marked somatic cells (Figure 2C). Together, these data demonstrate that the _Kdm5c_-KO brain aberrantly expresses germline genes but not somatic testicular genes, reflecting an erosion of the soma-germline boundary.

As of yet, research on germline gene silencing mechanisms has focused on a handful of key genes rather than assessing germline gene suppression genome-wide, due to the lack of a comprehensive gene list. We therefore generated a list of mouse germline-enriched genes using RNA-seq datasets of Kit^W/Wv^ mice that included males and females at embryonic day 12, 14, and 16[@sohGeneRegulatoryProgram2015] and adult male testes[@muellerIndependentSpecializationHuman2013]. We defined genes as germline-enriched if their expression met the following criteria: 1) their expression is greater than 1 FPKM in wild-type gonads 2) their expression in any non-gonadal tissue of adult wild type mice[@liComprehensiveMouseTranscriptomic2017] does not exceed 20% of their maximum expression in the wild-type germline, and 3) their expression in the germ cell-depleted gonads, for any sex or time point, does not exceed 20% of their maximum expression in the wild-type germline. These criteria yielded `r nrow(germgenes)` germline-enriched genes (Figure 2D), which was hereafter used as a resource to globally characterize germline gene misexpression with _Kdm5c_ loss (Supplementary table 1).

## _Kdm5c_-KO epiblast-like cells aberrantly express key regulators of germline identity 

```{r, epilcs, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}

#testis gene ontology results
epigerm <- read.csv("../results/DESeq2/germDEGs/germDEGs_EpiLC_XY5cKO.csv")
GO_EpiBrain <- read.csv("../results/GO_EpiLC_vs_Brain.csv")

epi_restab <- read.csv("../data/processed/restable_EpiLC_XY5cKO.csv", row.names = 1)
epi_restab$ENSEMBL <- rownames(epi_restab)

zscan4d_padj <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000090714',]['padj'][,1]
duxf3_padj <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000075046',]['padj'][,1]

zscan4d_l2fc <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000090714',]['log2FoldChange'][,1]
duxf3_l2fc <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000075046',]['log2FoldChange'][,1]


getresval <- function(df, col, symbol){
	val <- subset(df, SYMBOL == symbol)
	val <- val[,col]
	return(val)
}

```

Germ cells are typically distinguished from somatic cells soon after the embryo implants into the uterine wall[@magnusdottirHowMakePrimordial2014; @gunesdoganPrimordialGermCell2014], when germline genes are silenced in epiblast stem cells that will form the somatic tissues[@bardotMouseGastrulationCoordination2020]. This developmental time point can be modeled _in vitro_ through differentiation of na&#239;ve embryonic stem cells (nESCs) into epiblast-like stem cells (EpiLCs) (Figure 3A)[@hayashiReconstitutionMouseGerm2011; @samantaGeneratingPrimedPluripotent2020]. While some germline-enriched genes are also expressed in nESCs and in the 2-cell stage[@wellingDAZLRegulatesTet12015; @macfarlanEmbryonicStemCell2012; @suzukiLossMAXResults2016], they are silenced as they differentiate into EpiLCs[@borgelTargetsDynamicsPromoter2010; @mochizukiRepressionGermlineGenes2021]. Therefore, we tested if KDM5C was necessary for the initial silencing of germline genes in somatic lineages by evaluating the impact of _Kdm5c_ loss in male EpiLCs. 

_Kdm5c_-KO cell morpholgy during ESC to EpiLC differentiation appeared normal (Figure 3B) and EpiLCs properly expressed markers of primed pluripotency, such as _Dnmt3b_, _Fgf5_, _Pou3f1_, and _Otx2_ (Figure 3C). We then identified tissue-enriched DEGs in a RNA-seq dataset of wild-type and _Kdm5c_-KO EpiLCs[@samantaActivationXistEvolutionarily2022] (DESeq2, log2 fold change > `r l2fcco`, q < 0.1). Similar to the _Kdm5c_-KO brain, we observed general dysregulation of tissue-enriched genes, with the largest number of genes belonging to the brain and testis, although they were not significantly enriched (Figure 3D). Using our list of mouse germline-enriched genes assembled above, we found `r nrow(epigerm)` germline genes were misexpressed in male _Kdm5c_-KO EpiLCs. 

We then compared EpiLC germline DEGs to those expressed in the _Kdm5c_-KO brain to determine if germline genes are constitutively dysregulated or change over the course of development. The majority of germline DEGs were unique to either EpiLCs or the brain, with only _D1Pas1_ and _Cyct_ shared across all tissue/cell types (Figure 3E-F). EpiLC germline DEGs had particularly high enrichment of meiosis-related gene ontologies when compared to the brain (Figure 3G), such as meiotic cell cycle process (GO:1903046, p.adjust = `r signif(getGO("meiotic cell cycle process", GO_EpiBrain, 'p.adjust', "EpiLC"), 3)`) and meiotic nuclear division (GO:0140013, p.adjust = `r signif(getGO("meiotic nuclear division", GO_EpiBrain, 'p.adjust', "EpiLC"), 3)`). While there was modest enrichment of meiotic gene ontologies in both brain regions, the _Kdm5c_-KO hippocampus primarily expressed late-stage spermatogenesis genes involved in sperm axoneme assembly (GO:0007288, p.adjust = `r signif(getGO("sperm axoneme assembly", GO_EpiBrain, 'p.adjust', "Hippocampus"), 3)`) and sperm motility (GO:0097722, p.adjust = `r signif(getGO("sperm motility", GO_EpiBrain, 'p.adjust', "Hippocampus"), 3)`).

Notably, DEGs unique to _Kdm5c_-KO EpiLCs included key drivers of germline identity, such as _Stimulated by retinoic acid 8_ (_Stra8_: log2 fold change = `r signif(getresval(epigerm, 'log2FoldChange', 'Stra8'), 3)`, q = `r signif(getresval(epigerm, 'padj', 'Stra8'), 3)`) and _Deleted in azoospermia like_ (_Dazl_: log2 fold change = `r signif(getresval(epigerm, 'log2FoldChange', 'Dazl'), 3)`, q = `r signif(getresval(epigerm, 'padj', 'Dazl'), 3)`) (Figure 3H). These genes are typically expressed when primordial germ cells (PGCs) are committed to the germline fate and later in life to trigger meiotic gene expression programs[@koubovaRetinoicAcidRegulates2006; @linGermCellIntrinsicExtrinsic2008; @endoRetinoicAcidGerm2019]. Of note, some germline genes, including _Dazl_, are also expressed in the two-cell embryo[@macfarlanEmbryonicStemCell2012; @guptaGenomewideScreenReveals2023]. However, we did not see derepression of two-cell stage-specific genes, like _Duxf3 (Dux)_ (log2 fold change = `r signif(duxf3_l2fc, 3)`, q = `r signif(duxf3_padj, 3)`) and _Zscan4d_ (log2 fold change =  `r signif(zscan4d_l2fc, 3)`, q = `r signif(zscan4d_padj, 3)`) (Figure 3H), indicating _Kdm5c_-KO EpiLCs do not revert back to a 2-cell state. Altogether, _Kdm5c_-KO EpiLCs express key drivers of germline identity and meiosis while the brain primarily expresses spermiogenesis genes, indicating germline gene misexpression mirrors germline development during the progression of somatic development.
 

## Female epiblast-like cells have increased sensitivity to germline gene misexpression with _Kdm5c_ loss
```{r, DESeq2_params_XX, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the parameters used repeatedly, like the log2fold change cut off
source("../../KDM5C_Germ_Dev_Sex_RA/code/utilities/parameters.R")
```

It is currently unknown if the misexpression of germline genes is influenced by sex, as previous studies on germline gene repressors have focused on male cells[@pohlersRoleE2F6Restriction2005; @velascoDnmt3bRecruitmentE2F62010; @endohPCGF6PRC1SuppressesPremature2017; @dahletE2F6InitiatesStable2021; @mochizukiRepressionGermlineGenes2021]. Sex is particularly pertinent in the case of KDM5C because it partially escapes X chromsome inactivation (XCI), resulting in a higher dosage in females[@agulnikNovelGeneWidely1994; @carrelTissueLineagespecificVariation1996; @sheardownMouseSmcxGene1996; @xuSexSpecificExpressionXLinked2008]. We therefore explored the impact of chromosomal sex upon germline gene suppression by comparing their dysregulation in male _Kdm5c_ hemizygous knockout (XY _Kdm5c_-KO, XY 5CKO), female homozygous knockout (XX _Kdm5c_-KO, XX 5CKO), and female heterozygous knockout (XX _Kdm5c_-HET, XX 5CHET) EpiLCs [@samantaActivationXistEvolutionarily2022].

In EpiLCs, homozygous and heterozygous _Kdm5c_ knockout females expressed over double the number of germline-enriched genes than hemizygous males (Figure 4A). While the majority of germline DEGs in _Kdm5c_-KO males were also dysregulated in females (74%), many were sex-specific, such as _Tktl2_ and _Esx1_ (Figure 4B). We then compared the known functions of germline genes dysregulated only in females (XX only - unique to XX _Kdm5c_-KO, XX _Kdm5c_-HET, or both), only in males (XY only), or in all samples (shared) (Figure 4C). Female-specific germline DEGs were enriched for meiotic (GO:0051321 - meiotic cell cycle) and flagellar (GO:0003341 - cilium movement) functions, while male-specifc DEGs had roles in mitochondrial and cell signaling (GO:0070585 - protein localization to mitochondrion). Germline transcripts expressed in both sexes were enriched for meiotic (GO:0140013 - meiotic nuclear division) and egg-specific functions (GO:0007292 - female gamete generation). 

The majority of germline genes expressed in both sexes were more highly dysregulated in females compared to males (Figure 4D-F). This increased degree of dysregulation in females, along with the increased total number of germline genes, indicates females are more sensitive to losing KDM5C-mediated germline gene suppression. Female sensitivity could be due to impaired XCI in _Kdm5c_ mutants[@samantaActivationXistEvolutionarily2022], as many spermatogenesis genes lie on the X chromosome[@wangAbundanceXlinkedGenes2001; @khilMouseChromosomeEnriched2004]. However, female germline DEGs were not biased towards the X chromosome and had a a similar overall proportion of X chromosome DEGs compared to males (XY _Kdm5c_-KO - 10.29%, XX _Kdm5c_-HET - 7.43%, XX _Kdm5c_-KO - 10.59%) (Figure 4G). The majority of germline DEGs instead lie on autosomes for both male and female _Kdm5c_ mutants (Figure 4G). Thus, while female EpiLCs are more prone to germline gene misexpression with KDM5C loss, it is likely independent of XCI defects.


### Germline gene misexpression in _Kdm5c_ mutants is independent of germ cell sex

Although many germline genes have shared functions in the male and female germline, e.g. PGC formation,  meiosis, and genome defense, some have unique or sex-biased expression. Therefore, we wondered if _Kdm5c_ mutant males would primarily express sperm genes while mutant females would primarily express egg genes. To comprehensively assess whether germline gene sex corresponds with _Kdm5c_ mutant sex, we first filtered our list of germline-enriched genes for egg and sperm-biased genes (Figure 4H). We defined germ cell sex-biased genes as those whose expression in the opposite sex, at any time point, is no greater than 20% of the gene's maximum expression in a given sex. This criteria yielded `r nrow(subset(germgenes, sexBias == "XX"))` egg-biased, `r nrow(subset(germgenes, sexBias == "XY"))` sperm-biased, and `r nrow(subset(germgenes, sexBias == "unbiased"))` unbiased germline-enriched genes. We found regardless of sex, egg, sperm, and unbiased germline genes were dyregulated in all _Kdm5c_ mutants at similar proportions (Figure 4I-J). Furthermore, germline genes dysregulated exclusively in either male or female mutants were also not biased towards their corresponding germ cell sex (Figure 4I). Altogether, these results demonstrate sex differences in germline gene dysregulation is not due to sex-specific activation of sperm or egg transcriptional programs.
 

## KDM5C binds to a subset of germline gene promoters during early embryogenesis 

KDM5C binds to the promoters of several germline genes in embryonic stem cells (ESCs) but its binding is absent in neurons[@scandagliaLossKdm5cCauses2017; @aladhamiSystematicIdentificationFactors2023]. However, the lack of a comprehensive list of germline-enriched genes prohibited genome-wide characterization of KDM5C binding at germline gene promoters. Thus, it is unclear if KDM5C is enriched at germline gene promoters, what types of germline genes KDM5C regulates, and if its binding is maintained at any germline genes in neurons. 

```{r, KDM5C_ChIPseq, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
KDM5C_ChIP_EpiLC_promo <- read.csv("../data/processed/KDM5C_ChIPseq_boundpromoters_EpiLC.csv")

KDM5C_ChIP_PNC_promo <- read.csv("../data/processed/KDM5C_ChIPseq_boundpromoters_PNC.csv")

KDM5C_ChIP_EpiLC <- read.csv("../data/raw/ChIPseq_mm10_EpiLC_WTnoKO_consensus_peaks.bed", sep = "\t")

KDM5C_ChIP_PNC <- read.csv("../data/raw/ChIPseq_mm10_PNC_WTnoKO_consensus_peaks.bed", sep = "\t")

GO_KDM5C_ChIP <- read.csv("../results/ChIPseq_KDM5C_GO_EpiLC_vs_PNC_unique.csv")

```

To address these questions, we analyzed KDM5C chromatin immunoprecipitation followed by DNA sequencing (ChIP-seq) datasets in EpiLCs[@samantaActivationXistEvolutionarily2022] and primary forebrain neuron cultures (PNCs)[@iwaseMouseModelXlinked2016]. EpiLCs had a higher total number of high-confidence KDM5C peaks than PNCs (EpiLCs: `r nrow(KDM5C_ChIP_EpiLC)`, PNCs: `r nrow(KDM5C_ChIP_PNC)`, MACS2 q < 0.1 and fold enrichment > 1). KDM5C was primarily localized to gene promoters in both cell types (Promoters = transcription start site (TSS) &#177; `r promorange`bp, EpiLCs: `r nrow(KDM5C_ChIP_EpiLC_promo)`, PNCs: `r nrow(KDM5C_ChIP_PNC_promo)`), although PNCs showed increased localization to non-promoter regions (Figure 5A). 

The majority of promoters bound by KDM5C in PNCs were also bound in EpiLCs (513 shared promoters), however a large portion of gene promoters were bound by KDM5C only in EpiLCs (3,677 EpiLC only promoters) (Figure 5B). Genes bound by KDM5C in both PNCs and EpiLCs were enriched for functions involving nucleic acid turnover, such as deoxyribonucleotide metabolic process (GO:0009262, p.adjust = `r signif(getGO("deoxyribonucleotide metabolic process", GO_KDM5C_ChIP, 'p.adjust', "Shared"), 3)`) (Figure 5C). Germline-specific ontologies were enriched only in EpiLC-specific KDM5C-bound promoters, such as meiotic nuclear division (GO: 0007127 p.adjust = `r signif(getGO("meiosis I", GO_KDM5C_ChIP, 'p.adjust', "EpiLC only"), 3)`) (Figure 5C). There were no ontologies significantly enriched for PNC-specific KDM5C target genes. Using our mouse germline gene list, we observed evident KDM5C signal around the TSS of many germline genes in EpiLCs, but not in PNCs (Figure 5D). Based on our ChIP-seq peak cut-off criteria, KDM5C was highly enriched at 211 germline gene promoters in EpiLCs (`r signif(211/1288*100, 3)`% of all germline genes) (Figure 5E). Of note, KDM5C was only bound to about one third of _Kdm5c_-KO RNA-seq DEG promoters (EpiLC only DEGs: `r signif(22/63*100, 3)`%, Brain only DEGs: `r signif(6/20*100,3)`%) (Supplementary figure 1A-C). However, KDM5C did bind the promoter of 4 out of the 5 genes dysregulated in both the brain and EpiLCs (_D1Pas1_, _Hsf2bp_, _Cyct_, and _Stk31_). Representative examples of KDM5C-bound and unbound EpiLC DEGs are _Dazl_ and _Stra8_, repsectively (Figure 5F). Together, these results demonstrate KDM5C is recruited to a subset of germline genes in EpiLCs, including meiotic genes, but does not directly regulate germline genes in neurons. Furthermore, the majority of germline mRNAs expressed in _Kdm5c_-KO cells are dysregulated independent of direct KDM5C binding to their gene promoters. 

Many germline-specific genes are suppressed by the polycomb repressive complex 1.6 (PRC1.6), which contains transcription factor heterodimers E2F6/DP1 and MGA/MAX that respectively bind E2F and E-box motifs[@hurlinMgaDualspecificityTranscription1999; @pohlersRoleE2F6Restriction2005; @velascoDnmt3bRecruitmentE2F62010; @suzukiLossMAXResults2016; @endohPCGF6PRC1SuppressesPremature2017 @stielowMGAL3MBTL2E2F62018; @tatsumiDNMTsSETDB1Function2018; @dahletE2F6InitiatesStable2021; @mochizukiRepressionGermlineGenes2021]. PRC1.6 members may recruit KDM5C to germline gene promoters[@scandagliaLossKdm5cCauses2017], given their association with KDM5C in HeLa cells and ESCs[@tahilianiHistoneH3K4Demethylase2007; @guptaGenomewideScreenReveals2023]. We thus used HOMER[@heinzSimpleCombinationsLineageDetermining2010] to identify transcription factor motifs enriched at KDM5C-bound or unbound germline gene promoters (TSS &#177; 500 bp, q-value < 0.1). MAX and E2F6 binding sites were significantly enriched at germline genes bound by KDM5C in EpiLCs (MAX q-value: 0.0068, E2F6 q-value: 0.0673, E2F q-value: 0.0917), but not at germline genes unbound by KDM5C (Figure 5G). One third of KDM5C-bound promoters contained the consensus sequence for either E2F6 (E2F, 5'-TCCCGC-3'), MGA (E-box, 5'-CACGTG-3'), or both, but only 17% of KDM5C-unbound genes contained these motifs (Figure 5H). KDM5C-unbound germline genes were intstead enriched for multiple RFX transcription factor binding sites (RFX q-value < 0.0001, RFX2 q-value < 0.0001, RFX5 q-value < 0.0001) (Figure 5I, Supplementary figure 1D). RFX transcription factors bind X-box motifs[@gajiwalaStructureWingedhelixProtein2000] to promote ciliogenesis[@swobodaRFXTypeTranscriptionFactor2000; @ashiqueRfx4TranscriptionFactor2009] and among them is RFX2, a central regulator of post-meiotic spermatogeneis[@kistlerRFX2MajorTranscriptional2015; @wuTranscriptionFactorRFX22016]. Although _Rfx2_ is also not a direct target of KDM5C (Supplementary figure 1E), RFX2 mRNA is derepressed in _Kdm5c_-KO EpiLCs (Figure 5J). Thus, RFX2 is a candidate transcription factor for driving the ectopic expression of many KDM5C-unbound germline genes in _Kdm5c_-KO cells.


## KDM5C is recruited to CpG islands at germline promoters to facilitate _de novo_ DNA methylation

Previous work found two germline gene promoters have a marked reduction in DNA CpG methylation (CpGme) in the _Kdm5c_-KO adult hippocampus[@scandagliaLossKdm5cCauses2017]. Since histone 3 lysine 4 di- and trimethylation (H3K4me2/3) impede _de novo_ CpGme[@otaniStructuralBasisRecognition2009; @guoStructuralInsightAutoinhibition2015], KDM5C's removal of H3K4me2/3 may be required to suppress germline genes. However, KDM5C's catalytic activity was recently shown to be dispensible for suppressing _Dazl_ in ESCs[@guptaGenomewideScreenReveals2023]. To reconcile these observations, we hypothesized KDM5C erases H3K4me2/3 to promote the initial placement of CpGme at germline gene promoters in EpiLCs. 

To test this hypothesis, we first characterized KDM5C's expression as na&#239;ve ESCs differentiate into EpiLCs (Figure 6A). While _Kdm5c_ mRNA steadily decreased from 0 to 48 hours of diferentiation (Figure 6B), KDM5C protein initially increased from 0 to 24 hours but then decreased to near knockout levels by 48 hours (Figure 6C). We then characterized KDM5C's substrates (H3K4me2/3) at germline gene promoters with _Kdm5c_ loss using published ChIP-seq datasets[@samantaActivationXistEvolutionarily2022; @vallianatosMutuallySuppressiveRoles2020]. _Kdm5c_-KO samples showed a marked increase in H3K4me2 in EpiLCs (Figure 6D) and H3K4me3 in the amygdala (Figure 6E) around the TSS of germline genes. Together, these data suggest KDM5C acts during the transition between ESCs and EpiLCs to remove H3K4me2/3 at germline gene promoters.



```{r, CGIs, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
germgenes_CGI <- read.csv("../results/KDM5C_binding_allgerm_CGI.csv", header = TRUE)

CGI <- germgenes_CGI[germgenes_CGI$Promo_CGI == "CGI",]
KDM5C_bound_CGI  <- germgenes_CGI[germgenes_CGI$KDM5C_binding == "Bound" & germgenes_CGI$Promo_CGI == "CGI",]

KDM5C_bound  <- germgenes_CGI[germgenes_CGI$KDM5C_binding == "Bound",]

ESCvsEpiLC <- read.csv("../results/WGBS/WGBS_restab_annotated_allgermTSS_500bp_WT_ESCvsEpiLC.csv")

ESCvsEpiLC_25 <- ESCvsEpiLC[ESCvsEpiLC$meth.diff >= 25, ]

WTvsKO <- read.csv("../results/WGBS/WGBS_restab_annotated_allgermTSS_500bp_WTvsKO.csv")

WTvsKO_hypo <- WTvsKO[WTvsKO$meth.diff <= -25, ]
WTvsKO_hypo_5C <- WTvsKO_hypo[WTvsKO_hypo$KDM5C_binding == "Bound", ]


#get the infomration for wgbs
getWGBSinfo <- function(df, gene, column){
	df2 <- df
	info <- df2[df2$SYMBOL == gene,][[column]]
	return(info)
}

#get all promoters 5CKO vs WT WGBS information
exEpi_WTvsKO_all <- read.csv("../results/WGBS/WGBS_restab_annotated_allgenes_WTvsKO.csv")

#Gene ontology CGI
GO_CGI <- read.csv("../results/germline_CGI_GO.csv")

#Fisher's exact test KDM5C binding at CGI promoters
noCGI <- c(bound = nrow(germgenes_CGI[germgenes_CGI$KDM5C_binding == "Bound" & germgenes_CGI$Promo_CGI == "no",]), unbound = nrow(germgenes_CGI[germgenes_CGI$KDM5C_binding == "Unbound" & germgenes_CGI$Promo_CGI == "no",]))
yesCGI <- c(bound = nrow(KDM5C_bound_CGI), unbound = nrow(germgenes_CGI[germgenes_CGI$KDM5C_binding == "Unbound" & germgenes_CGI$Promo_CGI == "CGI",]))
CGI_5C <- data.frame(row.names = c("bound", "unbound"), yesCGI, noCGI)

CGI_5C_fish <- fisher.test(CGI_5C)

```

Germline genes accumulate CpG methylation (CpGme) at CpG islands (CGIs) during the transition from na&#239;ve to primed pluripotency[@meissnerGenomescaleDNAMethylation2008; @borgelTargetsDynamicsPromoter2010; @hackettPromoterDNAMethylation2012]. We first examined how many of our germline-enriched genes had promoter CGIs (TSS &#177; 500 bp) using the UCSC genome browser[@nassarUCSCGenomeBrowser2023]. Notably, out of `r nrow(germgenes)` germline-enriched genes, only `r nrow(CGI)` (`r signif(nrow(CGI)/nrow(germgenes)*100, 4)`%) had promoter CGIs (Figure 6F). CGI-containing germline genes had higher enrichment of meiotic gene ontologies compared to CGI-free genes, including meiotic nuclear division (GO:0140013, p.adjust = `r signif(getGO("meiotic nuclear division", GO_CGI, 'p.adjust', "CGI"), 3)`) and meiosis I (GO:0007127, p.adjust = `r signif(getGO("meiosis I", GO_CGI, 'p.adjust', "CGI"), 3)`) (Figure 6G). Germline genes with promoter CGIs were more highly expressed than CGI-free genes across spermatogenesis stages, with highest expression in meiotic spermatocytes (Figure 6H). Contrastingly, CGI-free genes only displayed substantial expression in post-meiotic round spermatids (Figure 6H). Although only a minor portion of germline gene promoters contained CGIs, CGIs strongly determined KDM5C's recruitment to germline genes (p = `r signif(CGI_5C_fish$p.value, 3)`, Odds Ratio = `r signif(CGI_5C_fish$estimate, 3)`, Fisher's exact test), with `r signif(nrow(KDM5C_bound_CGI)/nrow(KDM5C_bound)*100, 4)`% of KDM5C-bound germline gene promoters harboring CGIs (Figure 6G).

To assess how KDM5C loss impacts initial CpGme placement at germline gene promoters, we perfomed whole genome bislufite sequencing (WGBS) in male wild-type and _Kdm5c_-KO ESCs and 96-hour extend EpiLCs (exEpiLCs), when germline genes reach peak methylation levels[@mochizukiRepressionGermlineGenes2021] (Figure 6I). We first identified which germline gene promoters siginificantly gained CpGme in wild-type cells during nESC to exEpiLCs differentiation (methylKit[@akalinMethylKitComprehensivePackage2012], q < 0.01, |methylation difference| > 25%, TSS &#177; 500 bp). In wild-type cells, the majority of germline genes gained substantial CpGme at their promoter during differentiation (`r signif(nrow(ESCvsEpiLC_25)/nrow(ESCvsEpiLC)*100, 4)`%), regardless if their promoter contained a CGI (Figure 6J).

We then identified promoters differentially methylated in wild-type versus _Kdm5c_-KO exEpiLCs (methylKit, q < 0.01, |methylation difference| > 25%, TSS &#177; 500 bp). Of the `r nrow(exEpi_WTvsKO_all)` promoters assessed, `r nrow(exEpi_WTvsKO_all[exEpi_WTvsKO_all$meth.diff <= -25 & exEpi_WTvsKO_all$qvalue <= 0.01, ])` promoters were significantly hypomethylated and `r nrow(exEpi_WTvsKO_all[exEpi_WTvsKO_all$meth.diff >= 25 & exEpi_WTvsKO_all$qvalue <= 0.01, ])` promoters were significantly hypermethylated with KDM5C loss (Supplementary figure 2A). Many promoters hyper- and hypomethylated in _Kdm5c_-KO exEpiLCs belonged to genes with unknown functions. Hypomethylated promoters were significantly enriched for germline gene ontologies, such as meiotic nuclear division (GO:0140013, p.adjust = 0.012)(Supplementary figure 2B), with `r signif((nrow(WTvsKO_hypo)/nrow(exEpi_WTvsKO_all[exEpi_WTvsKO_all$meth.diff <= -25 & exEpi_WTvsKO_all$qvalue <= 0.01, ]))*100, 4)`% of hypomethylated promoters belonging to germline genes. Approximately half of germline promoters hypomethylated in _Kdm5c_-KO exEpiLCs are direct targets of KDM5C in EpiLCs (`r nrow(WTvsKO_hypo_5C)` out of `r nrow(WTvsKO_hypo)` hypomethylated promoters). 

Promoters that showed the most robust loss of CpGme in _Kdm5c_-KO exEpiLCs (lowest q-values) harbored CGIs (Figure 6K). CGI promoters, but not CGI-free promoters, had a significant reduction in CpGme with KDM5C loss as a whole (Figure 6L) (Non-CGI promoters p = 0.0846, CGI promoters p = 0.0081, Mann-Whitney U test). Significantly hypomethylated promoters included germline genes consistently dysregulated across multiple _Kdm5c_-KO RNA-seq datasets[@scandagliaLossKdm5cCauses2017], such as _D1Pas1_ (methlyation difference = `r signif(getWGBSinfo(WTvsKO_hypo, "D1Pas1", "meth.diff"),4)`%, q-value = `r signif(getWGBSinfo(WTvsKO_hypo, "D1Pas1", "qvalue"),3)`) and _Naa11_ (methlyation difference = `r signif(getWGBSinfo(WTvsKO_hypo, "Naa11", "meth.diff"),4)`%, q-value = `r signif(getWGBSinfo(WTvsKO_hypo, "Naa11", "qvalue"),3)`) (Figure 6M). Surprisingly, we found only a modest reduction in CpGme at _Dazl_'s promoter (methlyation difference = `r signif(getWGBSinfo(WTvsKO, "Dazl", "meth.diff"),4)`%, q-value = `r signif(getWGBSinfo(WTvsKO, "Dazl", "qvalue"),3)`) (Figure 6N). Altogether, these results demonstrate KDM5C is recruited to germline gene CGIs in EpiLCs to promote CpGme at germline gene promoters. Furthermore, this suggests while KDM5C's cataltyic activity is required for repression of some germline genes, some loci can compensate for KDM5C loss through other silencing mechansisms, even when retaining H3K4me2/3 around the TSS. 

# Discussion
In the above study, we demonstrate KDM5C's pivotal role in the development of tissue identity. We first characterized tissue-enriched genes expressed within the mouse _Kdm5c_-KO brain and identified substantial derepression of testis, liver, muscle, and ovary-enriched genes. Testis genes significantly enriched within the _Kdm5c_-KO amygdala and hippocampus are specific to the germline and not expressed within somatic cells. _Kdm5c_-KO epiblast-like cells (EpiLCs) aberrantly express key drivers of germline identity and meiosis, including _Dazl_ and _Stra8_, while the adult brain primarily expresses genes important for late spermatogenesis. We demonstrated that although _Kdm5c_ mutant sex did not influence whether sperm or egg-specific genes were misexpressed, female EpiLCs are more sensitive to germline gene de-repression. Germline genes can become aberrantly expressed in _Kdm5c_-KO cells via an indirect mechanism, such as activation through ectopic RFX transcription factors. Finally, we found KDM5C is dynamically regulated during ESC to EpiLC differentiation to promote long-term germline gene silencing through DNA methylation at CpG islands. Therefore, we propose KDM5C plays a fundamental role in the development of tissue identity during early embryogenesis, including the establishment of the soma-germline boundary. By systematically characterizing KDM5C's role in germline gene repression, we unveiled unique mechansisms governing the misexpression distinct germline gene classes within somatic lineages. Furthermore, these data provide molecular footholds which can be exploited to test the overarching contribution of ectopic germline gene expression upon neurodevelopment.

By comparing _Kdm5c_ mutant males and females, we revealed germline gene supression is sexually dimorphic. Although sex did not impact whether egg or sperm-specific genes were dysregulated, organismal sex did greatly influence the degree of germline gene dysregulation. Female EpiLCs are more severely impacted by loss of KDM5C-mediated germline gene suppression, yet this difference is not due to the increased number of germline genes on the X chromosome[@wangAbundanceXlinkedGenes2001; @khilMouseChromosomeEnriched2004]. Increased female sensitivity to germline gene de-repression may be related to females having a higher dose of KDM5C than males, due to its escape from XCI[@agulnikNovelGeneWidely1994; @carrelTissueLineagespecificVariation1996; @sheardownMouseSmcxGene1996; @xuSexSpecificExpressionXLinked2008]. Intriguingly, females with heterozygous loss of _Kdm5c_ also had over double the number of germline DEGs than hemizygous knockout males, even though their level of KDM5C should be roughly equivalent to that of wild-type males. Males could partially compensate for KDM5C's loss via the Y-chromsome homolog, KDM5D, which exhibits weaker demethylase activity than KDM5C[@iwaseXLinkedMentalRetardation2007]. However, KDM5D has not been reported to regulate germline gene expression. Altogether, these results suggests germline gene silencing mechanims differ between males and females, which warrants further study to elucidate the biological ramifications and underlying mechanisms. 

We found KDM5C is largely dispensible for promoting normal gene expression during development, yet is critical for suppressing ectopic developmental programs. It is important to note that while we highlighted KDM5C's regulation of germline genes, some germline-enriched genes like _Dazl_ are also expressed at the 2-cell stage and in na&#239;ve ESCs/inner cell mass for their role in pluripotency and self-renewal[@suzukiLossMAXResults2016; @torres-padillaTransposonsTotipotency2020;@yangChemicalinducedChromatinRemodeling2022; @guptaGenomewideScreenReveals2023]. These "self-renewal" germline genes are then silenced during ESC differentiation into epiblast stem cells/EpiLCs[@borgelTargetsDynamicsPromoter2010; @mochizukiRepressionGermlineGenes2021]. We found that while _Kdm5c_-KO EpiLCs express _Dazl_, they did not express 2-cell-specific genes like _Zscan4c_. These data suggest the 2-cell-like state reported in _Kdm5c_-KO ESCs[@guptaGenomewideScreenReveals2023] likely reflects KDM5C's primary role in germline gene repression. Germline gene misexpression in _Kdm5c_-KO EpiLCs may indicate they are differentiating into primordial germ cell-like cells (PGCLCs)[@hayashiReconstitutionMouseGerm2011; @magnusdottirHowMakePrimordial2014; @gunesdoganPrimordialGermCell2014]. Yet, _Kdm5c_-KO EpiLCs had normal cellular morphology and properly expressed markers for primed pluripotency, including _Otx2_ which blocks EpiLC differentiation into PGCs/PGCLCs[@zhangOTX2RestrictsEntry2018]. In addition to unimpaired EpiLC differentiation, _Kdm5c_-KO gross brain morphology is overall normal[@iwaseMouseModelXlinked2016] and hardly any brain-specific genes were significantly dysregulated. Thus, ectopic germline gene expression occurs along with overall proper somatic differentiation in _Kdm5c_-KO animals.

Our work provides novel insight into the cross-talk between H3K4me and CpGme, which are often mutually exclusive[@weberDistributionSilencingPotential2007]. In EpiLCs, loss of KDM5C binding at a subset of germline gene promoters, e.g. _D1Pas1_ and _Naa11_, strongly impaired CGI methylation, and resulted in their long-lasting de-repression into adulthood. Removal of H3K4me2/3 at CGIs is a plausible mechanism for KDM5C-mediated germline gene suppression[@scandagliaLossKdm5cCauses2017; @aladhamiSystematicIdentificationFactors2023], given H3K4me2/3 can oppose DNMT3 activity[@otaniStructuralBasisRecognition2009; @guoStructuralInsightAutoinhibition2015]. However, emerging work indicates many histone-modifying enzymes have non-cataltyic functions that influnce gene expression, sometimes even more potently than their catalytic roles[@aubertUnexpectedNoncatalyticRoles2019; @morganReevaluatingRolesHistonemodifying2020]. Indeed, KDM5C's catalytic activity was recently found to be dispensible for repressing _Dazl_ in ESCs [@guptaGenomewideScreenReveals2023]. In our study, _Dazl_'s promoter still gained CpGme in _Kdm5c_-KO exEpiLCs, even with elevated H3K4me2. _Dazl_ and a few other germline gene CGIs use multiple repressive mechanisms to facilitate CpGme[@pohlersRoleE2F6Restriction2005; @endohPCGF6PRC1SuppressesPremature2017; @dahletE2F6InitiatesStable2021; @mochizukiRepressionGermlineGenes2021]. This suggests alternative silencing mechansims are sufficient to recruit DNMT3s to some germline CGIs, while others may require KDM5C-mediated H3K4me removal to overcome promoter CGI escape from CpGme[@longProtectionCpGIslands2016; @weberDistributionSilencingPotential2007]. Furthermore, these results indicate the requirement for catalytic activity can change depending upon the locus and developmental stage, even for the same class of genes. Further experiments are required to determine if catalytically inactive KDM5C can suppress germline genes at later developmental stages.

By generating a comprehensive list of mouse germline-enriched genes, we were able to reveal distinct derepressive mechanisms governing early versus late-stage germline developmental programs. Previous work on germline gene silencing has focused on genes with promoter CGIs[@borgelTargetsDynamicsPromoter2010; @weberDistributionSilencingPotential2007], and indeed the majority of KDM5C targets in EpiLCs were germ cell identity genes harboring CGIs. However, over 70% of germline-enriched gene promoters lacked CGIs, including the many KDM5C-unbound germline genes that were de-repressed in _Kdm5c_-KO cells. CGI-free, KDM5C-unbound germline genes were primarily late-stage spermatogenesis genes and significantly enriched for RFX2 binding sites, a central regulator of spermiogenesis[@kistlerRFX2MajorTranscriptional2015; @wuTranscriptionFactorRFX22016]. These data suggest that once activated during early embryogenesis, drivers of germline identity like _Rfx2_, _Stra8_, and _Dazl_ turn on downstream germline programs, ultimately culminating in the expression of spermiogenesis genes in the adult _Kdm5c_-KO brain. Therefore, we propose KDM5C is recruited via promoter CGIs to genes that shape germ cell formation to act as a break against runaway activation of germline-specific programs. 

The above work provides the mechanistic foundation for KDM5C-mediated repression of tissue and germline-specific genes. However, the contribution of these ectopic, tissue-specific genes towards _Kdm5c_-KO neurological impairments is still unknown. In addition to germline genes, we also identified significant enrichment of muscle and liver-biased transcripts within the _Kdm5c_-KO brain. Intriguingly, select liver and muscle-biased DEGs do have known roles within the brain, such as the liver-enriched lipid metabolism gene _Apolipoprotein C-I (Apoc1)_ [@roulandRoleApolipoproteinC12022]. _APOC1_ dysregulation is implicated in Alzheimer's disease in humans[@leducAPOECholesterolHomeostasis2010] and overexpression of _Apoc1_ in the mouse brain can impair learning and memory[@abildayevaHumanApolipoproteinCI2008]. KDM5C may therefore be crucial for neurodevelopment by fine-tuning the expression of tissue-erniched, dosage-sensitive genes like _Apoc1_. 

Given germline genes have no known functions within the brain, their impact upon neruodevelopment is currently unknown. In _C. elegans_, misexpression of germline genes due to loss of _Retinoblastoma (Rb)_ homologs results in enhanced piRNA signaling ectopic P granule formation in neurons[@wangSomaticMisexpressionGermline2005; @wuRepressionGermlineRNAi2012]. Ectopic testicular germline transcripts have also been observed in a variety of cancers, including brain turmors in _Drosophila_ and mammals[@nielsenEctopicExpressionTestis2016; @adebayobabatundeCancerTestisGenes2017] and shown to promote cancer progression[@janicEctopicExpressionGermline2010; @ghafouri-fardExpressionCancerTestis2012; @ninBiologyCancerTestisAntigens2023]. Intriguingly, mouse models and human cells for other chromatin-linked neurodevelopmental disorders also display impaired soma-germline demarcation[@bonefasSomatogermlineTransformationChromatinlinked2021; @velascoGermlineGenesHypomethylation2014; @waltonDnmt3bPrefersGerm2014; @schaeferControlCognitionAdaptive2009; @samacoCrhOprm1Mediate2012], such as DNA methyltransferase 3b (DNMT3B), H3K9me1/2 methyltransferases G9A/GLP, methyl-CpG -binding protein 2 (MECP2)[@bonefasSomatogermlineTransformationChromatinlinked2021]. Recently, the transcription factor ZMYM2 (ZNF198), whose mutation causes neurodevelopmental-craniofacial syndrome with variable renal and cardiac abnormalities (OMIM #619522), was also shown to repress germline genes by promoting H3K4 methylation removal and DNA methylation[@graham-paquinZMYM2EssentialMethylation2023]. Thus, KDM5C is among a growing cohort of neurodevelopmental disorders with similar erosion of the germline versus soma boundary. Further research is required to determine the impact of these germline genes and the extent to which this phenomenon occurs in humans. 


# Materials and Methods

## Classifying tissue-enriched and germline-enriched genes
Tissue-enriched differentially expressd genes (DEGs) were determined by their classification in a previously published dataset from 17 male and female mouse tissues[@liComprehensiveMouseTranscriptomic2017]. This study defined tissue expression as greather than 1 Fragments Per Kilobase of transcript per Million mapped read (FPKM) and tissue enrichment as at least 4-fold higher expression than any other tissue.

We curated a list of germline-enriched genes using an RNA-seq dataset from wild-type and germline-depleted (Kit^W/Wv^) male and female mouse embryos from embryonic day 12, 14, and 16 [@sohGeneRegulatoryProgram2015], as well as adult male testes[@muellerIndependentSpecializationHuman2013]. Germline-enriched genes met the following criteria: 1) their expression is greater than 1 FPKM in wild-type germline 2) their expression in any wild-type somatic tissues[@liComprehensiveMouseTranscriptomic2017] does not exceed 20% of maximum expression in wild-type germline, and 3) their expression in the germ cell-depleted (Kit^W/Wv^) germline, for any sex or time point, does not exceed 20% of maximum expression in wild-type germline. We defined sperm and egg-biased genes as those whose expression in the opposite sex, at any time point, is no greater than 20% of the gene's maximum expression in a given sex. Genes that did not meet this threshold for either sex were classified as 'unbiased'.

## Cell culture
We utilized our previously established cultures of male wild-type and _Kdm5c_ knockout (-KO) embryonic stem cells[@samantaActivationXistEvolutionarily2022]. Sex was confirmed by genotyping _Uba1_/_Uba1y_ on the X and Y chromosomes with the following primers: 5'-TGGATGGTGTGGCCAATG-3', 5'-CACCTGCACGTTGCCCTT-3'. Deletion of _Kdm5c_ was confirmed through the primers 5'-ATGCCCATATTAAGAGTCCCTG-3', 5'-TCTGCCTTGATGGGACTGTT-3', and 5'-GGTTCTCAACACTCACATAGTG-3'.

Emrbyonic stem cells (ESCs) and epiblast-like cells were cultured using previously established methods[@samantaGeneratingPrimedPluripotent2020]. Briefly, ESCs were initially cultured in primed ESC (pESC) media consisting of KnockOut DMEM (Gibco#10829018), fetal bovine serum (Gibco#A5209501), KnockOut serum replacement (Invitrogen#10828028), Glutamax (Gibco#35050-061), Anti-Anti (Gibco#15240-062), MEM Non-essential amino acids (Gibco#11140-050), and beta-mercaptoethanol (Sigma#M7522). They were then transitioned into ground-state, "na&#239;ve" ESCs (nESCs) by culturing for four passages in N2B27 media containing DMEM/F12 (Gibco#11330032), Neurobasal media (Gibco#21103049), Gluamax (Gibco#35050-061), Anti-Anti (Gibco#15240-062), N2 supplement (Invitrogen#17502048), and B27 supplement without vitamin A (Invitrogen#12587-010), and beta-mercaptoethanol. Both pESC and nESC media were supplemented with 3 $\mu$M GSK3 inhibitor CHIR99021 (Sigma #SML1046-5MG), 1 $\mu$M MEK inhibitor PD0325901 (Sigma #PZ0162-5MG), and 1,000 units/mL leukemia inhibitory factor (LIF, Millipore#ESG1107).

nESCs were differentiated into epiblast-like cells (EpiLCs, 48 hours) and extendend EpiLCs (exEpiLCs, 96 hours) by culturing in N2B27 media containing DMEM/F12, Neurobasal media, Gluamax, Anti-Anti, N2 supplement, B27 supplement (Invitrogen#17504044), beta-mercaptoethanol, fibroblast growth factor 2 (FGF2, R&D Biotechne 233-FB), and activin A (R&D Biotechne 338AC050CF), as previously described [@samantaGeneratingPrimedPluripotent2020]. 

## Real time quantitative PCR (RT-qPCR)
nESCs were differentiated into EpiLCs as described above. Cells were lysed with Tri-reagent BD (Sigma #T3809) at 0, 24, and 48 hours of differentiation. RNA was phase separated with 0.1 uL/uL 1-bromo-3-chloropropane (Sigma #B9673) and then precipitated with with isopropanol (Sigma #I9516). For each sample, 2 ug of RNA was reverse transcribed using the ProtoScript II Reverse transcriptase kit from New England Biolabs (NEB #M0368S) and primed with oligo dT. Expression of _Kdm5c_ was detected using the primers 5'-CCCATGGAGGCCAGAGAATAAG-3' 5'-CTCAGCGGATAAGAGAATTTGCTAC-3' and normalized to TBP with the Power SYBR Green PCR Master Mix (ThermoFisher #4367659).

## Western Blot
Total protein was extracted during nESC to EpiLC differentiation at 0, 24, and 48 hours by sonicating cells at 20% amplitude for 15 seconds in 2X SDS sample buffer, then boiling at 100C for 10 minutes. Proteins were separated on a 7.5% SDS page gel, transferred overnight onto a fluorescent membrane, blotted for rabbit anti-KDM5C (in house, 1:500) and mouse anti-DAXX (Santa Cruz #(H-7): sc-8043, 1:500) imaged using the LiCor Odyssey CLx system. Band intensitity was quantified using ImageJ.

## RNA sequencing (RNA-seq) data analysis
After ensuring read quality via FastQC (v0.11.8), reads were then mapped to the mm10 _Mus musculus_ genome (Gencode) using STAR (v2.5.3a), during which we removed duplicates and kept only uniquely mapped reads. Count files were generated by FeatureCounts (Subread v1.5.0), and BAM files were converted to bigwigs using deeptools (v3.1.3) and visualized by the UCSC genome browser. RStudio (v3.6.0) was then used to analyze counts files by DESeq2 (v1.26.0)[@loveModeratedEstimationFold2014a] to identify differentially expressed genes (DEGs) with a q-value (p-adjusted via FDR/BenjaminiHochberg correction) less than 0.1 and a log2 fold change greater than 0.5. For all DESeq2 analyses, log2 fold changes were calculated with lfcShrink using the ashr package[@stephensFalseDiscoveryRates2016]. MA-plots were generated by ggpubr (v0.6.0), and Eulerr diagrams were generated by eulerr (v6.1.1). Boxplots and scatterplots were generated by ggpubr (v0.6.0) and ggplot2 (v3.3.2). The Upset plot was generated via the package UpSetR (v1.4.0)[@conwayUpSetRPackageVisualization2017]. Gene ontology (GO) analyses were performed by the R package enrichPlot (v1.16.2) using the biological processes setting and compareCluster.

## Chromatin immunoprecipitation followed by DNA sequecing (ChIP-seq) data analysis
ChIP-seq reads were aligned to mm10 using Bowtie1 (v1.1.2) allowing up to two mismatches. Only uniquely mapped reads were used for analysis. Peaks were called using MACS2 software (v2.2.9.1) using input BAM files for normalization, with filters for a q-value < 0.1 and a fold enrichment > 1. We removed black-listed genomic regions that often give aberrant signals. Common peak sets were obtained in R via DiffBind[estrogen@ross-innesDifferentialOestrogenReceptor2012] (v3.6.5). In the case of KDM5C ChIP-seq, _Kdm5c_-KO peaks were then subtracted from wild-type samples using bedtools (v2.25.0). Peak proximity to genome annotations was determined by ChIPSeeker[@yuChIPseekerBioconductorPackage2015] (v1.32.1). Gene ontology (GO) analyses were performed by the R package enrichPlot (v1.16.2) using the biological processes setting and compareCluster. Enriched motifs were identified using HOMER[@heinzSimpleCombinationsLineageDetermining2010]. Average binding across the genome was visualized using deeptools (v3.1.3). Bigwigs were visualized using the UCSC genome browser.

## CpG island (CGI) analysis
Locations of CpG islands were determined through the mm10 UCSC genome browser CpG island track[@nassarUCSCGenomeBrowser2023], which classified CGIs as regions that have greater than 50% GC content, are larger than 200 base pairs, and have a ratio of CG dinucleotides observed over the expected amount greater than 0.6. CGI genomic coordinates were then annotated using ChIPseeker[@yuChIPseekerBioconductorPackage2015] (v1.32.1) and filtered for ones that lie within promoters of our germline-enriched genes (TSS &#177; 500).

## Whole genome bisulfite sequencing (WGBS)
Genomic DNA (gDNA) from na&#239;ve ESCs and extended EpiLCs was extracted using the Wizard Genomic DNA Purification Kit (Promega A1120), following the instructions for Tissue Culture Cells. gDNA from two wild-types and two _Kdm5c_-KOs of each cell type was sent to Novogene for WGBS using the Illumina NovaSeq X Plus platform and sequenced for 150bp paired-end reads (PE150). All samples had greater than 99% bisulfite conversion rates. Reads were adapter and quality trimmed with Trim Galore (v0.6.10) and aligned to the mm10 genome using Bismark[@kruegerBismarkFlexibleAligner2011] (v0.22.1). Analysis of differential methylation at germline gene promoters was perfomed using methylKit[@akalinMethylKitComprehensivePackage2012] (v1.28.0) with a minimum coverage of 3 paired reads, a percentage cut-off of 25%, and q-value of 0.01. methylKit was also used to calculate average percentage methylation at germline gene promoters. Methylation bedgraph tracks were generated via Bismark and visualized using the UCSC genome browser.

## Data availability
### WGBS in wild-type and _Kdm5c_-KO ESCs and exEpiLCs
XXXX

### Published datasets
All published datasets are available at the Gene Expression Omnibus (GEO) https://www.ncbi.nlm.nih.gov/geo. Published RNA-seq datasets analyzed in this study included the male wild-type and _Kdm5c_-KO adult amygdala and hippocampus[@vallianatosMutuallySuppressiveRoles2020] (available at GEO: GSE127722) and male wild-type and _Kdm5c_-KO EpiLCs[@samantaActivationXistEvolutionarily2022] (available at GEO: GSE96797). 

Previously published ChIP-seq experiments included KDM5C in wild-type and _Kdm5c_-KO EpiLCs[@samantaActivationXistEvolutionarily2022] (available at GEO: GSE96797) and mouse primary neuron cultures (PNCs) from the cortex and hippocampus[@iwaseMouseModelXlinked2016] (available at GEO: GSE61036). ChIP-seq of histone 3 lysine 4 dimethylation in male wild-type and _Kdm5c_-KO EpiLCs[@samantaActivationXistEvolutionarily2022] is also available at GEO: GSE96797. ChIP-seq of histone 3 lysine 4 trimethylation in wild-type and _Kdm5c_-KO male amygdala[@vallianatosMutuallySuppressiveRoles2020] are avilable at GEO: GSE127817.

### Data analysis
Scripts used to generate the results, tables, and figures of this study are available via the GitHub repository: XXX

# Acknowledgements
We thank Drs. Sundeep Kalantry, Milan Samanta, and Rebecca Malcore for providing protocols and expertise in culturing mouse ESCs and EpiLCs, as well as providing wild-type and _Kdm5c_-KO ESCs used in this study. We thank Dr. Jacob Mueller for his insight in germline gene regulation and directing us to the germline-depleted mouse models. We also thank Drs. Gabriel Corfas, Kenneth Kwan, Natalie Tronson, Michael Sutton, Stephanie Bielas, Donna Martin, and the members of the Iwase, Sutton, Bielas, and Martin labs for helpful discussions and critiques of the data. We thank members of the University of Michigan Reproductive Sciences Program for providing feedback throughout the development of this work. This work was supported by grants from the National Institutes of Health (NIH) (National Institute of Neurological Disorders and Stroke: NS089896, 5R21NS104774, and NS116008 to S.I.), Farrehi Family Foundation Grant (to S.I.), the University of Michigan Career Training in Reproductive Biology (NIH T32HD079342, to K.M.B.), the NIH Early Stage Training in the Neurosciences Training Grant (T32-NS076401 to K.M.B.), and the Michigan Predoctoral Training in Genetics Grant (NIH T32GM007544, to I.V.)

# Author contributions
K.M.B. and S.I. conceived the study and designed the experiments. I.V. generated the ESC and exEpiLC WGBS data. K.M.B performed the data analysis and all other experiments. K.M.B and S.I. wrote and edited the manuscript. 


# References
<div id="refs"></div>

\newpage
# Figures and Tables
- Supplementary table 1: list of all germline genes. 
	- Columns to include: 
		- KDM5C bound vs not
		- Log2fc in EpiLC, brain (separate columns?)
    - CGI vs non


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/Brain_tissue_genes.pdf}
  \caption[Tissue-enriched genes are misexpressed in the \textit{Kdm5c}-KO brain]{\textbf{Tissue-enriched genes are misexpressed in the \textit{Kdm5c}-KO brain.} \textbf{A-B.} Expression of tissue-enriched genes (Li et al 2017) in the male \textit{Kdm5c}-KO amygdala (A) and hippocampus (B). Left - MA plot of mRNA-sequencing. Right - Number of tissue-enriched differentially expressed genes (DEGs). * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test \textbf{C.} Left - UCSC browser view of an example aberrantly expressed testis-enriched DEG, \textit{FK506 binding protein 6 (Fkbp6)} in the wild-type (WT) and \textit{Kdm5c}-KO (5CKO) amygdala (red) and hippocampus (teal) (Average, n = 4). Right - Expression of \textit{Cyct} in wild-type tissues from NCBI Gene, with testis highlighted in blue and brain tissues highlighted in red. \textbf{D.} Left - UCSC browser view of an example ovary-enriched DEG, \textit{Zygotic arrest 1 (Zar1)}. Right - Expression of \textit{Zar1} in wild-type tissues from NCBI Gene, with ovary highlighted in teal and brain tissues highlighted in red. \textbf{E.} Left - UCSC browser view of an example liver-enriched DEG, \textit{Apolipoprotein C-I (Apoc1)}. Right - Expression of \textit{Apoc1} in wild-type tissues from NCBI Gene, with liver highlighted in orange and brain tissues highlighted in red.}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/What_are_germline_genes.pdf}
  \caption[Aberrant transcription of germline genes in the \textit{Kdm5c}-KO in the brain]{\textbf{Aberrant transcription of germline genes in the \textit{Kdm5c}-KO in the brain.} \textbf{A.} enrichPlot gene ontology (GO) of \textit{Kdm5c}-KO amygdala and hippocampus testis-enriched DEGs \textbf{B.} Expression of testis DEGs in wild-type (WT) testis versus germ cell-depleted (W/Wv) testis (Mueller et al 2013). Expression is in Fragments Per Kilobase of transcript per Million mapped reads (FPKM). \textbf{C.} Number of testis DEGs that were classified as cell-type specific markers in a single cell RNA-seq dataset of the testis (Green et al 2018). Germline cell types are highlighted in green, somatic cell types in black. \textbf{D.} Sankey diagram of mouse genes filtered for germline enrichment based on their expression in wild-type and W/Wv mice and in adult mouse non-gonadal tissues (Li et al 2017).}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/EpiLCs_too.pdf}
  \caption[\textit{Kdm5c}-KO epiblast-like cells express key drivers of germline identity ]{\textbf{\textit{Kdm5c}-KO epiblast-like cells express key drivers of germline identity A.} Top - Diagram of \textit{in vivo} differentiation of embryonic stem cells (ESCs) of the inner cell mass into epiblast stem cells. Bottom - \textit{in vitro} differentiation of ESCs into epiblast-like cells (EpiLCs). \textbf{B.} Representative images of male wild-type (WT) and \textit{Kdm5c}-KO (5CKO) cells during ESC to EpiLC differentiation. Brightfield images taken at 20X. \textbf{C.} No significant difference in primed pluripotency maker expression in wild-type versus \textit{Kdm5c}-KO EpiLCs. Welch's t-test, expression in transcripts per million (TPM). \textbf{D.} Number of tissue-enriched differentially expressed genes (DEGs) in \textit{Kdm5c}-KO EpiLCs. * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test. \textbf{E.} Upset plot displaying the overlap of germline DEGs expressed in \textit{Kdm5c}-KO EpiLCs, amygdala (AMY), and hippocampus (HIP) RNA-seq datasets. \textbf{F.} UCSC browser view of an example germline gene, \textit{D1Pas1}, that is dysregulated \textit{Kdm5c}-KO EpiLCs (top, purple. Average, n = 3), amygdala (middle, red. Average, n = 4), and hippocampus (bottom, blue. Average, n = 4). \textbf{G.} enrichPlot gene ontology analysis comparing enriched biological processes for \textit{Kdm5c}-KO EpiLC, amygdala, and hippocampus germline DEGs. \textbf{H.} Top left - Example germline identity DEGs unique to EpiLCs. Top right - Example 2-cell genes that are not dysregulated in \textit{Kdm5c}-KO EpiLCs. p-values for Welch's t-test. Bottom - UCSC browser view of \textit{Dazl} and \textit{Stra8} expression in wild-type and \textit{Kdm5c}-KO EpiLCs (Average, n = 3).}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/EpiLC_XXvsXY.pdf}
  \caption[Chromosomal sex influences \textit{Kdm5c}-KO germline gene misexpression.]{\textbf{Chromosomal sex influences \textit{Kdm5c}-KO germline gene misexpression.} \textbf{A.} Total number of germline-enriched RNA-seq DEGs for male hemizygous \textit{Kdm5c} knockout EpiLCs (XY 5CKO, purple), female heterozygous \textit{Kdm5c} knockout (XX 5CHET, orange), female homozygous \textit{Kdm5c} knockout (XX 5CKO, light orange) EpiLCs. \textbf{B.} Left - Eulerr overlap of \textit{Kdm5c} mutant male and female EpiLC germline DEGs. Right - Example of germline DEGs unique to females or males, \textit{Tktl2} and \textit{Esx1}.\textbf{C.} enrichPlot gene ontology analysis comparing enriched biological processes for germline DEGs shared between \textit{Kdm5c} mutant males and females (Shared), or unique to one sex (XX only or XY only). \textbf{D.} Heatmap of germline DEGs shared between male and female mutants. Color is the average log 2 fold change from sex-matched wild-type. \textbf{E.} Number of genes within each cluster from D. Clusters with higher expression in females compared to males (XX-biased) highlighted in orange. \textbf{F.}  UCSC browser view of a male and female shared germline DEG \textit{D1Pas1} that is more highly expressed in female mutants (Average, n = 3). \textbf{G.} Left - Number of all female germline DEGs located on each chromosome over the total number of germline-enriched genes on that chromsome. Right - Percentage of germline DEGs that lie on each chromosome for each \textit{Kdm5c} mutant. X chromosome highlighted in black. \textbf{H.} Sankey diagram classifying egg-biased (pink) and sperm-biased (blue) and unbiased (purple) mouse germline-enriched genes. \textbf{I.} Number of egg, sperm, or unbiased germline DEGs for male and female \textit{Kdm5c} mutants. \textbf{J.} Example bigwigs of egg-biased (\textit{Zp3}), sperm-biased (\textit{Dkkl1}), and unbiased (\textit{Dazl}) germline genes dysregulated in both male and female \textit{Kdm5c} mutants.}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/KDM5C_ChIPseq.pdf}
  \caption[KDM5C binds to a subset of germline gene promoters during early embryogenesis.]{\textbf{KDM5C binds to a subset of germline gene promoters during early embryogenesis.} \textbf{A.}  ChIPseeker localization of KDM5C peaks at different genomic regions in EpiLCs (top) and hippocampal and cortex primary neuron cultures (PNCs, bottom). \textbf{B.} Overlap of genes with KDM5C bound to their promoters (TSS $\pm$ 500) in EpiLCs (purple) and PNCs (blue). \textbf{C.} Gene ontology (GO) comparison of genes with KDM5C bound to their promoter in EpiLCs and PNCs. Genes were classified as either bound in EpiLCs only (EpiLC only), unique to PNCs (PNC only, no significant ontologies) or bound in both PNCs and EpiLCs (shared). \textbf{D.} Average KDM5C binding around the transcription start site (TSS) of all germline-enriched genes in EpiLCs (left) and PNCs (right). \textbf{E.} Eulerr overlap of germline-enriched genes (green) with significant KDM5C binding at their promoter in EpiLCs (purple). \textbf{F.} Example KDM5C ChIP-seq signal around the \textit{Dazl} TSS but not \textit{Stra8} in EpiLCs. \textbf{G.} HOMER motif analysis of all KDM5C-bound germline gene promoters, highlighting significant enrichment of MAX, E2F6, and E2F motifs. \textbf{H.} Number of all gene promoters bound or unbound by KDM5C with instances of the E2F or E-box consensus sequence. \textbf{I.} HOMER motif analysis of all KDM5C-unbound germline gene promoters, highlighting significant enrichment of RFX family transcription factor motifs. \textbf{J.} Expression of RNA-seq DEG \textit{Rfx2} in wild-type and \textit{Kdm5c}-KO EpiLCs. P-value of Welch's t-test, expression in transcripts per million (TPM).}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering
  \includegraphics{../submission/compiled_figs/KDM5C_Mechanism.pdf}
  \caption[KDM5C promotes long-term silencing of germline genes via DNA methylation at CpG islands]{\textbf{KDM5C promotes long-term silencing of germline genes via DNA methylation at CpG islands.} \textbf{A.} Diagram of embryonic stem cell (ESC) to epiblast-like cell (EpiLC) differentiation and collection time points. \textbf{B.} Real time quantitative PCR (RT-qPCR) of \textit{Kdm5c} mRNA expression in wild-type (WT) and \textit{Kdm5c}-KO (5CKO) ESCs at 0, 24, and 48 hours of differentiation into EpiLCs. Expression calculated in comparison to TBP mRNA expression (2\textsuperscript{-deltaCT}). * p < 0.05, ** p < 0.01, *** p < 0.001, Welch's t-test. \textbf{C.} KDM5C protein expression normalized to DAXX. Quantified intensity using ImageJ (artificial units - au). Right - representative lanes of Western blot for KDM5C and DAXX. * p < 0.05, ** p < 0.01, *** p < 0.001, Welch's t-test. \textbf{D.} Top - Representative UCSC browser view of histone 3 lysine 4 dimethylation (H3K4me2) ChIP-seq signal at two germline genes in wild-type and \textit{Kdm5c}-KO EpiLCs. Bottom - Average H3K4me2 signal at the TSS of all germline-enriched genes in wild-type (dark purple) and \textit{Kdm5c}-KO (light purple) EpiLCs. \textbf{E.} Top - Representative UCSC browser view of histone 3 lysine 4 trimethylation (H3K4me3) ChIP-seq signal at two germline genes in the wild-type (WT) and \textit{Kdm5c}-KO (5CKO) adult amygdala. Bottom - Average H3K4me3 signal at the transcription start site (TSS) of all germline-enriched genes in wild-type (dark red) and \textit{Kdm5c}-KO (light red) amygdala. \textbf{F.} Percentage of germline genes that harbor CpG islands (CGIs) in their promoters (CGI promoter, red), based on UCSC annotation. Left - percentage of all germline-enriched genes, middle - KDM5C-bound germline genes, and right - KDM5C-unbound germline genes. (Legend continued on next page.)}
  \label{figurelabel}
\end{figure}
\newpage
\addtocounter{figure}{-1}
\begin{figure} [t!]
  \caption[]{\textbf{KDM5C promotes long-term silencing of germline genes via DNA methylation at CpG islands.} (Legend continued.) \textbf{G.} enrichPlot gene ontology analysis of germline genes with (CGI-promoter) or without (CGI-free) CGIs in their promoter. \textbf{H.} Expression of germline genes with (CGI-promoter, red) or without (CGI-free, salmon) CGIs in their promoter across stages of spermatogenesis from Green et al 2018. SPG - spermatogonia, Pre-Lep - preleptotene spermatocytes, S-Cytes - meiotic spermatocytes, S-Tids - post-meiotic haploid round spermatids, E.Sp - elongating spermatids. Wilcoxon test, * p < 0.05, ** p < 0.01, *** p < 0.001, **** p < 0.0001. \textbf{I.} Diagram of ESC to extended EpiLC (exEpiLC) differentiation. \textbf{J.} Volcano plot of whole genome bisulfite sequencing (WGBS) comparing CpG methylation (CpGme) at germline gene promoters (TSS $\pm$ 500) in wild-type (WT) ESCs versus exEpiLCs. Significantly differentially methylated promoters (q < 0.01, |methylation difference| > 25\%) with CGIs in red, CGI-free promoters in light blue \textbf{K.} Volcano plot of WGBS of wild-type (WT) versus \textit{Kdm5c}-KO exEpiLCs for germline gene promoters. Promoters with CGIs in red, CGI-free promoters in light blue. \textbf{L.} Histogram of average percent CpGme at the promoter of germline genes with or without CGIs. Wild-type in navy and \textit{Kdm5c}-KO (KO) in light blue. Dashed lines are average methylation for each genotype, p-values for Mann-Whitney U test. \textbf{M.} UCSC browser view of germline genes, showing UCSC annotated CGI with number of CpGs, representative CpGme in wild-type (WT) and \textit{Kdm5c}-KO (5CKO) ESCs and exEpiLCs, and KDM5C ChIP-seq signal in EpiLCs.}
\end{figure}

\clearpage




