---
title:  Erosion of somatic tissue identity with loss of the X-linked intellectual disability factor KDM5C
output: pdf_document
geometry: margin = 1in
font-size: 11pt
header-includes:
   - \usepackage{caption}
   - \captionsetup[figure]{font=footnotesize, labelfont=bf}
   - \usepackage{sectsty} 
   - \usepackage{helvet}
   - \renewcommand{\familydefault}{\sfdefault}
   - \usepackage{lineno}
   - \linenumbers
   - \renewcommand\linenumberfont{\normalfont\small}
   - \usepackage{setspace}
   - \usepackage{siunitx}
   - \spacing{1.5}
   - \pagenumbering{arabic}
   - \usepackage{indentfirst}

bibliography: references.bib
csl: cell-reports.csl
indent: true
editor_options:
  chunk_output_type: console
---
```{r, knitrcommas, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#adds commas to large numbers
library(huxtable)

knitr::knit_hooks$set(inline = function(x) {
  prettyNum(x, big.mark=",")
})

```
Katherine M. Bonefas, Ilakkiya Venkatachalam, and Shigeki Iwase.

# Abstract 

Mutations in numerous chromatin-modifying enzymes cause neurodevelopmental disorders (NDDs). While, loss of repressive chromatin regulators can lead to the aberrant transcription of tissue-specific genes outside of their intended context, the mechanisms and consequences are largely unknown. Here, we explored this cellular identity crisis in NDDs by investigating the role of lysine demethylase 5c (KDM5C), an eraser of histone 3 lysine 4 di and tri-methylation (H3K4me2/3). We found male _Kdm5c_ knockout (-KO) mice, which recapitulate key behavioral phenotypes of Claes-Jensen X-linked intellectual disability, aberrantly expresses many liver, muscle, ovary, and testis genes within the amygdala and hippocampus. Gonad-enriched genes expressed in the _Kdm5c_-KO brain were typically unique to germ cells, indicating an erosion of the soma-germline boundary. Germline genes are typically decommissioned in somatic lineages in the post-implantation epiblast, yet _Kdm5c_-KO epiblast-like cells (EpiLCs) aberrantly expressed key regulators of germline identity and meiosis, including _Dazl_ and _Stra8_. Germline gene dysregualtion was sexually dimorphic, as female _Kdm5c_-KO EpiLCs were more prone to germline gene misexpression than knockout males. Using a curated list of mouse germline-enriched genes, we found KDM5C is selectively recruited to a subset of germline gene promoters that contain CpG islands (CGIs) to facilitate DNA CpG methylation (CpGme) during ESC to EpiLC differentiation. However, CGI-free germline gene promoters lacking KDM5C, particularly late-stage spermatogenesis genes, can also become activated in _Kdm5c_-KO cells via ectopic activation by RFX transcription factors. Thus, ectopic germline transcriptional programs can progress in the background of typical _Kdm5c_-KO development, due to downstream activation by key drivers of germline gene expression. 


# Introduction

A single genome holds the instructions to generate the myriad of cell types found an organism. This is, in part, accomplished by chromatin regulators that can either promote or impede lineage-specific gene expression through DNA and histone modifications[@strahlLanguageCovalentHistone2000; @jenuweinTranslatingHistoneCode2001; @lewisGeneComplexControlling1978a; @kennisonDosagedependentModifiersPolycomb1988a; @kassisPolycombTrithoraxGroup2017]. Human genetic studies revealed mutations in chromatin regulators are a major cause of neurodevelopmental disorders (NDDs)[@gabrieleChromatinBasisNeurodevelopmental2018] and many studies have identified their importance in regulating the expression of brain-specific genes. Loss of some chromatin regulators can also result in the ectopic transcription of tissue-specific genes outside of their target environment, such as the misexpression of liver-specific genes within adult neurons[@schaeferControlCognitionAdaptive2009]. However, the mechanisms underlying ectopic gene expression and its impact upon neurodevelopment are poorly understood. 

To elucidate the role of tissue identity in chromatin-linked NDDs, it is essential to first characterize the nature of the dysregulated genes and the molecular mechanisms governing their de-repression. Here we focus on lysine demethylase 5C (KDM5C), also known as SMCX or JARID1C, which erases histone 3 lysine 4 di- and trimethylation (H3K4me2/3), a permissive chromatin modification enriched at gene promoters[@iwaseXLinkedMentalRetardation2007]. Pathogenic mutations in _KDM5C_ cause Intellectual Developmental Disorder, X-linked, Syndromic, Claes-Jensen Type (MRXSCJ, OMIM: 300534). MRXSCJ is more common and severe in males and its neurological phenotypes include intellectual disability, seizures, aberrant aggression, and autistic behaviors[@claesNovelSyndromicForm2000; @jensenMutationsJARID1CGene2005; @carmignacFurtherDelineationFemale2020]. Male _Kdm5c_ knockout (-KO) mice recapitulate key MRXSCJ phenotypes, including hyperaggression, increased seizure propensity, and learning impairments[@iwaseMouseModelXlinked2016; @scandagliaLossKdm5cCauses2017]. RNA sequencing (RNA-seq) of the _Kdm5c_-KO hippocampus revealed ectopic expression of some germline genes within the brain[@scandagliaLossKdm5cCauses2017]. However, it is unknown whether other types of testis genes are dysregulated, at what point in development germline gene misexpression begins, and if other tissue-specific genes are aberrantly transcribed with KDM5C loss. 

Distinguishing between germ cells and somatic cells is a key feature of multicellularity[@devlinPanmetazoanViewGermlinesoma2023] that occurs during early embryogenesis in many metazoans[@lehmannGermlineStemCells2012]. In mammals, chromatin regulators are crucial for decommissioning germline genes in somatic cells during the transition from na&#239;ve to primed pluripotency. Initially, germline gene promoters gain repressive histone H2A lysine 119 monoubiquitination (H2AK119ub1)[@endohPCGF6PRC1SuppressesPremature2017] and histone 3 lysine 9 trimethylation (H3K9me3)[@endohPCGF6PRC1SuppressesPremature2017; @mochizukiRepressionGermlineGenes2021] in embryonic stem cells and are then decorated with DNA CpG methylation (CpGme) in the post-implantation embryo[@velascoDnmt3bRecruitmentE2F62010; @hackettPromoterDNAMethylation2012; @mochizukiRepressionGermlineGenes2021]. The contribution of KDM5C to this process remains unclear. It is additionally unknown if KDM5C-mediated germline gene repression is impacted by chromosomal sex, as females have a higher KDM5C dose than males due to its escape from X chromosome inactivation[@agulnikNovelGeneWidely1994; @carrelTissueLineagespecificVariation1996; @sheardownMouseSmcxGene1996; @xuSexSpecificExpressionXLinked2008]. Furthermore, studies on germline gene repression have primarily focused on marker genes important for germ cell development rather than germline genes as a whole, given the lack of a curated list for germline-enriched genes. Therefore, it is unknown if the mechanism of repression differs for certain classes of germline genes, e.g. meiotic genes versus spermatid differentiation genes. 

To illuminate KDM5C's role in tissue identity, here we characterized the aberrant expression of tissue-enriched genes within the _Kdm5c_-KO brain and epiblast-like stem cells (EpiLCs), an _in vitro_ model of the post-implantation embryo. We curated list of mouse germline-enriched genes, which enabled genome-wide analysis of germline gene silencing mechanisms for the first time. Based on the data presented below, we propose KDM5C plays a fundamental role in the development of tissue identity during early embryogenesis, including the establishment of the soma-germline boundary. 


# Results

## Tissue-enriched genes are aberrantly expressed in the _Kdm5c_-KO brain

```{r, DESeq2_params, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the parameters used repeatedly, like the log2fold change cut off
source("../code/utilities/parameters.R")
```

Previous RNA sequencing (RNA-seq) of the adult male _Kdm5c_-KO hippocampus revealed ectopic expression of some germline genes unique to the testis[@scandagliaLossKdm5cCauses2017]. It is currently unclear if the testis is the only tissue type misexpressed in the _Kdm5c_-KO brain. We thus characterized the role of KDM5C in brain tissue identity by systematically assessing the dysregulation of genes specific to or enriched in 17 mouse tissues[@liComprehensiveMouseTranscriptomic2017]. We analyzed our published mRNA-seq datasets of the adult amygdala and hippocampus from wild-type and constitutive _Kdm5c_-KO male mice[@vallianatosMutuallySuppressiveRoles2020].

```{r, tissue_genes, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the number and stats for tissue-enriched gene expression
tissue_res <- read.csv("../data/processed/TissueSpecific_AMYHIP_numberofgenesDEGs.csv", header = TRUE)

#get the value you want to print 
#col is which data column, tiss is which tissue
gettissval <- function(col, tiss){
	val <- subset(tissue_res, tissue == tiss)
	val <- val[,col]
	return(val)
}

```

We found a large proportion significantly upregulated genes within the _Kdm5c_-KO brain (DESeq2[@loveModeratedEstimationFold2014a], log2 fold change > `r l2fcco`, q < 0.1) are typically enriched in non-brain tissues (Amygdala: 35%, Hippocampus: 24%) (Figure 1A-B). For both the amygdala and hippocampus, the majority of tissue-enriched differentially expressed genes (DEGs) were testis genes (Figure 1A-C). Even though the testis has the largest total number of tissue-biased genes (`r gettissval("count", "Testis")` genes) compared to any other tissue, testis-biased DEGs were significantly enriched for both brain regions (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Testis"), 3)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Testis"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Testis"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Testis"), 3)`, Fisher's Exact Test). One example of a testis-enriched gene misexpressed in the _Kdm5c_-KO brain is _FK506 binding protein 6 (Fkbp6)_, a known regulator of PIWI-interacting RNAs (piRNAs) and meiosis[@crackowerEssentialRoleFkbp62003; @xiolRoleFkbp6Chaperone2012] (Figure 1C). 

Interestingly, we also observed significant enrichment of ovary-biased DEGs in both the amygdala and hippocampus (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Ovary"), 3)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Ovary"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Ovary"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Ovary"), 3)`, Fisher's Exact) (Figure 1A-B). Ovary-enriched DEGs included _Zygotic arrest 1 (Zar1)_, which sequesters mRNAs in oocytes for meiotic maturation[@chengMammalianOocytesStore2022] (Figure 1D). Given that the _Kdm5c_-KO mice we analyzed are male, these data demonstrate that the ectopic expression of tissue-enriched genes is independent of organismal sex.

Although not consistent across brain regions, we also found significant enrichment of DEGs biased towards two non-gonadal tissues - the liver (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Liver"), 2)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Liver"), 3)`, Fisher's Exact Test) and the muscle (Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Muscle"), 2)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Muscle"), 3)`, Fisher's Exact Test) (Figure 1A-B). Among liver-biased DEG derepressed in both the hippocampus and amygdala is _Apolipoprotein C-I (Apoc1)_, a lipoprotein metabolism and transport gene[@roulandRoleApolipoproteinC12022] that has been implicated in Alzheimer's disease[@leducAPOECholesterolHomeostasis2010] (Figure 1E). 

For all _Kdm5c_-KO tissue-enriched DEGs, aberrantly expressed mRNAs are polyadenylated and spliced into mature transcripts (Figure 1C-E). Of note, we observed little to no dysregulation of brain-enriched genes (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Brain"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Brain"), 3)`, Fisher's Exact), despite the fact these are brain samples and the brain has the second highest total number of tissue-enriched genes (`r gettissval("count", "Brain")` genes). Altogether, these results suggest the aberrant expression of tissue-enriched genes within the brain is a major effect of KDM5C loss.

```{r, germgenes, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
germgenes <- read.csv("../data/processed/germGENES20.csv", header = TRUE)

#testis gene ontology results
testisGO <- read.csv("../results/GO_Brain_TestisDEGs.csv")

#function to get the GO ID
getGO <- function(x, df, column, cluster){
	df_test <- df
	
	if (cluster == "no") {
		out <- df_test[df_test$Description == x,][column][,1]
	} else {
		out <- df_test[df_test$Description == x & df_test$Cluster == cluster,][column][,1]
	}
	outformat <- ifelse(column == "ID", toString(out), out) #an attempt to remove the commas from GO ID numbers, didn't work
	return(outformat) 
}

```
### Germline genes are misexpressed in the _Kdm5c_-KO brain

In addition to germ cells, the testis also contains both somatic cells, such as Leydig cells that support hormone production and germline functions. To determine if _Kdm5c_-KO results in ectopic expression of somatic testicular genes, we first evaluated their known functions through gene onotology analysis. We found _Kdm5c_-KO testis-enriched DEGs had high enrichment of germline-relevant ontologies, including spermatid development (GO: 0007286, p.adjust = `r signif(getGO("spermatid development", testisGO, 'p.adjust', "no"), 3)`) and sperm axoneme assembly (GO: 0007288, p.adjust = `r signif(getGO("sperm axoneme assembly", testisGO, 'p.adjust', "no"), 3)`) (Figure 2A). 

To validate the cell-type origin of testicular DEGs, we then compared their expression in somatic and germ cells within the testis. We first evlauted their expression in wild-type testes versus testes with germ cell depletion[@muellerIndependentSpecializationHuman2013], which was accomplished by heterozygous _W_ and _Wv_ mutations in the enzymatic domain of _c-Kit_ (Kit^W/Wv^)[@handelSertoliCellDifferentiation1979]. Almost all _Kdm5c_-KO testis-enriched DEGs lost expression with germ cell depletion (Figure 2B). We then assessed testis-enriched DEG expression in a published single cell RNA-seq dataset that identified cell type-specific markers within the testis[@greenComprehensiveRoadmapMurine2018]. Some _Kdm5c_-KO testis-enriched DEGs were classified as specific markers for different germ cell developmental stages (e.g. spermatogonia, spermatocytes, round spermatids, and elongating spermatids), yet none marked somatic cells (Figure 2C). Together, these data demonstrate that the _Kdm5c_-KO brain aberrantly expresses germline genes, but not somatic testicular genes, reflecting an erosion of the soma-germline boundary.

As of yet, research on germline silencing mechanisms has focused on a handful of key genes rather than assessing germline gene suppression genome-wide due to the lack of a comprehensive mouse germline gene list. We therefore generated a list of germline-enriched genes using RNA-seq datasets of Kit^W/Wv^ mice that included males and females at embryonic day 12, 14, and 16[@sohGeneRegulatoryProgram2015] and adult male testes[@muellerIndependentSpecializationHuman2013]. We defined genes as germline-enriched if their expression met the following criteria: 1) their expression is greater than 1 FPKM in wild-type gonads 2) their expression in any non-gonadal tissue of adult wild type mice[@liComprehensiveMouseTranscriptomic2017] does not exceed 20% of their maximum expression in the wild-type germline, and 3) their expression in the germ cell-depleted gonads, for any sex or time point, does not exceed 20% of their maximum expression in the wild-type germline. These criteria yielded `r nrow(germgenes)` germline-enriched genes (Figure 2D), which was hereafter used as a resource to globally characterize germline gene misexpression with _Kdm5c_ loss (Supplementary table 1).

## _Kdm5c_-KO epiblast-like cells aberrantly express key regulators of germline identity 

```{r, epilcs, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}

#testis gene ontology results
epigerm <- read.csv("../results/DESeq2/germDEGs/germDEGs_EpiLC_XY5cKO.csv")
GO_EpiBrain <- read.csv("../results/GO_EpiLC_vs_Brain.csv")

epi_restab <- read.csv("../data/processed/restable_EpiLC_XY5cKO.csv", row.names = 1)
epi_restab$ENSEMBL <- rownames(epi_restab)

zscan4d <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000090714',]['padj'][,1]
duxf3 <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000075046',]['padj'][,1]


```

Germ cells are typically distinguished from somatic cells soon after the embryo implants into the uterine wall[@magnusdottirHowMakePrimordial2014; @gunesdoganPrimordialGermCell2014]. A subset of epiblast stem cells become the primordial germ cells (PGCs) while the remainder differentiate into the ectoderm, mesoderm, and endoderm to form the somatic tissues[@bardotMouseGastrulationCoordination2020]. This developmental time point can be modeled _in vitro_ through differentiation of embryonic stem cells (ESCs) into epiblast-like stem cells (EpiLCs) (Figure 3A)[@hayashiReconstitutionMouseGerm2011; @samantaGeneratingPrimedPluripotent2020]. While some germline-enriched genes are also expressed in na&#239;ve embryonic stem cells (ESCs) and in the 2-cell stage[@wellingDAZLRegulatesTet12015; @macfarlanEmbryonicStemCell2012; @suzukiLossMAXResults2016], they are silenced as they differentiate into EpiLCs[@borgelTargetsDynamicsPromoter2010; @mochizukiRepressionGermlineGenes2021]. Therefore, we tested if KDM5C was necessary for the initial silencing germline genes in somatic lineages by evaluating the impact of _Kdm5c_ loss in male EpiLCs. 

_Kdm5c_-KO cell morpholgy during ESC to EpiLC differentiation appeared normal (Figure 3A) and EpiLCs properly expressed markers of primed pluripotency, such as _Dnmt3b_, _Fgf5_, _Pou3f1_, and _Otx2_ (Figure 3B). We then identified tissue-enriched DEGs in our previously published RNA-seq dataset of wild-type and _Kdm5c_-KO EpiLCs[@samantaActivationXistEvolutionarily2022] (DESeq2, log2 fold change > `r l2fcco`, q < 0.1). Similar to the _Kdm5c_-KO brain, we observed general dysregulation of tissue-enriched genes, with the largest number of genes belonging to the brain and testis, although they were not significantly enriched (Figure 3B). 

Using the list of mouse germline-enriched genes curated above, we found `r nrow(epigerm)` were misexpressed in male _Kdm5c_-KO EpiLCs. We then compared EpiLC germline DEGs to those expressed in the _Kdm5c_-KO brain to determine if germline genes are constitutively dysregulated or change over the course of development. We found the majority of germline DEGs were unique to either EpiLCs or the brain, with only _Cyct_ shared across all tissue/cell types (Figure 3E-F). EpiLCs had particularly high enrichment of meiosis-related gene ontologies (Figure 3G), such as meiotic cell cycle process (GO:1903046, p.adjust = `r signif(getGO("meiotic cell cycle process", GO_EpiBrain, 'p.adjust', "EpiLC"), 3)`) and meiotic nuclear division (GO:0140013, p.adjust = `r signif(getGO("meiotic nuclear division", GO_EpiBrain, 'p.adjust', "EpiLC"), 3)`). While there was modest enrichment of meiotic gene ontologies in both brain regions, the _Kdm5c_-KO hippocampus primarily expressed late-stage spermatogenesis genes involved in sperm axoneme assembly (GO:0007288, p.adjust = `r signif(getGO("sperm axoneme assembly", GO_EpiBrain, 'p.adjust', "Hippocampus"), 3)`) and sperm motility (GO:0097722, p.adjust = `r signif(getGO("sperm motility", GO_EpiBrain, 'p.adjust', "Hippocampus"), 3)`).

Notably, DEGs unique to _Kdm5c_-KO EpiLCs included key drivers of germline identity, such as _Stimulated by retinoic acid 8 (Stra8)_ and _Deleted in azoospermia like (Dazl)_ (Figure 3H). These genes are typically expressed when PGCs commit to the germline fate, but are also expressed later in life to trigger meiotic gene expression programs[@koubovaRetinoicAcidRegulates2006; @linGermCellIntrinsicExtrinsic2008; @endoRetinoicAcidGerm2019]. Of note, some germline genes, including _Dazl_, are also expressed in the two-cell embryo[@macfarlanEmbryonicStemCell2012; @guptaGenomewideScreenReveals2023]. However, we did not see derepression of two-cell stage-specific genes, like _Duxf3 (Dux)_ (q = `r signif(duxf3, 3)`) and _Zscan4d_ (q = `r signif(zscan4d, 3)`) (Figure 3H), indicating _Kdm5c_-KO EpiLCs do not revert back to a 2-cell-like state. Altogether, these results demonstrate _Kdm5c_-KO EpiLCs express key drivers of germline identity and meiosis while the _Kdm5c_-KO brain primarily expresses spermiogenesis genes. 

## Female epiblast-like cells have increased sensitivity to germline gene misexpression with _Kdm5c_ loss
```{r, DESeq2_params_XX, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the parameters used repeatedly, like the log2fold change cut off
source("../../KDM5C_Germ_Dev_Sex_RA/code/utilities/parameters.R")
```

It is currently unknown if the misexpression of germline genes is influenced by sex, as previous studies on germline gene repressors have focused on males[@pohlersRoleE2F6Restriction2005; @velascoDnmt3bRecruitmentE2F62010; @endohPCGF6PRC1SuppressesPremature2017; @dahletE2F6InitiatesStable2021; @mochizukiRepressionGermlineGenes2021]. Sex is particularly pertinent in the case of KDM5C because it lies on the X chromosome and partially escapes X chromsome inactivation (XCI), resulting in a higher dosage in females[@agulnikNovelGeneWidely1994; @carrelTissueLineagespecificVariation1996; @sheardownMouseSmcxGene1996; @xuSexSpecificExpressionXLinked2008]. We therefore explored the impact of chromosomal sex upon germline gene misexpession by comparing their dysregulation in male _Kdm5c_ hemizygous knockout (XY _Kdm5c_-KO), female homozygous knockout (XX _Kdm5c_-KO), and female heterozygous knockout (XX _Kdm5c_-HET) EpiLCs. [@samantaActivationXistEvolutionarily2022].

Homozygous and heterozygous _Kdm5c_ knockout females expressed over double the number of germline-enriched genes than hemizygous males (Figure 4A). While the majority of germline DEGs in _Kdm5c_-KO males were also dysregulated in females (74%), there were also many male-specific and female-specific germline DEGs, such as _Tktl2_ and _Esx1_ (Figure 4B). We compared the known functions of germline genes dysregulated only in females (XX only - dysregulated in XX _Kdm5c_-KO, XX _Kdm5c_-HET, or both), only in males (XY only), or in all samples (shared) (Figure 4C). Female-specific germline DEGs were enriched for meiotic (GO:0051321 meiotic cell cycle) and flagellar (GO:0003341 cilium movement) functions, while male-specifc DEGs had roles in mitochondrial and cell signaling (GO:0070585 protein localization to mitochondrion). Germline transcripts expressed in both sexes were enriched for meiotic (GO:0140013 meiotic nuclear division) and egg-specific functions (GO:0007292 female gamete generation). 

The majority of germline genes expressed in both sexes had a greater log2 fold change in females compared to males (Figure 4D). This increased degree of dysregulation in females, along with the increased total number of germline genes, indicates females are more sensitive to losing KDM5C-mediated germline gene suppression. Female sensitivity could be due to impaired XCI in _Kdm5c_ mutants[@samantaActivationXistEvolutionarily2022], as many spermatogenesis genes lie on the X chromosome[@wangAbundanceXlinkedGenes2001; @khilMouseChromosomeEnriched2004]. However, female germline DEGs were not biased towards the X chromosome and had a a similar overall proportion of X chromosome DEGs compared to males (XY _Kdm5c_-KO - 10.29%, XX _Kdm5c_-HET - 7.43%, XX _Kdm5c_-KO - 10.59%) (Figure 4G). The majority of germline DEGs instead lie on autosomes for both male and female _Kdm5c_ mutants (Figure 4G). Thus, while female EpiLCs are more prone to germline gene misexpression with KDM5C loss, it is likely independent of XCI defects.


### Germline gene misexpression in _Kdm5c_ mutants is independent of germ cell sex

While many germline genes act in both the male and female germline, some display sex-biased expression or have functions unique to eggs and sperm. Therefore, we wondered if _Kdm5c_ mutant males were more likely to express sperm genes and if mutant females would instead express egg genes. To comprehensively assess whether germline gene sex corresponds with _Kdm5c_ mutant sex, we filtered our list of germline-enriched genes for egg and sperm-biased genes. We defined germ cell sex-biased genes as those whose expression in the opposite sex, at any time point, is no greater than 20% of the gene's maximum expression in a given sex. This yielded `r nrow(subset(germgenes, sexBias == "XX"))` egg-biased, `r nrow(subset(germgenes, sexBias == "XY"))` sperm-biased, and `r nrow(subset(germgenes, sexBias == "unbiased"))` unbiased germline-enriched genes (Figure 4H). We found egg, sperm, and unbiased germline genes were dyregulated in all _Kdm5c_ mutants (Figure 4I-J). Germline genes dysregulated exclusively in _Kdm5c_ mutant males and females were also not biased towards their corresponding germ cell sex (Figure 4I). This indicates sex differences in germline gene dysregulation is not due to sex-specific activation of sperm or egg transcriptional programs. Altogether, these results demonstrate that both male and female _Kdm5c_ mutant cells express both sperm and egg-biased genes.

## KDM5C binds to a subset of germline gene promoters during early embryogenesis 

KDM5C binds to the promoters of several germline genes in embryonic stem cells (ESCs) but its binding is absent in neurons[@scandagliaLossKdm5cCauses2017]. However, the lack of a comprehensive list of germline-enriched genes prohibited comprehensive characterization of KDM5C binding at germline gene promoters. Thus, it is unclear if KDM5C is enriched at germline gene promoters, what types of germline genes KDM5C regulates, and if its binding is maintained at any germline genes in neurons. 

```{r, KDM5C_ChIPseq, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
KDM5C_ChIP_EpiLC_promo <- read.csv("../data/processed/KDM5C_ChIPseq_boundpromoters_EpiLC.csv")

KDM5C_ChIP_PNC_promo <- read.csv("../data/processed/KDM5C_ChIPseq_boundpromoters_PNC.csv")

KDM5C_ChIP_EpiLC <- read.csv("../data/raw/ChIPseq_mm10_EpiLC_WTnoKO_consensus_peaks.bed", sep = "\t")

KDM5C_ChIP_PNC <- read.csv("../data/raw/ChIPseq_mm10_PNC_WTnoKO_consensus_peaks.bed", sep = "\t")

GO_KDM5C_ChIP <- read.csv("../results/ChIPseq_KDM5C_GO_EpiLC_vs_PNC_unique.csv")

```

To address these questions, we analyzed KDM5C chromatin immunoprecipitation followed by DNA sequencing (ChIP-seq) datasets in EpiLCs[@samantaActivationXistEvolutionarily2022] and primary forebrain neuron cultures (PNCs)[@iwaseMouseModelXlinked2016]. EpiLCs had a higher total number of KDM5C peaks than PNCs (EpiLCs: `r nrow(KDM5C_ChIP_EpiLC)`, PNCs: `r nrow(KDM5C_ChIP_PNC)`, MACS2 q < 0.1 and fold enrichment > 1, removal of _Kdm5c_-KO false positive peaks) and KDM5C was primarily localized to gene promoters in both cell types (EpiLCs: `r nrow(KDM5C_ChIP_EpiLC_promo)`, PNCs: `r nrow(KDM5C_ChIP_PNC_promo)` &#177; `r promorange`bp from the TSS), although PNCs showed increased localization to non-promoter regions (Figure 5A). 

The majority of promoters bound by KDM5C in PNCs were also bound in EpiLCs (513 shared promoters), however a large portion of gene promoters were only bound by KDM5C in EpiLCs (3677 EpiLC only promoters) (Figure 5B). Genes bound by KDM5C in both PNCs and EpiLCs were enriched for functions involving nucleic acid turnover, such as deoxyribonucleotide metabolic process (GO:0009262, p.adjust = `r signif(getGO("deoxyribonucleotide metabolic process", GO_KDM5C_ChIP, 'p.adjust', "Shared"), 3)`) (Figure 5C). Germline-specific ontologies were only enriched in KDM5C-bound promoters unique to EpiLCs, such as meiotic nuclear division (GO: 0007127 p.adjust = `r signif(getGO("meiosis I", GO_KDM5C_ChIP, 'p.adjust', "EpiLC only"), 3)`) and meiotic cell cycle process (GO:1903046, p.adjust = `r signif(getGO("meiotic cell cycle process", GO_KDM5C_ChIP, 'p.adjust', "EpiLC only"), 3)`) (Figure 5C). There were no ontologies significantly enriched for genes bound by KDM5C only in PNCs. We next evaluated KDM5C binding around the transcription start site (TSS) of all germline-enriched genes. KDM5C binding around the TSS of many germline genes is evident in EpiLCs, but absent in PNCs (Figure 5D). Based on our ChIP-seq peak cut-off criteria, KDM5C was strongly bound to about 10% of germline gene promoters in EpiLCs (Figure 5E). 

 In EpiLCs, KDM5C was only bound to about one third of _Kdm5c_-KO RNA-seq DEG promoters (EpiLC only DEGs: `r 18/50*100`%, Brain only DEGs: `r signif(7/21*100,3)`%). However, KDM5C was bound to the promoter at 3 out of the 4 genes dysregulated in both the brain and EpiLCs (Supplementary figure XXX). Representative examples of KDM5C-bound and unbound germline DEGs are _Dazl_ and _Stra8_, repsectively (Figure 5F). Together, these results demonstrate KDM5C is recruited to a subset of germline genes in EpiLCs, including enrichment of meiotic genes, but does not directly regulate germline genes in neurons. Furthermore, the majority of germline mRNAs expressed in _Kdm5c_-KO cells are dysregulated independent of direct KDM5C binding to their promoters. 

 One notable gene that lacked KDM5C binding was _Stra8_, even though its mRNA is expressed in _Kdm5c_-KO EpiLCs ().

Many germline-specific genes are suppressed by the transcription factor heterodimers E2F6/DP1 and MGA/MAX, which respectively bind E2F and E-box motifs[@hurlinMgaDualspecificityTranscription1999; @pohlersRoleE2F6Restriction2005; @velascoDnmt3bRecruitmentE2F62010; @stielowMGAL3MBTL2E2F62018; @dahletE2F6InitiatesStable2021]. To elucidate if KDM5C is recruited to germline gene promoters by a similar mechanism, we used HOMER to identify transcription factor motifs enriched at KDM5C-bound or unbound germline gene promoters[@heinzSimpleCombinationsLineageDetermining2010] (TSS &#177; 500 bp, q-value < 0.1). MAX and E2F6 binding sites were significantly enriched at germline genes bound by KDM5C in EpiLCs, but not at germline genes unbound by KDM5C (MAX q-value: 0.0068, E2F6 q-value: 0.0673, E2F q-value: 0.0917) (Figure 5G). One third of KDM5C-bound promoters contained the consensus sequence for either E2F6 (E2F, 5'-TCCCGC-3'), MGA (E-box, 5'-CACGTG-3'), or both, but only 17% of KDM5C-unbound genes contained these motifs (Figure 5H). KDM5C-unbound germline genes were intstead enriched for multiple RFX transcription factor binding sites (RFX q-value < 0.0001, RFX2 q-value < 0.0001, RFX5 q-value < 0.0001) (Figure 5I). RFX transcription factors bind X-box motifs[@gajiwalaStructureWingedhelixProtein2000] to promote ciliogenesis[@swobodaRFXTypeTranscriptionFactor2000; @ashiqueRfx4TranscriptionFactor2009] and among them is RFX2, a central regulator of post-meiotic spermatogeneis[@kistlerRFX2MajorTranscriptional2015; @wuTranscriptionFactorRFX22016]. Interestingly, RFX2 mRNA is derepressed in _Kdm5c_-KO EpiLCs (Figure 5J), however it is also not a direct target of KDM5C (Supplementary figure XXX). Thus, RFX2 is a candidate transcription factor for driving the ectopic expression of KDM5C-unbound germline genes in _Kdm5c_-KO cells.


## KDM5C is recruited to germline gene promoters harboring CpG islands to facilitate _de novo_ DNA methylation

In the early embryo, germline gene promoters are initially decorated with repressive histone modifications and are then silenced long-term via DNA CpG methylation (CpGme)[@borgelTargetsDynamicsPromoter2010; @endohPCGF6PRC1SuppressesPremature2017; @liuPolycombGroupProtein2020; @mochizukiRepressionGermlineGenes2021]. Our results above indicate KDM5C also acts at germline gene promoters during this time period. However, how KDM5C interacts with other germline gene silencing mechanisms is currently unclear. KDM5C is generally thought to supress transcription through erasure of histone 3 lysine 4 di- and trimethylation (H3K4me2/3)[@iwaseXLinkedMentalRetardation2007], yet KDM5C's catalytic activity was recently shown to be dispensible for suppressing _Dazl_ in undifferentiated ESCs[@guptaGenomewideScreenReveals2023]. Since H3K4me3 impedes _de novo_ CpGme placement[@otaniStructuralBasisRecognition2009; @guoStructuralInsightAutoinhibition2015], KDM5C's catalytic activity may instead be required in the post-implantation embryo for long-term silencing of germline genes. In support of this, CpGme is markedly reduced at two germline gene promoters in the _Kdm5c_-KO adult hippocampus[@scandagliaLossKdm5cCauses2017]. 

Based on the above observations, we hypothesized KDM5C erases H3K4me3 to promote the initial placement of CpGme at germline gene promoters in EpiLCs. To test this hypothesis, we first characterized KDM5C's substrates, histone 3 lysine 4 di- and trimethylation (H3K4me2/3), at germline gene promoters in our previously published ChIP-seq datasets in male wild type and _Kdm5c_-KO amygdala[@vallianatosMutuallySuppressiveRoles2020] and EpiLCs[@samantaActivationXistEvolutionarily2022]. In congruence with previous work in the _Kdm5c_-KO hippocampus[@scandagliaLossKdm5cCauses2017], we observed aberrant accumulation of H3K4me3 around the transcription start site (TSS) of germline genes in the _Kdm5c_-KO amygdala (Figure 6A). We additionally found a marked increase in H3K4me2 around the TSS of germline genes in _Kdm5c_-KO EpiLCs (Figure 6B). To elucidate KDM5C's embyonic role, we then characterized KDM5C's mRNA and protein expression during male ESC to EpiLC differentiation (Figure 6C). While _Kdm5c_ mRNA steadily decreased from 0 to 48 hours of diferentiation (Figure 6D), KDM5C protein initially increased from 0 to 24 hours but then decreased to near knockout levels by 48 hours (Figure 6E). Together, these data suggest KDM5C acts during the transition between ESCs and EpiLCs to remove H3K4me at germline gene promoters.


```{r, CGIs, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
germgenes_CGI <- read.csv("../results/KDM5C_binding_allgerm_CGI.csv", header = TRUE)

CGI <- germgenes_CGI[germgenes_CGI$Promo_CGI == "CGI",]

ESCvsEpiLC <- read.csv("../results/WGBS/WGBS_restab_annotated_allgermTSS_500bp_WT_ESCvsEpiLC.csv")

ESCvsEpiLC_25 <- ESCvsEpiLC[ESCvsEpiLC$meth.diff >= 25, ]

WTvsKO <- read.csv("../results/WGBS/WGBS_restab_annotated_allgermTSS_500bp_WTvsKO.csv")

WTvsKO_hypo <- WTvsKO[WTvsKO$meth.diff <= -25, ]
WTvsKO_hypo_5C <- WTvsKO_hypo[WTvsKO_hypo$KDM5C_binding == "Bound", ]


#get the infomration for wgbs
getWGBSinfo <- function(df, gene, column){
	df2 <- df
	info <- df2[df2$SYMBOL == gene,][[column]]
	return(info)
}

```

In wild-type cells, germline genes accumulate CpG methylation (CpGme) at CpG islands (CGIs) during the transition from na&#239;ve to primed pluripotency[@meissnerGenomescaleDNAMethylation2008; @borgelTargetsDynamicsPromoter2010; @hackettPromoterDNAMethylation2012], reaching peak methylation levels when differentiated into EpiLCs for 96 hours (extended EpiLCs, exEpiLCs)[@mochizukiRepressionGermlineGenes2021]. We first identified how many germline genes contained CGIs using the UCSC genome browser[@nassarUCSCGenomeBrowser2023] and found out of `r nrow(germgenes)` germline-enriched genes, only `r nrow(CGI)` (`r signif(nrow(CGI)/nrow(germgenes)*100, 4)`%) contained CGIs within their promoters (TSS &#177; 500 bp) (Figure 6F). CGI-containing germline genes were enriched for meiotic gene ontologies, including meiotic nuclear division (GO:XXXX, p.adj) and meiosis I (GO:XXXX, p.adj) when compared to CGI-free genes (Figure 6G). Although a minor portion of germline gene promoters contained CGIs, CGIs strongly determined KDM5C's recruitment to germline gene promoters (FISHER'S XXXX), with 79% of KDM5C-bound germline genes containing CGIs (Figure 6G). 

We then perfomed whole genome bislufite sequencing (WGBS) in male wild-type and _Kdm5c_-KO ESCs and 96 hour extended EpiLCs (exEpiLCs) to assess how KDM5C loss impacted initial CpGme placement at germline gene promoters (Figure 6H). We first identified which germline gene promoters siginificantly gained CpGme in wild-type cells during ESC to exEpiLCs differentiation (methylKit[@akalinMethylKitComprehensivePackage2012], q < 0.01, |methylation difference| >= 25%, TSS &#177; 500 bp). In wild-type cells, the majority of germline genes gained substantial CpGme at their promoter during differentiation (`r signif(nrow(ESCvsEpiLC_25)/nrow(ESCvsEpiLC)*100, 4)`%), regardless if their promoter contained a CGI (Figure 6I).

We then identified germline gene promoters differentially methylated in wild-type versus _Kdm5c_-KO exEpiLCs (methylKit, q < 0.01, |methylation difference| >= 25%, TSS &#177; 500 bp) and found `r nrow(WTvsKO_hypo)` germline promoters were significantly hypomethylated with _Kdm5c_ loss (Figure 6J). Approximately half of germline promoters hypomethylated in _Kdm5c_-KO exEpiLCs are direct targets of KDM5C in EpiLCs (`r nrow(WTvsKO_hypo_5C)` out of `r nrow(WTvsKO_hypo)` hypomethylated DMRs). We then evaluated promoter CpGme at germline genes ectopically transcibed in either _Kdm5c_-KO EpiLCs or within the brain and found promoter CpGme was substantially reduced in about half of germline DEGs (Figure 6K). Significantly hypomethylated promoters included genes consistently dysregulated across multiple _Kdm5c_-KO RNA-seq datasets, such as _D1Pas1_ (methlyation difference = `r signif(getWGBSinfo(WTvsKO_hypo, "D1Pas1", "meth.diff"),4)`%, q-value = `r signif(getWGBSinfo(WTvsKO_hypo, "D1Pas1", "qvalue"),3)`) (Figure 6L). Suprisingly, we only observed a modest reduction in CpGme at _Dazl_'s promoter (methlyation difference = `r signif(getWGBSinfo(WTvsKO, "Dazl", "meth.diff"),4)`%, q-value = `r signif(getWGBSinfo(WTvsKO, "Dazl", "qvalue"),3)`) (Figure 6M). Altogether, these results demonstrate KDM5C is recruited to germline gene CGIs to promote CpGme at select germline gene promoters during early embryogenesis, however other germline gene silencing mechansism can sufficiently compensate for KDM5C's loss at select germline gene promoters. 


# Discussion
In the above study, we demonstrate KDM5C's pivotal role in the development of tissue identity.
We first characterized the misexpression of tissue-enriched genes within the _Kdm5c_-KO brain and identified substantial dysregulation of testis, liver, muscle, and ovary-enriched genes. Testis genes significantly enriched within the _Kdm5c_-KO amygdala and hippocampus are specific to germ cells and not expressed within testis somatic cells. _Kdm5c_-KO epiblast-like cells (EpiLCs) aberrantly express key drivers of germline identity and meiosis, including _Dazl_ and _Stra8_, while the _Kdm5c_-KO brain primarily expresses genes important for late spermatogenesis. We demonstrated that while _Kdm5c_ mutant sex did not influence whether sperm or egg-specific genes were misexpressed, female EpiLCs are more sensitive to germline gene de-repression. 
Germline-enriched mRNAs can become aberrantly expressed in _Kdm5c_-KO cells indirect of KDM5C, as although KDM5C is enriched at germline gene promoters in EpiLCs, it is only bound to a subset of germline-enriched DEGs. Finally, we found KDM5C is dynamically regulated during ESC to EpiLC differentiation and promotes long-term germline gene silencing through DNA methylation at CpG islands. Therefore, we propose KDM5C plays a fundamental role in the development of tissue identity during early embryogenesis, including the establishment of the soma-germline boundary. By systematically characterizing KDM5C's role in germline gene repression, including its interaction with known silencing mechanisms, we unveiled unique repressive mechansisms governing distinct classes of germline gene in somatic lineages. Furthermore, these data provide molecular footholds that can then be exploited to test the ultimate contribution of ectopic germline gene expression upon neurodevelopment.

It is important to note that while we highlighted KDM5C's regulation of germline genes, some germline-enriched genes are also expressed at the 2-cell stage and in na&#239;ve ESCs for their role in pluripotency and self-renewal. For example, _Dazl_ is primarily known for committing PGCs to the germline fate and regulating the translation of germline-specific mRNAs, but is also expressed at the 2-cell stage[@yangChemicalinducedChromatinRemodeling2022], in na&#239;ve ESCs[@wellingDAZLRegulatesTet12015], and in the inner cell mass [@wellingDAZLRegulatesTet12015]. KDM5C may therefore negatively regulate totipotency, as KDM5C suppresses _Dazl_ in ESCs[@guptaGenomewideScreenReveals2023; @aladhamiSystematicIdentificationFactors2023] and _Kdm5c_-KO ESCs aberrantly express 2-cell-specific genes like _Zscan4c_[@guptaGenomewideScreenReveals2023]. However, misexpression of _Dazl_ in _Kdm5c_-KO ESCs was independent of the 2-cell specific transcription factor _Dux_[@guptaGenomewideScreenReveals2023] and, unlike _Dazl_, KDM5C does not bind to the _Dux_ promoter. 

Although expressed in na&#239;ve ESCs, _Dazl_ and other "self-renewal" germline genes are silenced during differentiation into epiblast stem cells/EpiLCs[@borgelTargetsDynamicsPromoter2010; @mochizukiRepressionGermlineGenes2021]. We found that while _Kdm5c_-KO EpiLCs also expressed _Dazl_, they did not express 2-cell specific genes. Misexpression of germline genes in _Kdm5c_-KO EpiLCs could indicate they are differentiating into primordial germ cell-like cells (PGCLCs)[@magnusdottirHowMakePrimordial2014; @gunesdoganPrimordialGermCell2014; @hayashiReconstitutionMouseGerm2011]. Yet, _Kdm5c_-KO EpiLCs had normal cellular morphology and properly expressed markers for primed pluripotency, including _Otx2_ which is known to repress EpiLC differentiation into PGCs/PGCLCs[@zhangOTX2RestrictsEntry2018]. Altogether, these data suggest _Kdm5c_-KO germline gene misexpression occurs ectopically in conjunction with typical developmental programs and the 2-cell-like state observed in _Kdm5c_-KO ESCs is an indirect consequence of KDM5C's primary role in germline gene repression.

Although eggs and sperm employ the same transcriptional programs for shared functions, e.g. PGC formation,  meiosis, and genome defense, some germline genes are sex specific. We found _Kdm5c_ mutant males and females expressed both sperm and egg-biased genes, indicating the mechanism of derepression is independent of a given germline gene's sex. However, organismal sex did greatly influence the degree of germline gene dysregulation, as female _Kdm5c_-KO EpiLCs had over double the number of germline-enriched DEGs compared to males. X chromosome inactivation (XCI) defects could explain why _Kdm5c_ knockout females are more prone to germline gene dysregulation, given the X chromosome harbors a large number of spermatogenesis genes[@wangAbundanceXlinkedGenes2001; @khilMouseChromosomeEnriched2004] and loss of _Kdm5c_ impairs XCI[@samantaActivationXistEvolutionarily2022]. However, female germline DEGs were not significantly biased towards the X chromosome. Sex differences in germline gene suppression may be instead connected to females having a higher dose of KDM5C than males, due to its partial escape from XCI[@agulnikNovelGeneWidely1994; @carrelTissueLineagespecificVariation1996; @sheardownMouseSmcxGene1996; @xuSexSpecificExpressionXLinked2008]. Intriguingly, females with heterozygous loss of _Kdm5c_ also had over double the number of germline DEGs than hemizygous knockout males, even though their level of KDM5C should be roughly equivalent to that of wild-type males. Altogether, these results suggest female EpiLCs are more prone to transitioning to a germ cell-like state than males and require more KDM5C to maintain somatic cellular identity. Future studies are required to illuminate if this phenomenon is unique to sexually dimorphic chromatin regulators or a general feature of female cells.

It is yet unknown how KDM5C is recruited to germline gene promoters, given that KDM5C itself does not contain domains for sequence-specific binding[@iwaseXLinkedMentalRetardation2007]. In HeLa cells and ESCs[@tahilianiHistoneH3K4Demethylase2007; @guptaGenomewideScreenReveals2023], KDM5C assocaiates with members of the polycomb repressive complex 1.6 (PRC1.6), which is recruited to germline gene promoters through E2F6/DP1 and MGA/MAX heterodimers[@endohPCGF6PRC1SuppressesPremature2017; @stielowMGAL3MBTL2E2F62018]. While MAX and E2F6 motifs were enriched at KDM5C-bound germline genes in EpiLCs, only about one-third of promoters contained their consesus sequence. Additionally, some germline genes can become active in _Kdm5c_-KO cells independent of KDM5C binding to their promoters. Germline genes unbound by KDM5C were primarily late-stage spermatogenesis genes and significantly enriched for RFX transcription factors, including RFX2, a central regulator of spermiogenesis[@kistlerRFX2MajorTranscriptional2015; @wuTranscriptionFactorRFX22016]. Another notable KDM5C-unbound DEG is _Stra8_, a meiotic transcription factor that is turned on in germ cells with retionic acid signaling and DAZL expression[@endoPeriodicRetinoicAcid2015; @bowlesRetinoidSignalingDetermines2006]. Misexpression of _Dazl_ and _Rfx2_ and their downstream targets in _Kdm5c_-KO cells suggests that once activated, ectopic germline genes can then turn on other aberrant germline programs to loosely mimic germ cell development. 

Unlike the previously characterized germline gene suppressors that place repressive histone modifications[@karimiDNAMethylationSETDB12011; @hackettPromoterDNAMethylation2012; @auclairEHMT2DirectsDNA2016; @endohPCGF6PRC1SuppressesPremature2017; @tatsumiDNMTsSETDB1Function2018; @liuPolycombGroupProtein2020; @mochizukiRepressionGermlineGenes2021], KDM5C removes the active histone mark histone 3 lysine 4 di- and trimethylation (H3K4me2/3)[@iwaseXLinkedMentalRetardation2007]. Yet, KDM5C's catalytic activity may not be required for germline gene silencing, as it was recently found to be dispensible for repressing _Dazl_ in ESCs [@guptaGenomewideScreenReveals2023]. Emerging work indicates many histone-modifying enzymes have non-cataltyic functions that can influnce gene expression, sometimes even more potently than their catalytic roles[@aubertUnexpectedNoncatalyticRoles2019; @morganReevaluatingRolesHistonemodifying2020]. Although dispensible in ESCs, KDM5C's catalytic activity could be necessary to silence germline genes in EpiLCs, as H3K4me3 impedes _de novo_ CpGme placement[@otaniStructuralBasisRecognition2009; @guoStructuralInsightAutoinhibition2015]. In support of this, CpGme is significantly eroded at at least two germline promoters in the adult _Kdm5c_-KO hippocampus[@scandagliaLossKdm5cCauses2017]. We found _Kdm5c_-KO cells accumulate H3K4me2/3 around the TSS of germline genes and have impaired CpGme placement at CGIs for select germline gene promoters. KDM5C's shifting role in germline gene repression during the transition from na&#239;ve to primed pluripotency indicates the requirement of catalytic and non-catlytic gene regulatory mechanisms can change depending upon the locus and developmental stage. 

- DNAme and CpG islands
	- Loss of many chromatin regulators affects CpGme (Multiple, non-redundant silencing mechansisms) - seems to be the crux point of germline gene silencing
	- Germline genes are methylated at promoter CGIs, which are typically not methylated for other types of genes
	- Because CGIs are typically resistant to CpGme (accurate?), germline CGIs may require a highly repressive histone landscape to recruit sufficient DNMTs to these loci
	- We found only 30% of germline-enriched genes have CGIs, but the majority of CGI-free germline genes still gain CpGme around the TSS. Unclear what the repressive mechanism is for these genes.
	- CGIs highly determine KDM5C recruitment to germline gene promoters.
			- KDM5C is known to be enriched at CGIs (in neurons? are these methylated? or is its germline CGI function different from its somatic CGI function?). 
	- KDM5C loss impacts germline gene CGI methylation, but not really the non-CGI TSS methylation
		- Other studies on germline gene repressors have shown they are important for CGIme, unclear if they participate in non-CGI TSS CpGme
	- Altogether demonstrates different classes of germline genes use distinct silencing mechanisms 
	

The above work provides the mechanistic foundation for KDM5C's repression of germline-specific transcription and offers novel insight into how dysregulation of _Kdm5c_-KO tissue identity changes over the course of development. However, the contribution of these ectopic, tissue-specific genes towards _Kdm5c_-KO neurological impairments is still unknown. In addition to germline genes, we also identified significant enrichment of muscle, liver, and even ovary-biased transcripts within the male _Kdm5c_-KO brain. Intriguingly, select liver and muscle-biased DEGs do have known roles within the brain, such as the liver-enriched lipid metabolism gene _Apolipoprotein C-I (Apoc1)_ [@roulandRoleApolipoproteinC12022]. _APOC1_ dysregulation is implicated in Alzheimer's disease in humans[@leducAPOECholesterolHomeostasis2010] and overexpression of _Apoc1_ in the mouse brain can impair learning and memory[@abildayevaHumanApolipoproteinCI2008]. KDM5C may therefore be crucial for neurodevelopment by fine-tuning the expression of tissue-erniched, dosage-sensitive genes like _Apoc1_. Given germline genes have no known functions within the brain, their impact upon neruodevelopment is currently unknown. Ectopic germline transcripts have been observed in a variety of cancers[@nielsenEctopicExpressionTestis2016; @adebayobabatundeCancerTestisGenes2017] and can drive brain turmor formation in _Drosophila_[@janicEctopicExpressionGermline2010], indicating their dysregulation may promote genome instability and cellular de-differentiation. Intriguingly, some models for other chromatin-linked neurodevelopmental disorders also display impaired soma-germline demarcation[@bonefasSomatogermlineTransformationChromatinlinked2021; @velascoGermlineGenesHypomethylation2014; @waltonDnmt3bPrefersGerm2014; @schaeferControlCognitionAdaptive2009; @samacoCrhOprm1Mediate2012]. Like KDM5C, the chromatin regulators underlying these conditions - DNA methyltransferase 3b (DNMT3B), H3K9me1/2 methyltransferases G9A/GLP, methyl-CpG -binding protein 2 (MECP2) - primarily silence gene expression. Thus, KDM5C is among a growing cohort of chromatin-linked neurodevelopmental disorders that have a similar underlying phenotype of germline versus soma dysregulation However, further research is required to determine the impact of these germline genes and the extent to which this phenomenon occurs in humans. 


# Materials and Methods

## Classifying tissue-enriched and germline-enriched genes
Tissue-enriched differentially expressd genes (DEGs) were determined by their classification in a previously published dataset from 17 male and female mouse tissues[@liComprehensiveMouseTranscriptomic2017]. This study defined tissue expression as greather than 1 Fragments Per Kilobase of transcript per Million mapped read (FPKM) and tissue enrichment as at least 4-fold higher expression than any other tissue.

We curated a list of germline-enriched genes using an RNA-seq dataset from wild-type and germline-depleted (Kit^W/Wv^) male and female mouse embryos from embryonic day 12, 14, and 16 [@sohGeneRegulatoryProgram2015], as well as adult male testes[@muellerIndependentSpecializationHuman2013]. Germline-enriched genes met the following criteria: 1) their expression is greater than 1 FPKM in wild-type germline 2) their expression in any wild-type somatic tissues[@liComprehensiveMouseTranscriptomic2017] does not exceed 20% of maximum expression in wild-type germline, and 3) their expression in the germ cell-depleted (Kit^W/Wv^) germline, for any sex or time point, does not exceed 20% of maximum expression in wild-type germline. 


## Cell culture
We utilized our previously established cultures of male wild-type and _Kdm5c_ knockout (-KO) embryonic stem cells[@samantaActivationXistEvolutionarily2022]. Sex was confirmed by genotyping _Uba1_/_Uba1y_ on the X and Y chromosomes with the following primers: 5'-TGGATGGTGTGGCCAATG-3', 5'-CACCTGCACGTTGCCCTT-3'. Deletion of _Kdm5c_ was confirmed through the primers 5'-ATGCCCATATTAAGAGTCCCTG-3', 5'-TCTGCCTTGATGGGACTGTT-3', and 5'-GGTTCTCAACACTCACATAGTG-3'.

Emrbyonic stem cells (ESCs) and epiblast-like cells were cultured using previously established methods[@samantaGeneratingPrimedPluripotent2020]. Briefly, ESCs were initially cultured in primed ESC (pESC) media consisting of KnockOut DMEM (Gibco#10829018), fetal bovine serum (Gibco#A5209501), KnockOut serum replacement (Invitrogen#10828028), Glutamax (Gibco#35050-061), Anti-Anti (Gibco#15240-062), MEM Non-essential amino acids (Gibco#11140-050), and beta-mercaptoethanol (Sigma#M7522). They were then transitioned into ground-state, "na&#239;ve" ESCs (nESCs) by culturing for four passages in N2B27 media containing DMEM/F12 (Gibco#11330032), Neurobasal media (Gibco#21103049), Gluamax, Anti-Anti, N2 supplement (Invitrogen#17502048), and B27 supplement without vitamin A (Invitrogen#12587-010), and beta-mercaptoethanol. Both pESC and nESC media were supplemented with 3 $\mu$M GSK3 inhibitor CHIR99021 (Sigma #SML1046-5MG), 1 $\mu$M MEK inhibitor PD0325901 (Sigma #PZ0162-5MG), and 1,000 units/mL leukemia inhibitory factor (LIF, Millipore#ESG1107).

nESCs were differentiated into epiblast-like cells (EpiLCs, 48 hours) and extendend EpiLCs (exEpiLCs, 96 hours) by culturing in N2B27 media containing DMEM/F12, Neurobasal media, Gluamax, Anti-Anti, N2 supplement, B27 supplement (Invitrogen#17504044), beta-mercaptoethanol, fibroblast growth factor 2 (FGF2, R&D Biotechne 233-FB), and activin A (R&D Biotechne 338AC050CF), as previously described [@samantaGeneratingPrimedPluripotent2020]. 

## Immunocytochemistry (ICC)
ICC of DAZL in EpiLCs was performed by first growing cells on fibronectin-coated coverslips. Cells were then washed thrice with phosphobuffered saline (PBS), fixed in 4% paraformaldehyde, washed thrice in PBS, and blacked in PBS containing 0.3% Triton X-100, and 5% fetal bovine serum for 1 hour. They were then washed thrice with PBS and incubating in primary antibody (Rabbit anti DAZL, abcam ab34139, 1:200) in the blocking solution overnight at 4C with gentle rocking. The next day, cells were rinsed thrice with PBS, and incubated in secondary antibody (Alexafluor 488 Invitrogen #710369, 1:1,000) with DAPI (1:1,000) in blocking buffer for 1 hour at room temperature. Coverslips were then rinsed thrice in PBS, and mounted onto slides using Prolong Gold (Invitrogen #P36930). Images were taken blinded for genotype, chosen based on similar levels of DAPI signal, and quantified via ImageJ before unblinding. 

## RNA sequencing (RNA-seq)
After ensuring read quality via FastQC (v0.11.8), reads were then mapped to the mm10 _Mus musculus_ genome (Gencode) using STAR (v2.5.3a), during which we removed duplicates and kept only uniquely mapped reads. Count files were generated by FeatureCounts (Subread v1.5.0), and BAM files were converted to bigwigs using deeptools (v3.1.3) and visualized by the UCSC genome browser. RStudio (v3.6.0) was then used to analyze counts files by DESeq2 (v1.26.0)[@loveModeratedEstimationFold2014a] to identify differentially expressed genes (DEGs) with a q-value (p-adjusted via FDR/BenjaminiHochberg correction) less than 0.1 and a log2 fold change greater than 0.5. For all DESeq2 analyses, log2 fold changes were calculated with lfcShrink using the ashr package[@stephensFalseDiscoveryRates2016]. MA-plots were generated by ggpubr (v0.6.0), and Eulerr diagrams were generated by eulerr (v6.1.1). Boxplots and scatterplots were generated by ggpubr (v0.6.0) and ggplot2 (v3.3.2). The Upset plot was generated via the package UpSetR (v1.4.0)[@conwayUpSetRPackageVisualization2017]. Gene ontology (GO) analyses were performed by the R package enrichPlot (v1.16.2) using the biological processes setting and compareCluster.

## Chromatin immunoprecipitation followed by DNA sequecing (ChIP-seq)
ChIP-seq reads were aligned to mm10 using Bowtie1 (v1.1.2) allowing up to two mismatches. Only uniquely mapped reads were used for analysis. Peaks were called using MACS2 software (v2.2.9.1) using input BAM files for normalization, with filters for a q-value < 0.1 and a fold enrichment > 1. We removed black-listed genomic regions that often give aberrant signals. Common peak sets were obtained in R via DiffBind (v3.6.5). In the case of KDM5C ChIP-seq, _Kdm5c_-KO peaks were then subtracted from wild-type samples using bedtools (v2.25.0). Peak proximity to genome annotations was determined by ChIPSeeker (v1.32.1). Gene ontology (GO) analyses were performed by the R package enrichPlot (v1.16.2) using the biological processes setting and compareCluster. Enriched motifs were identified using HOMER[@heinzSimpleCombinationsLineageDetermining2010]. Average binding across the genome was visualized using deeptools (v3.1.3). Bigwigs were visualized using the UCSC genome browser.

## Whole genome bisulfite sequencing (WGBS)
Genomic DNA (gDNA) from na&#239;ve ESCs and extended EpiLCs was extracted using the Wizard Genomic DNA Purification Kit (Promega A1120), following the instructions for Tissue Culture Cells. gDNA from two wild-types and two _Kdm5c_-KOs of each cell type was sent to Novogene for WGBS using the Illumina NovaSeq X Plus platform and sequenced for 150bp paired-end reads (PE150). 
- bismark
- Methylkit

## Data availability
### Published datasets
All published datasets are available at the Gene Expression Omnibus (GEO) https://www.ncbi.nlm.nih.gov/geo. Published RNA-seq datasets analyzed in this study included the male wild-type and _Kdm5c_-KO adult amygdala and hippocampus[@vallianatosMutuallySuppressiveRoles2020] (available at GEO: GSE127722) and male wild-type and _Kdm5c_-KO EpiLCs[@samantaActivationXistEvolutionarily2022] (available at GEO: GSE96797). 

Previously published ChIP-seq experiments included KDM5C in wild-type and _Kdm5c_-KO EpiLCs[@samantaActivationXistEvolutionarily2022] (available at GEO: GSE96797) and mouse primary neuron cultures (PNCs) from the cortex and hippocampus[@iwaseMouseModelXlinked2016] (available at GEO: GSE61036). ChIP-seq of histone 3 lysine 4 dimethylation in male wild-type and _Kdm5c_-KO EpiLCs[@samantaActivationXistEvolutionarily2022] is also available at GEO: GSE96797. ChIP-seq of histone 3 lysine 4 trimethylation in wild-type and _Kdm5c_-KO male amygdala[@vallianatosMutuallySuppressiveRoles2020] are avilable at GEO: GSE127817.

### Data analysis
Scripts used to generate the results, tables, and figures of this study are available via a GitHub repository: XXX


# Acknowledgements
- Jacob Mueller for providing insight in germline gene regulation. 
- Sundeep Kalantry for providing reagents and expertise in culturing mouse emrbyonic stem cells and epiblast-like cells
- Ilakkiya
- Funding sources

# References
<div id="refs"></div>

\newpage
# Figures and Tables
- Supplementary table 1: list of all germline genes. 
	- Columns to include: 
		- KDM5C bound vs not
		- DEG in EpiLC, brain, both, neither (separate columns?)


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/Brain_tissue_genes.pdf}
  \caption[Tissue-enriched genes are misexpressed in the \textit{Kdm5c}-KO brain]{\textbf{Tissue-enriched genes are misexpressed in the \textit{Kdm5c}-KO brain.} \textbf{A-B.} Expression of tissue-enriched genes (Li et al 2017) in the male \textit{Kdm5c}-KO amygdala (A) and hippocampus (B). Left - MA plot of mRNA-sequencing. Right - Number of tissue-enriched differentially expressed genes (DEGs). * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test \textbf{C.} Left - Average bigwigs of an example aberrantly expressed testis-enriched DEG, \textit{FK506 binding protein 6 (Fkbp6)} in the wild-type (WT) and \textit{Kdm5c}-KO (5CKO) amygdala (red) and hippocampus (teal). Right - Expression of \textit{Cyct} in wild-type tissues from NCBI Gene, with testis highlighted in blue and brain tissues highlighted in red. \textbf{D.} Left - Average bigwigs of an example ovary-enriched DEG, \textit{Zygotic arrest 1 (Zar1)}. Right - Expression of \textit{Zar1} in wild-type tissues from NCBI Gene, with ovary highlighted in teal and brain tissues highlighted in red. \textbf{E.} Left - Average bigwigs of an example liver-enriched DEG, \textit{Apolipoprotein C-I (Apoc1)}. Right - Expression of \textit{Apoc1} in wild-type tissues from NCBI Gene, with liver highlighted in orange and brain tissues highlighted in red.}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/What_are_germline_genes.pdf}
  \caption[Aberrant transcription of germline genes in the \textit{Kdm5c}-KO in the brain]{\textbf{Aberrant transcription of germline genes in the \textit{Kdm5c}-KO in the brain.} \textbf{A.} enrichPlot gene ontology (GO) of \textit{Kdm5c}-KO amygdala and hippocampus testis-enriched DEGs \textbf{B.} Expression of testis DEGs in wild-type (WT) testis versus germ cell-depleted (W/Wv) testis (Mueller et al 2013). Expression is in Fragments Per Kilobase of transcript per Million mapped reads (FPKM). \textbf{C.} Number of testis DEGs that were classified as cell-type specific markers in a single cell RNA-seq dataset of the testis (Green et al 2018). Germline cell types are highlighted in green, somatic cell types in black. \textbf{D.} Sankey diagram of mouse genes filtered for germline enrichment based on their expression in wild-type and germline-depleted mice and in adult mouse non-gonadal tissues (Li et al 2017).}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/EpiLCs_too.pdf}
  \caption[\textit{Kdm5c}-KO epiblast-like cells express key drivers of germline identity ]{\textbf{\textit{Kdm5c}-KO epiblast-like cells express key drivers of germline identity A.} Top - Diagram of \textit{in vivo} differentiation of embryonic stem cells (ESCs) of the inner cell mass into epiblast stem cells. Bottom - \textit{in vitro} differentiation of ESCs into epiblast-like cells (EpiLCs). \textbf{B.} Number of tissue-enriched differentially expressed genes (DEGs) in \textit{Kdm5c}-KO EpiLCs. * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test. \textbf{C.}  Average bigwigs of an example germline gene, \textit{Cyct}, that is dysregulated \textit{Kdm5c}-KO EpiLCs. \textbf{D.} No significant difference in primed pluripotency maker expression in wild-type versus \textit{Kdm5c}-KO EpiLCs. Welch's t-test, expression in transcripts per million (TPM). \textbf{E.} Representative images of wild-type (WT) and \textit{Kdm5c}-KO cells during ESC to EpiLC differentiation. Brightfield images taken at 20X.\textbf{F.} Upset plot displaying the overlap of germline DEGs expressed in \textit{Kdm5c}-KO EpiLCs, amygdala (AMY), and hippocampus (HIP) RNA-seq datasets. \textbf{G.} enrichPlot comparing enriched biological process gene ontologies for \textit{Kdm5c}-KO EpiLC, amygdala, and hippocampus germline DEGs. \textbf{H.} Top left - Example germline identity DEGs unique to EpiLCs. Top right - Example 2-cell genes that are not dysregulated in \textit{Kdm5c}-KO EpiLCs. p-values for Welch's t-test. Bottom - Average bigwigs of \textit{Dazl} and \textit{Stra8} expression in wild-type and \textit{Kdm5c}-KO EpiLCs \textbf{I.} Immunocytochemistry of DAZL in male wild-type (WT) and \textit{Kdm5c}-KO (5CKO) EpiLCs. Percentage of DAZL-positive cells normalized to number of DAPI-positive cells, p-value for Welch's t-test.}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/EpiLC_XXvsXY.pdf}
  \caption[Chromosomal sex influences \textit{Kdm5c}-KO germline gene misexpression.]{\textbf{Chromosomal sex influences \textit{Kdm5c}-KO germline gene misexpression.} \textbf{A-B.} Expression of tissue-enriched genes (Li et al 2017) in the male \textit{Kdm5c}-KO amygdala (A) and hippocampus (B). Left - MA plot of mRNA-sequencing. Right - Number of tissue-enriched differentially expressed genes (DEGs). * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test \textbf{C.} Left - Average bigwigs of an example aberrantly expressed testis-enriched DEG, \textit{FK506 binding protein 6 (Fkbp6)} in the wild-type (WT) and \textit{Kdm5c}-KO (5CKO) amygdala (red) and hippocampus (teal). Right - Expression of \textit{Cyct} in wild-type tissues from NCBI Gene, with testis highlighted in blue and brain tissues highlighted in red. \textbf{D.} Left - Average bigwigs of an example ovary-enriched DEG, \textit{Zygotic arrest 1 (Zar1)}. Right - Expression of \textit{Zar1} in wild-type tissues from NCBI Gene, with ovary highlighted in teal and brain tissues highlighted in red. \textbf{E.} Left - Average bigwigs of an example liver-enriched DEG, \textit{Apolipoprotein C-I (Apoc1)}. Right - Expression of \textit{Apoc1} in wild-type tissues from NCBI Gene, with liver highlighted in orange and brain tissues highlighted in red.}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/KDM5C_ChIPseq.pdf}
  \caption[KDM5C binds to a subset of germline gene promoters during early embryogenesis]{\textbf{KDM5C binds to a subset of germline gene promoters during early embryogenesis A.} ChIPseeker localization of KDM5C peaks at different genomic regions in EpiLCs (top) and hippocampal and cortex primary neuron cultures (PNCs, bottom). \textbf{B.} Overlap of genes with KDM5C bound to their promoters in EpiLCs (purple) and PNCs (blue). \textbf{C.} Gene ontology (GO) comparision of genes with KDM5C bound to their promoter in EpiLCs and PNCs. Genes were classified as either bound in EpiLCs only (EpiLC only), unique to PNCs (PNC only, no significant ontologies) or bound in both PNCs and EpiLCs (shared). \textbf{D.} Average KDM5C binding around the transcription start site (TSS) of all germline-enriched genes in EpiLCs (left) and PNCs (right). \textbf{E.} KDM5C binding at the promoters of RNA-seq germline DEGs. Genes were classified as either only dysregulated in EpiLCs (EpiLC only), genes dyresgulated in the hippocampus or amygdala but not EpiLCs (brain only), or genes dysregulated in both EpiLCs and the brain (common). (Legend continued on next page.)}
  \label{figurelabel}
\end{figure}
\addtocounter{figure}{-1}
\begin{figure} [t!]
  \caption[]{\textbf{KDM5C binds to a subset of germline gene promoters during early embryogenesis.} (Legend continued.) \textbf{F.} Example KDM5C ChIP-seq bigwigs of DEGs unqiue to EpiLCs. Although both are expressed in \textit{Kdm5c}-KO EpiLCs, KDM5C is only bound to the \textit{Dazl} promoter and not the \textit{Stra8} promoter in EpiLCs. \textbf{G.} Example KDM5C ChIP-seq bigwigs of RNA-seq DEGs common between the brain and EpiLCs. KDM5C is bound to the \textit{D1Pas1} promoter but not the \textit{XXX} promoter in EpiLCs. \textbf{H.} Example KDM5C ChIP-seq bigwigs of RNA-seq DEGs unique to the brain. KDM5C is bound to the \textit{XXX} promoter but not the \textit{Meig1} promoter}
\end{figure}

\newpage
\begin{figure}
  \centering
  \includegraphics{../submission/compiled_figs/KDM5C_Mechanism.pdf}
  \caption[KDM5Cs catalytic activity promotes long-term silencing of germline genes via DNA methylation]{\textbf{KDM5Cs catalytic activity promotes long-term silencing of germline genes via DNA methylation.} \textbf{A.} Left - Bigwigs of representative histone 3 lysine 4 trimethylation (H3K4me3) ChIP-seq peaks at two germline genes in the wild-type (WT) and \textit{Kdm5c}-KO (5CKO) adult amygdala. Right - Average H3K4me3 at the transcripiton start site (TSS) of all germline-enriched genes in wild-type (dark red) and \textit{Kdm5c}-KO (light red) amygdala. \textbf{B.} Left - Bigwigs of representative histone 3 lysine 4 dimethylation (H3K4me2) ChIP-seq peaks at representative germline genes in wild-type and \textit{Kdm5c}-KO EpiLCs. Right - Average H3K4me2 at the TSS of all germline-enriched genes in wild-type (dark purple) and \textit{Kdm5c}-KO (light purple) EpiLCs. \textbf{C.} Diagram of embryonic stem cell (ESC) to epiblast-like cell (EpiLC) differentiation protocol and collection time points for RNA and protein. \textbf{D.} Real time quantitative PCR (RT-qPCR) of \textit{Kdm5c} RNA expression, calculated in comparision to TBP expression (2\textsuperscript{-deltaCT}). * p < 0.05, ** p < 0.01, *** p < 0.001, Welch's t-test. \textbf{E.} KDM5C protein expression normalized to DAXX. Quantified intensity using ImageJ (artificial units - au). Right - representative lanes of Western blot for KDM5C and DAXX. * p < 0.05, ** p < 0.01, *** p < 0.001, Welch's t-test \textbf{F.} XXX {G.} XXX.}
  \label{figurelabel}
\end{figure}


\clearpage
# Notes
## Things to do
- Move dazl to new figure if other staining works
- Use Ddx4 staining (with dazl?). If not should add RNA to diagram because its a key germline gene.
- Motif analysis
	- Discussion - talk about motifs

## Dazl
We were particularly interested in the aberrant transcription of _Dazl_, since it is essential for germ cell development and promotes the translation of germline mRNAs[@liDAZLMasterTranslational2019]. A significant portion of germline transcripts misexpressed in _Kdm5c_-KO EpiLCs are known binding targets of DAZL, including _Stra8_[@mikedisDAZLMediatesBroad2020] (p = 1.698e-07, Fishers Exact Test). This suggests expression of DAZL protein could promote the translation of other aberrant germline transcripts, influencing their ability to impact _Kdm5c_-KO cellular function. We thus tested DAZL protein expression in _Kdm5c_-KO EpiLCs through immunocytochemistry (Figure 3I). We observed about 25% of _Kdm5c_-KO EpiLCs expressed DAZL protein and it was localized to the cytoplasm (p = 0.0015, Welch's t-test), consistent with the pattern of DAZL expression in spermatogonia[@mikedisDAZLMediatesBroad2020]. Altogether these results suggest tissue-specific genes are misexpressed during _Kdm5c_-KO embryogenesis, including key drivers of germline identity that can be translated into protein.

- We additionally found _Kdm5c_-KO EpiLCs ectopically express DAZL protein that is localized to the cytoplasm, similar to its morphology in spermatogonia[@mikedisDAZLMediatesBroad2020]. **note: maybe just put in results**. Could move around depending upon if I get pheno working.

## Discussion notes

- For other paper:
	- for methods: Heatmaps of gene expression were generated using the base R functions scale and hclust and visualized using the R package ComplexHeatmap (v2.12.1).
	- - Might be good to look at for retinoic acid paper (WT germ expression): https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0005654


- Talk about different motifs if we do see differences there and if it explains direct vs indirect dysregulation
	- Could also mention PRC1.6/KDM5C binds pcgf6 in ESCs
- Maybe talk about other regulators/if anything is known about them long term? Or save for 2nd paper.
	- another germline repressor: https://pubmed.ncbi.nlm.nih.gov/35345626/





- end with something like: this indicates kdm5c not only modulates pluripotency and self-renewal in ESCs, but also has a role in long-term silencing of germline genes 
	- then transition into the long term silencing mechanism paragraph

- KDM5C is dynamically regulated during the window of embryonic germline gene silencing, our EpiLC ChIP-seq is likely catching the tail end of KDM5C's main involvement. 




- Our results indicate KDM5C is a pivotal repressor of germline programs in somatic cells. 
- Studies on other chromatin regulators have focused on key marker genes to identify germline gene misexpression, such as _Dazl_. 
- Using a germline-depleted mouse model, we curated a list of mouse germline-enriched genes to globally assess germline gene dysregulation. 
- Many of the genes dysregulated in the mature _Kdm5c_-KO brain were important for late stages of spermatogenesis, such as those important for sperm flagellar structure. Contrastingly, _Kdm5c_-KO EpiLCs aberrantly expressed key drivers of germ cell identity, including _Dazl_ and _Stra8_.


- While a significant portion of KDM5C-bound promoters are germline genes, many germline genes expressed during _Kdm5c_-KO embryogenesis are not directly bound by kdm5c. 
- One notable example is Stra8, a transcription factor important for germ cell specific transcription and meiotic initiation


- The including the demarcation between soma and germline fates. 


	the _Kdm5c_-KO brain also 
	Previous work identified testis-enriched genes in _Kdm5c_ genes
	- 
	- However unlike the gonadal-biased DEGs, 



- The demarcation of the germline versus soma is a key feature of multicellularity and sexually dimorphic reproduction



- Anything known about tissue-biased gene expression in other H3K4me regulators?
- Our data suggests the germline developmental program is occuring ectopically as _Kdm5c_-KOs progresses through somatic tissue development
- tissue-biased gene expression:



- Altogether, these data indicate that while some germline genes are misexpressed due to direct loss of KDM5C binding during emryogenesis, secondary downstream mechanisms can also promote their aberrant transcription.  




- Papers to read/reference:
	- Reconstitution of the Mouse Germ Cell Specification Pathway in Culture by Pluripotent Stem Cells: https://www.cell.com/fulltext/S0092-8674(11)00771-9
	- two cell gene list used by Suzuki et al Max paper is based on 2 cell sequencing: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3395470/


## Figure outline:

**Figure 1: Misexpression of tissue-specific genes in the _Kdm5c_-KO brain**
	* MA-plot and bar graphs of tissue-enriched genes
	* Example testis-specific genes (NCBI and bigwigs)
	* An example ovary tissue-specific gene 
	* An example muscle/liver tissue-specific gene (NCBI and bigwigs)


**Figure 2: The male _Kdm5c_-KO brain expresses male and female germline-enriched genes**
	* Gene ontology of testis DEGs in the amygdala and hippocampus - ontologies are germline ontologies
	* Expression of testis DEGs in germline-depleted testis (this is adult testis data)
	* scRNAseq of testis - # of testis DEGs that are germline-specific markers
	* Although far fewer, 5CKO brain also expresses ovary-enriched genes(NCBI and bigwigs of Zar1) 
	* These ovary enriched genes are also germline specific (NCBI/Li tissues are in adult ovary, so it would be best to show they're oocyte-specific in adult ovary. But I don't think there's a published adult female W/Wv dataset. Could try looking at scRNAseq or just do TPM in embryonic W/Wv data since oocytes are developed at this point? Or both?)
	* Defining what is/isn't a germline gene, and which are male/female biased using embryonic W/Wv data

**Figure 3: Kdm5c-KO epiblast-like cells express key drivers regulators of germline identity**
	* A) ESC to EpiLC differentiation Left - Morphology is unchanged, 
	* B) 5CKO EpiLCs express EpiLC differentiation genes similar to WT lvls
	* C) Male EpiLCs express germline genes (example Cyct again)
	* Overlap between brain and EpiLC germline genes - show they're mostly unique
	* GO of Brain and EpiLC germline genes (meiotic enriched) 
	* Bigwigs or TPM of master regulators
		* Show that while some are also 2-cell genes (Dazl), 2-cell specific genes aren't dysregulated (Zscan4). Important point because published KDM5C dazl paper is saying KDM5C is a 2-cell regulator, but as far as I can tell only genes shared between germline and 2-cell are dysregulated.  
	* Staining of Dazl (+ Stra8 if I can get it to work)


**Figure 4: Loss of KDM5C's catalytic activity impairs DNAme placement and long-term silencing of germline genes**
	* Increase in H3K4me3 in Kdm5c-KO amygdala at germline genes
	* Increase in H3K4me2 in EpiLCs at germline genes 
	* Kdm5c binding in EpiLCs vs PNCs to show that germline repression is happening in early embryo
	* Previous studies only looked at ESCs, unknown if catalytic activity is required for long-term repression, especially since DNA methylation is placed later). KDM5C RNA and protein ESC --> EpiLC (increasing then decreasing)
	* RNA expression of germline genes with catalytic dead rescue (Ilakkiya)
	* DNA methylation in WT and 5CKO EpiLCs (Ilakkiya)

**Figure 5: Ectopic, germline-like phenotypes in Kdm5c-KO ESCs/EpiLCs**
	* Sycp3 staining
	* DDX4 staining and repression of retrotransposons
	* Cilia??


Gaps in knowledge addressed: 
* Are other tissue-enriched genes dysregulated, or only testis, germline genes?
* Curating a robust list of male and female germline genes
	* Should talk about 2-cell genes vs germline genes - way to systematically categorize? 
* Mechanism behind long-term germline gene misexpression
	* Recent evidence suggests loss of KDM5C in ESCs express some germline genes
		* Unclear if catalytic activity is required for long-term silencing
		* Unclear if their dysregulation lasts throughout life or  the same between brain or not
	* When in development does it begin? - Recent evidence suggests some germline genes expressed in 5CKO ESCs but unclear if their dysregulation lasts throughout differentiation and if the identity of germline genes are different compared to the brain
* Are there functional consequences to germline gene misexpression?

Introduction:
* Chromatin regulators are important for cellular identity
	* H3K4me1-3 linked to active gene promoters and enhancers
	* Surprisingly, mutations in Chromatin regulators lead to many NDDs (including many H3K4 regulators)
		* Recent studies have shown some chromatin regulators are important for regulating neuron-specific gene expression/chromatin stat_compare_means 
	* However, loss of some chromatin regulators can also lead to ectopic expression of tissue-enriched genes
		* Very few studies have looked at these genes and it's unclear if these genes contribute to NDD impairments. 
	* Necessary to first characterize the mechanism behind their derepression to identify molecular footholds into testing their contribution to neuronal impairments and potential for therapeutic intervention

* Loss of KDM5C can result in the misexpression of genes typically only found in the testis
	* Misexpression of tissue-enriched genes hasn't been systematically characterized - Unclear if these genes are exceptions or if other tissue-specific genes are dysregulated
	* Interestingly, these genes (Cyct, D1pas1) typically function in the germline
	* Germ cells (meiotic cells) are typically distinguished from somatic cells very early on in embyrogenesis and is a key feature of multicellularity
	* Chromatin regulators are very important for decommissioning germline genes and act successively the embryo implants into the uterine wall
		* Most studies have focused on ESCs, which have a similar transcriptome to germ cells / 2-cells
		* recently, KDM5C was shown to repress DAZL in ESCs, independent of its catalytic activity
		* However, DNA methylation is lost in the mature 5CKO brain, DNA methylation is placed later and it's Unclear if it's required for long-term repression (maybe too specific, just trying to go into the fact that the mechanism is partially understood but unclear)
	* Systematic characterization of ectopic germline genes hasn't been done
		* unknown if other germline-enriched genes are dysregulated, including oocyte-specific genes
		* Crucially, it's unknown if misexpression of the germline program leads to functional consequences in 5CKO cells.


**Germline gene repression background:**

Interestingly, some of the ectopic testis transcripts identified in the _Kdm5c_-KO brain are typically expressed in germ cells[@scandagliaLossKdm5cCauses2017]. Unlike somatic cells, germ cells (e.g. sperm and eggs) undergo meiosis and pass on their genetic material to the next generation. The germline and the soma are typically distinguished during early embryogenesis, when germline genes are silenced in epiblast stem cells soon after implantation and only reactivated in a subset to form the germline. Chromatin regulators play a key role in decommissioning germline genes as the embryo transitions from na&#239;ve to primed pluripotency by placing repressive histone H2A lysine 119 monoubiquitination (H2AK119ub1)[@endohPCGF6PRC1SuppressesPremature2017], histone 3 lysine 9 trimethylation (H3K9me3)[@endohPCGF6PRC1SuppressesPremature2017; @mochizukiRepressionGermlineGenes2021], and DNA CpG methylation[@velascoDnmt3bRecruitmentE2F62010; @hackettPromoterDNAMethylation2012; @mochizukiRepressionGermlineGenes2021] at germline gene promoters. KDM5C may also be involved in this early decommissioning of germline genes, as re-expression of KDM5C in neurons fails to surpress their dysregulation[@scandagliaLossKdm5cCauses2017]. In support of this, KDM5C was very recently shown to repress _Deleted in azoospermia like (Dazl)_, a key regulator of germline development, in mouse embryonic stem cells (ESCs)[@guptaGenomewideScreenReveals2023; @aladhamiSystematicIdentificationFactors2023]. In support of this, two independent screens in mouse embryonic stem cells (ESCs) recently identified KDM5C as a repressor of _Deleted in azoospermia like (Dazl)_, a key regulator of germline development. However, KDM5C's role in embryonic germline gene repression is currently unclear, given that Dazl is also expressed in ESCs and in the 2-cell stage and germline gene misexpression has yet to be globally characterized during _Kdm5c_-KO embryogenesis.  
