---
title:  Erosion of somatic tissue identity with loss of the X-linked intellectual disability factor KDM5C
output: pdf_document
geometry: margin = 0.5in
font-size: 11pt
header-includes:
   - \usepackage{caption}
   - \captionsetup[figure]{font=footnotesize, labelfont=bf}
   - \usepackage{sectsty} 
   - \usepackage{helvet}
   - \renewcommand{\familydefault}{\sfdefault}
   - \usepackage{lineno}
   - \linenumbers
   - \renewcommand\linenumberfont{\normalfont\small}
   - \usepackage{setspace}
   - \spacing{1.5}
   - \pagenumbering{arabic}
   - \usepackage{indentfirst}

bibliography: references.bib
csl: cell-reports.csl
indent: true
editor_options:
  chunk_output_type: console
---
```{r, knitrcommas, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#adds commas to large numbers
library(huxtable)

knitr::knit_hooks$set(inline = function(x) {
  prettyNum(x, big.mark=",")
})

```



Gaps in knowledge addressed: 
* Are other tissue-enriched genes dysregulated, or only testis, germline genes?
* Curating a robust list of male and female germline genes
	* Should talk about 2-cell genes vs germline genes - way to systematically categorize? 
* Mechanism behind long-term germline gene misexpression
	* Recent evidence suggests loss of KDM5C in ESCs express some germline genes
		* Unclear if catalytic activity is required for long-term silencing
		* Unclear if their dysregulation lasts throughout life or  the same between brain or not
	* When in development does it begin? - Recent evidence suggests some germline genes expressed in 5CKO ESCs but unclear if their dysregulation lasts throughout differentiation and if the identity of germline genes are different compared to the brain
* Are there functional consequences to germline gene misexpression?

Introduction:
* Chromatin regulators are important for cellular identity
	* H3K4me1-3 linked to active gene promoters and enhancers
	* Surprisingly, mutations in Chromatin regulators lead to many NDDs (including many H3K4 regulators)
		* Recent studies have shown some chromatin regulators are important for regulating neuron-specific gene expression/chromatin stat_compare_means 
	* However, loss of some chromatin regulators can also lead to ectopic expression of tissue-enriched genes
		* Very few studies have looked at these genes and it's unclear if these genes contribute to NDD impairments. 
	* Necessary to first characterize the mechanism behind their derepression to identify molecular footholds into testing their contribution to neuronal impairments and potential for therapeutic intervention

* Loss of KDM5C can result in the misexpression of genes typically only found in the testis
	* Misexpression of tissue-enriched genes hasn't been systematically characterized - Unclear if these genes are exceptions or if other tissue-specific genes are dysregulated
	* Interestingly, these genes (Cyct, D1pas1) typically function in the germline
	* Germ cells (meiotic cells) are typically distinguished from somatic cells very early on in embyrogenesis and is a key feature of multicellularity
	* Chromatin regulators are very important for decommissioning germline genes and act successively the embryo implants into the uterine wall
		* Most studies have focused on ESCs, which have a similar transcriptome to germ cells / 2-cells
		* recently, KDM5C was shown to repress DAZL in ESCs, independent of its catalytic activity
		* However, DNA methylation is lost in the mature 5CKO brain, DNA methylation is placed later and it's Unclear if it's required for long-term repression (maybe too specific, just trying to go into the fact that the mechanism is partially understood but unclear)
	* Systematic characterization of ectopic germline genes hasn't been done
		* unknown if other germline-enriched genes are dysregulated, including oocyte-specific genes
		* Crucially, it's unknown if misexpression of the germline program leads to functional consequences in 5CKO cells.

# Introduction

To form a complete organism, embryonic stem cells must differentiate into a myriad of discrete cellular identities. This is in part accomplished by chromatin regulators that can either promote or impede lineage-specific gene expression through histone and DNA modifications[@strahlLanguageCovalentHistone2000; @jenuweinTranslatingHistoneCode2001]. Although initially identified for their roles in cellular identity [@lewisGeneComplexControlling1978a; @kennisonDosagedependentModifiersPolycomb1988a], recent advancements in next generation sequencing technologies unexpectedly found many neurodevelopmental disorders (NDDs) are caused by or linked to mutations in chromatin regulators. This relationship is partially explained by their regulation of brain-specific genes or chromatin states, such as modulating genes involved in synaptic maturation[@zhouBrainspecificPhosphorylationMeCP22006] or the transition between neuronal and glial developmental programs[@hirabayashiPolycombLimitsNeurogenic2009]. However, loss of some chromatin regulators can also lead to the misexpression of tissue-specific genes outside of their intended environment[@lewisGeneComplexControlling1978a; @kennisonDosagedependentModifiersPolycomb1988a; @kassisPolycombTrithoraxGroup2017]. Currently, very few studies have explored the misexpression of non-neuronal, tissue-specific genes in chromatin-linked neurodevelopmental disorders[@schaeferControlCognitionAdaptive2009; @scandagliaLossKdm5cCauses2017] and it is unclear if this partial loss of brain identity contributes to neurodevelopmental impairments. To elucidate their contribution to neurodevelopmental impairments, it is essential to first characterize the types of genes misexpressed, the developmental time point the dysregulation begins, and the molecular mechanism underlying their de-repression. Characterizing these features will enable us to identify molecular footholds common between NDDs that can then be exploited for potential therapeutics.

In this study, we characterized the misexpression of tissue-enriched genes with loss of the chromatin regulator lysine demethylase 5C (KDM5C, also known as SMCX or JARID1C), a histone 3 lysine 4 demethylase. Pathogenic mutations in _KDM5C_ cause Intellectual Developmental Disorder, X-linked, Syndromic, Claes-Jensen Type (MRXSCJ, OMIM: 300534), whose features include short stature, intellectual disability, seizures, aberrant aggression, and autistic behaviors[@claesNovelSyndromicForm2000; @jensenMutationsJARID1CGene2005; @carmignacFurtherDelineationFemale2020]. Previous work has demonstrated constitutive _Kdm5c_ knockout (-KO) mice recapitulate key MRXSCJ patient phenotypes, including hyperaggression and learning impairments[@iwaseMouseModelXlinked2016]. Next generation RNA sequencing (RNA-seq) in the _Kdm5c_-KO hippocampus unexpectedly revealed ectopic expression of testis-enriched genes within the brain[@scandagliaLossKdm5cCauses2017]. However, it is currently unclear if misexpression in the _Kdm5c_-KO brain is unique to testis genes, as other tissue-enriched genes have not been systematically evaluated.  

Interestingly, some of the ectopic testis transcripts identified in the _Kdm5c_-KO brain are typically expressed in germ cells[@scandagliaLossKdm5cCauses2017]. Unlike somatic cells, germ cells (e.g. sperm and eggs) undergo meiosis and pass on their genetic material to the next generation. The germline and the soma are typically distinguished during early embryogenesis, when germline genes are silenced in epiblast stem cells soon after implantation and only reactivated in a subset to form the germline. Chromatin regulators play a key role in decommissioning germline genes as the embryo transitions from naive to primed pluripotency by placing repressive histone H2A lysine 119 monoubiquitination (H2AK119ub1)[@endohPCGF6PRC1SuppressesPremature2017], histone 3 lysine 9 trimethylation (H3K9me3)[@endohPCGF6PRC1SuppressesPremature2017; @mochizukiRepressionGermlineGenes2021], and DNA CpG methylation[@velascoDnmt3bRecruitmentE2F62010; @hackettPromoterDNAMethylation2012; @mochizukiRepressionGermlineGenes2021] at germline gene promoters. KDM5C may also be involved in this early decommissioning of germline genes, as re-expression of KDM5C in neurons fails to surpress their dysregulation[@scandagliaLossKdm5cCauses2017]. In support of this, KDM5C was very recently shown to repress _Deleted in azoospermia like (Dazl)_, a key regulator of germline development, in mouse embryonic stem cells (ESCs)[@guptaGenomewideScreenReveals2023; @aladhamiSystematicIdentificationFactors2023]. In support of this, two independent screens in mouse embryonic stem cells (ESCs) recently identified KDM5C as a repressor of _Deleted in azoospermia like (Dazl)_, a key regulator of germline development. However, KDM5C's role in embryonic germline gene repression is currently unclear, given that Dazl is also expressed in ESCs and in the 2-cell stage and germline gene misexpression has yet to be globally characterized during _Kdm5c_-KO embryogenesis.  

To elucidate KDM5C's role in tissue identity, here we characterized the aberrant transcription of tissue-enriched genes within the _Kdm5c_-KO brain and epiblast-like cells (EpiLCs), an _in vitro_ model of the post-implantation embryo. We identified general dysregulation of tissue-enriched genes in both the adult _Kdm5c_-KO brain and EpiLCs, including misexpression of liver, muscle, and ovary genes. The _Kdm5c_-KO amygdala and hippocampus had significant enrichment of testis-biased genes that are unique to germ cells. To better characterize germline gene misexpression, we then generated a dataset of germline-enriched genes by comparing gene expression in gonads with germ cell depletion. We found _Kdm5c_-KO EpiLCs primarily expressed unique germline genes compared to the mature _Kdm5c_-KO brain, including _Dazl_ and _Stra8_, key drivers of germline identity and meiosis. While KDM5C is directly bound to some germline gene promoters in EpiLCs, it is not directly bound to many germline-enriched mRNAs expressed with _Kdm5c_-KO cells, indicating germline genes can be aberrantly transcribed through indirect mechanisms. Finally, we found KDM5C loss impairs the placement of DNA methylation at germline gene promoters as ESCs differentiate into EpiLCs. Therefore, we propose KDM5C plays a crucial role in the development of tissue identity during early embryogenesis, including establishment of the soma-germline boundary. 


**note: need a better conclusion sentence - work on when we know what's happening with last figure/functional consequences**


# Results

## Tissue-enriched genes, including testis genes, are aberrantly expressed in the _Kdm5c_-KO brain

- **note: should I compare amygdala and hippocampus? scandaglia only looked at hippocampus**

```{r, DESeq2_params, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the parameters used repeatedly, like the log2fold change cut off
source("../code/utilities/parameters.R")
```


Previous RNA sequencing (RNA-seq) in the adult _Kdm5c_ knockout (-KO) hippocampus identified a few transcripts that are typically expressed in the testis[@scandagliaLossKdm5cCauses2017]. However, the expression of tissue-specific genes within the _Kdm5c_-KO brain has not systematically characterized. Currently, it is unclear if other tissue-specific transcripts are also expressed within the _Kdm5c_-KO brain and if misexpression of testis-enriched genes are a major consequence of KDM5C loss, given the high abundance of testis-enriched genes within the mouse transcriptome. Therefore, we first globally assessed the expression of previously characterized tissue-enriched genes[@liComprehensiveMouseTranscriptomic2017] in our published mRNA-seq datasets of the _Kdm5c_-KO amygdala and hippocampus[@vallianatosMutuallySuppressiveRoles2020] 

**note: the idea is there but need to work on wording. The extent to which brain tissue identity is altered is unknown and if testis genes are a major effect**


```{r, tissue_genes, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the number and stats for tissue-enriched gene expression
tissue_res <- read.csv("../data/processed/TissueSpecific_AMYHIP_numberofgenesDEGs.csv", header = TRUE)

#get the value you want to print 
#col is which data column, tiss is which tissue
gettissval <- function(col, tiss){
	val <- subset(tissue_res, tissue == tiss)
	val <- val[,col]
	return(val)
}

```


We found many genes significantly upregulated within the _Kdm5c_-KO brain (DESeq2[@loveModeratedEstimationFold2014a], log2 fold change > `r l2fcco`, q < 0.1) are typically enriched in non-brain tissues, with the majority of differentially expressed genes (DEGs) displaying biased expression towards the testis (Figure 1A-C). Even though the testis has the largest number of tissue-biased genes compared to any other tissue (`r gettissval("count", "Testis")` genes), testis-biased DEGs were significantly enriched for both brain regions (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Testis"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Testis"), 3)`, Fisher's Exact).


Even though the _Kdm5c_-KO mice are male, we also observed significant enrichment of ovary-biased DEGs within both brain regions (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Ovary"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Ovary"), 3)`, Fisher's Exact) (Figure 1D). Intriguingly, many ovary and testis-biased DEGs have functions specific to germ cells and have no known role in the brain. For example, the testis-biased DEG _Cytochrome C, testis (Cyct)_ is a testis-specific cytochrome c isoform found in spermatozoa mitochondria and is important for flagellar beating[@goldbergCytochromeImmunofluorescentLocalization1977; @narisawaTestisSpecificCytochromeNull2002] (Figure 1C). The ovary-enriched DEG _Zygotic arrest 1 (Zar1)_ was recently shown to sequester mRNAs in oocytes for meiotic maturation and early zygote development[@chengMammalianOocytesStore2022] (Figure 1D). Although not consistent across brain regions, we also found enrichment of two non-gonadal tissues - the liver (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Liver"), 3)`, Fisher's Exact) and the muscle (Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Muscle"), 3)`, Fisher's Exact). One example of a liver-biased DEG is _Apolipoprotein C-I (Apoc1)_, which is involved in lipoprotein metabolism (Figure 1E). Overall, these tissue-enriched genes show little to no expression in the wild-type brain, yet our mRNA-seq data indicate they are polyadenylated and spliced into mature transcripts (Figure 1C-E). 

Of note, we did not observe enrichment of brain-enriched genes (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Brain"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Brain"), 3)`, Fisher's Exact), even though the brain has the second highest total number of tissue-enriched genes (`r gettissval("count", "Brain")` genes). Altogether, these results suggest misexpression of testis and other tissue-enriched genes within the _Kdm5c_-KO brain is not due to random de-repression of all genes, but rather due to a dysregulation of tissue identity.


### Germline genes are aberrantly expressed in the male _Kdm5c_-KO brain
```{r, germgenes, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
germgenes <- read.csv("../data/processed/germGENES20.csv")

#testis gene ontology results
testisGO <- read.csv("../results/GO_Brain_TestisDEGs.csv")

#function to get the GO ID
getGO <- function(x, df, column, cluster){
	df_test <- df
	
	if (cluster == "no") {
		out <- df_test[df_test$Description == x,][column][,1]
	} else {
		out <- df_test[df_test$Description == x & df_test$Cluster == cluster,][column][,1]
	}
	return(out)
}


```

The testis contains both germ cells (e.g. spermatogonia) and somatic cells (e.g. Leydig cells) that support hormone production and germline functions. We then wondered if the high enrichment of testis genes in the _Kdm5c_-KO brain reflected a failure to demarcate between the soma and germline. Gene ontology analysis of testis-biased DEGs revealed enrichment of germline-relevant ontologies, including spermatid development (`r getGO("spermatid development", testisGO, 'ID', "no")`, p.adjust = `r signif(getGO("spermatid development", testisGO, 'p.adjust', "no"), 3)`) and sperm axoneme assembly (`r getGO("sperm axoneme assembly", testisGO, 'ID', "no")`, p.adjust = `r signif(getGO("sperm axoneme assembly", testisGO, 'p.adjust', "no"), 3)`) (Figure 2A). 

To further validate if these testis DEGs are truly germline genes, we then evaluated their expression in germ cell-depleted testes via  _c-Kit_ mutation (Kit^W/Wv^)[@muellerIndependentSpecializationHuman2013]. Heterozygous _W_ and _Wv_ mutations in the enzymatic domain of _c-Kit_ prevents the maturation of germ cells, resulting in overall germ cell depletion[@handelSertoliCellDifferentiation1979]. We found almost all _Kdm5c_-KO testis-enriched DEGs lose their expression with germ cell depletion (Figure 2B). The only testis DEG that did not show considerable downregulation with germline depletion was _FK506 binding protein 6 (Fkbp6)_, a known regulator of piRNA expression and meiosis in germ cells[@crackowerEssentialRoleFkbp62003; @xiolRoleFkbp6Chaperone2012]. We additionally assessed testis DEG expression in a published testis single cell RNA-seq dataset that identified cell type-specific markers[@greenComprehensiveRoadmapMurine2018]. We found that while some testis-enriched DEGs were classified as specific markers for different germ cell types (e.g. spermatogonia, spermatocytes, round spermatids, and elongating spermatids) none marked somatic cells (Figure 2C). Together, these data suggest the _Kdm5c_-KO brain expresses germline genes.

We then aimed to globally characterize germline gene misexpression in _Kdm5c_-KO mice, but lacked a comprehensive list of mouse germline-enriched genes. To facilitate downstream analyses, we generated a list of male and female germline-enriched genes by evaluating gene expression in wild-type and germline-depleted (Kit^W/Wv^) mice. We assessed gene expression in male and female mice at embryonic day 12, 14, and 16, as well as male postnatal day 6 and adult testes. We defined genes as germline-enriched if their expression met the following criteria: 1) their expression is greater than 1 FPKM in wild-type germline 2) their expression in any wild-type, non-gonadal tissue[@liComprehensiveMouseTranscriptomic2017] does not exceed 20% of maximum expression in wild-type germline, and 3) their expression in the germ cell-depleted germline, for any sex or time point, does not exceed 20% of their maximum expression in the wild-type germline. These criteria yielded `r nrow(germgenes)` germline-enriched genes (Figure 2D), which was hereafter used as a resource for assessing misexpression of germline genes (Supplementary table 1).


## _Kdm5c_-KO epiblast-like cells aberrantly express master regulators of germline identity 

```{r, epilcs, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}

#testis gene ontology results
epigerm <- read.csv("../results/DESeq2/germDEGs/germDEGs_EpiLC_XY5cKO.csv")
GO_EpiBrain <- read.csv("../results/GO_EpiLC_vs_Brain.csv")

epi_restab <- read.csv("../data/processed/restable_EpiLC_XY5cKO.csv", row.names = 1)
epi_restab$ENSEMBL <- rownames(epi_restab)

zscan4d <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000090714',]['padj'][,1]

```

Misexpression of germline genes in the _Kdm5c_-KO brain suggests mutants fail to demarcate germline and somatic cellular identity. Germ cells are typically distinguished from somatic cells soon after the embryo implants into the uterine wall[@magnusdottirHowMakePrimordial2014; @gunesdoganPrimordialGermCell2014] when a subset of epiblast stem cells become the primordial germ cells (PGCs) while the remainder differentiate into the ectoderm, mesoderm, and endoderm to form the body's somatic tissues[@bardotMouseGastrulationCoordination2020]. This developmental time point can be modeled _in vitro_ through differentiation of embryonic stem cells (ESCs) into post-implantation epiblast-like stem cells (EpiLCs) (Figure 3A, top). Previous studies have demonstrated that while some germline-enriched genes are also expressed in embryonic stem cells (ESCs), they are silenced as they differentiate into EpiLCs[@mochizukiRepressionGermlineGenes2021]. Therefore, we assessed if KDM5C was necessary for initial germline gene silencing by assessing their expression in male _Kdm5c_-KO EpiLCs using our previously generated RNA-seq dataset[@samantaActivationXistEvolutionarily2022]. 

We did not observe any apparent morphology changes between wild-type and _Kdm5c_-KO cells during ESC to EpiLC differentiation (Figure 3A, bottom) and found no significant difference in primed pluripotency genes that indicate proper EpiLC differentiation (Figure 3B). We then identified _Kdm5c_-KO EpiLC DEGs through DESeq2[@loveModeratedEstimationFold2014a] (log2 fold change > `r l2fcco`, q < 0.1) and again observed general dysregulation of tissue-enriched genes, with the largest number of genes belonging to the brain and testis, although they were not significantly enriched (Figure 3C).

Using our curated list, we found `r format(nrow(epigerm))` germline-enriched genes were misexpressed in _Kdm5c_-KO EpiLCs, including _Cyct_ (Figure 3D). To evaluate whether germline mRNAs are constitutively dysregulated or change over the course of development, we then compared EpiLC germline DEGs to those expressed in the _Kdm5c_-KO brain. The majority of EpiLC germline DEGs were unique to EpiLCs, with only _Cyct_ shared across sequencing datasets (Figure 3E). We then compared the function of EpiLC and brain germline DEGs through gene ontology and found particularly high enrichment of meiosis-related gene ontologies in EpiLCs (Figure 3F), such as meiotic cell cycle (`r getGO("meiotic cell cycle", GO_EpiBrain, 'ID', "EpiLC")`, p.adjust = `r signif(getGO("meiotic cell cycle", GO_EpiBrain, 'p.adjust', "EpiLC"), 3)`). Contrastingly, the _Kdm5c_-KO hippocampus instead expressed genes involved in late-stage sperm development, those involved in the sperm axoneme. This suggests the germline developmental program may be occuring ectopically as _Kdm5c_-KO cells progresses through somatic tissue development **(note: placement is awkward. Even put here or talk about in dicussion because ChIPseq could help support this? - KDM5C not bound to a lot of brain DEGs)**.

While a few meiosis-related genes are misexpressed in the _Kdm5c_-KO brain, DEGs unique to _Kdm5c_-KO EpiLCs included key drivers of germline identity, such as _Stimulated by retinoic acid 8 (Stra8)_ and _Deleted in azoospermia like (Dazl)_ (Figure 3G). These genes are typically expressed during embryonic germ cell development to commit PGCs to the germline fate, but are also expressed later in life to trigger meiotic gene expression programs[@koubovaRetinoicAcidRegulates2006; @linGermCellIntrinsicExtrinsic2008; @endoRetinoicAcidGerm2019]. Of note, some germline genes, including _Dazl_, are also expressed in the two-cell embryo. However, we did not see misexpression of two-cell embryo-specific genes, like _Zscan4d_ (q = `r signif(zscan4d, 3)`), indicating _Kdm5c_-KO in EpiLCs does not revert cells back to a 2-cell state (Figure 3G). 

_Dazl_ is an RNA-binding protein that regulates the translation of germline mRNAs and is essential for germ cell function[@liDAZLMasterTranslational2019]. Interestingly, a significant portion of germline transcripts expressed in _Kdm5c_-KO EpiLCs are known binding targets of DAZL, including _Stra8_[@mikedisDAZLMediatesBroad2020] (p = 1.698e-07, Fisherâ€™s Exact Test). This suggests expression of DAZL protein would enable the translation of other aberrant germline transcripts, influencing their ability to impact on _Kdm5c_-KO cellular function. We therefore stained _Kdm5c_-KO EpiLCs for DAZL protein expression through immunocytochemistry (Figure 3H). We found about 25% of _Kdm5c_-KO EpiLCs expressed DAZL protein in their cytoplasm (p = 0.0015, Welch's t-test), consistent with the subcellular localization observed when DAZL is stained in spermatogonia[@mikedisDAZLMediatesBroad2020]. Altogether these results suggest _Kdm5c_-KO EpiLCs fail to decommission germline genes during early embryogenesis, including key drivers of germline identity that can be translated into protein.

### KDM5C-mediated removal of H3K4me3 during early embryogenesis promotes long-term repression of germline genes 

**Figure 4: Loss of KDM5C's catalytic activity impairs DNAme placement and long-term silencing of germline genes**
	A) KDM5C binding in EpiLCs vs PNCs - number of peaks and percentage of bound genomic regions
	B) KDM5C binding in Epilcs vs pncs germline genes - GO 
	C) Left - bar graph of germline DEGs KDM5C is bound to in EpiLCs Right 
	D) Example bigwigs
		* Brain and EpiLC shared, EpiLC-specific, Brain-specific, 
	E) motif analysis of KDM5C-bound vs unbound genes


	--- separate into two figures
	D) Bigwigs of Increase in H3K4me3 in Kdm5c-KO amygdala at germline genes
	E) Increase in H3K4me2 in EpiLCs at germline genes (left - bigwigs, right average signal at germline gene TSS? Just KDM5C-bound genes?)
	* Previous studies only looked at ESCs, unknown if catalytic activity is required for long-term repression, especially since DNA methylation is placed later). 
	F) KDM5C expression ESC --> EpiLC (increasing then decreasing) Left - RNA right - protein 
	G) RNA expression of germline genes with catalytic dead rescue (Ilakkiya)
	H) DNA methylation in WT and 5CKO EpiLCs (Ilakkiya)

	Direct vs indirect binding motif analysis

Previous work suggests KDM5C represses germline genes during early development, since KDM5C binding is gradually lost at hippocampal germline DEG promoters over development[@scandagliaLossKdm5cCauses2017] and re-expression of KDM5C in neuronal cultures fails to suppress their transcription. However, we identified many germline genes that were only dysregulated in _Kdm5c_-KO EpiLCs and not in the mature brain, including key regulators of meiosis and germline identity. While KDM5C was recently shown to bind to _Dazl_ in ESCs[@guptaGenomewideScreenReveals2023; @aladhamiSystematicIdentificationFactors2023], it's unclear if KDM5C binds to other germline drivers that are expressed in _Kdm5c_-KO EpiLCs and if this binding is maintained as cells differentiate.

```{r, KDM5C_ChIPseq, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
KDM5C_ChIP_EpiLC <- read.csv("../data/processed/KDM5C_ChIPseq_boundpromoters_EpiLC.csv", header = T)

KDM5C_ChIP_PNC <- read.csv("../data/processed/KDM5C_ChIPseq_boundpromoters_PNC.csv", header = T)

```

To better elucidate the mechanism behind KDM5C-mediated germline gene silencing, we analyzed KDM5C chromatin immunoprecipitation followed by DNA sequencing (ChIP-seq) datasets we previously generated in EpiLCs[@samantaActivationXistEvolutionarily2022] and primary hippocampal and cortical neuron cultures (PNCs)[@iwaseMouseModelXlinked2016]. We identified a higher number of KDM5C peaks in EpiLCs overall (EpiLCs: `r nrow(KDM5C_ChIP_EpiLC)`, PNCs: `r nrow(KDM5C_ChIP_PNC)`, MACS2 q < 0.1 and fold enrichment > 1) and while KDM5C was primarily localized to gene promoters in both cell types (EpiLCs: , PNCs: , +/- kb TSS), PNCs showed increased localization to distal intergenic regions (EpiLCs: , PNCs: ) (Figure 4A). Gene ontology analysis of KDM5C-bound promoters in EpiLCs revealed high enrichment of germline-related ontologies, including meiotic nuclear division (GO:0140013) and synaptonemal complex assembly (GO:0007130) (Figure 4B). Contrastingly, germline ontologies are not enriched in PNCs and KDM5C-bound genes instead perform non-germline-specific functions, such as heterocycle synthesis (GO: 0018130) and pyrimidine metabolism (GO: 0072527).


We then assessed KDM5C binding at germline genes expressed _Kdm5c_-KO cells to assess if their dysreuglation is due to direct loss of KDM5C binding during embryogenesis.

	* KDM5C was bound to the promoter of many genes dysregulated in both EpiLCs and brain datasets, including _D1Pas1_ and _Cyct_.
	* KDM5C was not bound to many of the brain-specific germline DEGs, such as XXX.
	* While KDM5C binds to many germline DEGs in EpiLCs, including _Dazl_, many were also unbound. This included the germline transcription factor _Stra8_.
	* All genes lacked binding in PNCs.
	* Altogether, these data indicate that while some germline genes are misexpressed due to direct loss of KDM5C binding during emryogenesis, secondary downstream mechanisms can also promote their aberrant transcription.  

---- 

Although KDM5C is generally thought to supress transcription through erasure of histone 3 lysine 4 di- and trimethylation (H3K4me2/3), recent studies in ESCs have suggested KDM5C's repression _Dazl_ is independent of its catalytic activity[@guptaGenomewideScreenReveals2023]. It is currently unclear if KDM5C's catalytic activity aids in the long-term repression of germline genes, such as enabling the placement of DNA methylation. 

* In congruence with previous work in the _Kdm5c_-KO hippocampus[@scandagliaLossKdm5cCauses2017], we observed ectopic H3K4me3 peaks in the _Kdm5c_-KO amygdala at the transcription start site (TSS) of germline genes (Figure 4).
* We additionally observed a marked increase in H3K4me2 at germline gene promoters in _Kdm5c_-KO EpiLCs.
	* This increase was greatest at germline genes directly bound by KDM5C (?)





# Discussion
- The demarcation of the germ vs soma is a key feature of multicellularity
- Other H3K4me regulators, anything known about tissue-biased gene expression?
- This suggests the germline developmental program is occuring ectopically as _Kdm5c_-KOs progresses through somatic tissue development
- tissue-biased gene expression:
	- However unlike the gonadal-biased DEGs, many liver and muscle-biased DEGs have a known involvement in brain function. For example, the liver-biased DEG is _Apolipoprotein C-I (Apoc1)_, is important for lipoprotein metabolism but has also been shown to influence learning and memory (Figure 1E).
- Otx2 is properly expressed in EpiLCs and prevents pgc identity https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6485399/ 

* Papers to read/reference:
	* Reconstitution of the Mouse Germ Cell Specification Pathway in Culture by Pluripotent Stem Cells: https://www.cell.com/fulltext/S0092-8674(11)00771-9
	* two cell gene list used by Suzuki et al Max paper is based on 2 cell sequencing: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3395470/




# References
<div id="refs"></div>

\newpage
# Figures and Tables

\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/Brain_tissue_genes.pdf}
  \caption[Tissue-enriched genes are misexpressed in the \textit{Kdm5c}-KO brain]{\textbf{Tissue-enriched genes are misexpressed in the \textit{Kdm5c}-KO brain.} \textbf{A.} Expression of tissue-enriched genes in the male \textit{Kdm5c}-KO amygdala. Left - MA plot of mRNA-sequencing. Right - Number of tissue-enriched differentially expressed genes (DEGs). * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test \textbf{B.} Expression of tissue-enriched genes in the male \textit{Kdm5c}-KO hippocampus. \textbf{C.} Left - Average bigwigs of an example aberrantly expressed testis-enriched DEG, \textit{Cytochrome C, testis-specific (Cyct)} in the wild-type (WT) and \textit{Kdm5c}-KO (5CKO) amygdala (red) and hippocampus (teal). Right - Expression of \textit{Cyct} in wild-type tissues from NCBI Gene, with testis highlighted in blue and brain tissues highlighted in red. \textbf{D.} Left - Average bigwigs of an example ovary-enriched DEG, \textit{Zygotic arrest 1 (Zar1)}. Right - Expression of \textit{Zar1} in wild-type tissues from NCBI Gene, with ovary highlighted in teal and brain tissues highlighted in red. \textbf{E.} Left - Average bigwigs of an example liver-enriched DEG, \textit{Apolipoprotein C-I (Apoc1)}. Right - Expression of \textit{Apoc1} in wild-type tissues from NCBI Gene, with liver highlighted in orange and brain tissues highlighted in red.}
  \label{figurelabel}
\end{figure}

__

\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/What_are_germline_genes.pdf}
  \caption[Aberrant transcription of germline genes in the \textit{Kdm5c}-KO in the brain]{\textbf{Aberrant transcription of germline genes in the \textit{Kdm5c}-KO in the brain.} \textbf{A.} enrichPlot gene ontology (GO) of \textit{Kdm5c}-KO amygdala and hippocampus testis-enriched DEGs \textbf{B.} Expression of testis DEGs in wild-type (WT) testis versus germ cell-depleted (W/Wv) testis. Expression is in Fragments Per Kilobase of transcript per Million mapped reads (FPKM). \textbf{C.} Number of testis DEGs that were classified as cell-type specific markers in a single cell RNA-seq dataset of the testis (Green et al 2018). Germline cell types are highlighted in green, somatic cell types in black. \textbf{D.} Sankey diagram of mouse genes filtered for germline enrichment based on their expression in wild-type and germline-depleted mice and in adult mouse non-gonadal tissues (Li et al 2017).}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/EpiLCs_too.pdf}
  \caption[\textit{Kdm5c}-KO epiblast-like cells express key drivers of germline identity ]{\textbf{\textit{Kdm5c}-KO epiblast-like cells express key drivers regulators of germline identity A.} Top - Diagram of \textit{in vivo} differentiation of embryonic stem cells (ESCs) of the inner cell mass into epiblast stem cells. Middle - \textit{in vitro} differentiation of ESCs into epiblast-like cells (EpiLCs). Bottom - representative images of wild-type (WT) and \textit{Kdm5c}-KO ESC to EpiLC differentiation. Brightfield images taken at 20X. \textbf{B.} No significant difference in primed pluripotency maker expression in wild-type versus \textit{Kdm5c}-KO EpiLCs. Welch's t-test, expression in transcripts per million (TPM). \textbf{C.} Number of tissue-enriched differentially expressed genes (DEGs). * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test. \textbf{D.} Average bigwigs of an example germline gene, \textit{Cyct}, that is dysregulated \textit{Kdm5c}-KO EpiLCs. \textbf{E.} Upset plot displaying the overlap of germline DEGs expressed in \textit{Kdm5c}-KO EpiLCs, amygdala (AMY), and hippocampus (HIP) RNA-seq datasets. \textbf{F.} enrichPlot comparing enriched gene ontologies for \textit{Kdm5c}-KO EpiLC, amygdala, and hippocampus germline DEGs. \textbf{G.} Top - Example germline identity DEGs unique to EpiLCs, p-values for Welch's t-test. Bottom - Average bigwigs of \textit{Dazl} and \textit{Stra8} expression in wild-type and \textit{Kdm5c}-KO EpiLCs \textbf{H.} Immunocytochemistry of DAZL in male wild-type (WT) and \textit{Kdm5c}-KO (5CKO) EpiLCs. Percentage of DAZL-positive cells normalized to DAPI, p-value for Welch's t-test}
  \label{figurelabel}
\end{figure}


\newpage
## Figure outline:

**Figure 1: Misexpression of tissue-specific genes in the _Kdm5c_-KO brain**
	* MA-plot and bar graphs of tissue-enriched genes
	* Example testis-specific genes (NCBI and bigwigs)
	* An example ovary tissue-specific gene 
	* An example muscle/liver tissue-specific gene (NCBI and bigwigs)


**Figure 2: The male _Kdm5c_-KO brain expresses male and female germline-enriched genes**
	* Gene ontology of testis DEGs in the amygdala and hippocampus - ontologies are germline ontologies
	* Expression of testis DEGs in germline-depleted testis (this is adult testis data)
	* scRNAseq of testis - # of testis DEGs that are germline-specific markers
	* Although far fewer, 5CKO brain also expresses ovary-enriched genes(NCBI and bigwigs of Zar1) 
	* These ovary enriched genes are also germline specific (NCBI/Li tissues are in adult ovary, so it would be best to show they're oocyte-specific in adult ovary. But I don't think there's a published adult female W/Wv dataset. Could try looking at scRNAseq or just do TPM in embryonic W/Wv data since oocytes are developed at this point? Or both?)
	* Defining what is/isn't a germline gene, and which are male/female biased using embryonic W/Wv data

**Figure 3: Kdm5c-KO epiblast-like cells express key drivers regulators of germline identity**
	* A) ESC to EpiLC differentiation Left - Morphology is unchanged, 
	* B) 5CKO EpiLCs express EpiLC differentiation genes similar to WT lvls
	* C) Male EpiLCs express germline genes (example Cyct again)
	* Overlap between brain and EpiLC germline genes - show they're mostly unique
	* GO of Brain and EpiLC germline genes (meiotic enriched) 
	* Bigwigs or TPM of master regulators
		* Show that while some are also 2-cell genes (Dazl), 2-cell specific genes aren't dysregulated (Zscan4). Important point because published KDM5C dazl paper is saying KDM5C is a 2-cell regulator, but as far as I can tell only genes shared between germline and 2-cell are dysregulated.  
	* Staining of Dazl (+ Stra8 if I can get it to work)


**Figure 4: Loss of KDM5C's catalytic activity impairs DNAme placement and long-term silencing of germline genes**
	* Increase in H3K4me3 in Kdm5c-KO amygdala at germline genes
	* Increase in H3K4me2 in EpiLCs at germline genes 
	* Kdm5c binding in EpiLCs vs PNCs to show that germline repression is happening in early embryo
	* Previous studies only looked at ESCs, unknown if catalytic activity is required for long-term repression, especially since DNA methylation is placed later). KDM5C RNA and protein ESC --> EpiLC (increasing then decreasing)
	* RNA expression of germline genes with catalytic dead rescue (Ilakkiya)
	* DNA methylation in WT and 5CKO EpiLCs (Ilakkiya)

**Figure 5: Ectopic, germline-like phenotypes in Kdm5c-KO ESCs/EpiLCs**
	* Sycp3 staining
	* DDX4 staining and repression of retrotransposons
	* Cilia??
