---
title:  Erosion of somatic tissue identity with loss of the X-linked intellectual disability factor KDM5C
output: pdf_document
geometry: margin = 0.5in
font-size: 11pt
header-includes:
   - \usepackage{caption}
   - \captionsetup[figure]{font=footnotesize, labelfont=bf}
   - \usepackage{sectsty} 
   - \usepackage{helvet}
   - \renewcommand{\familydefault}{\sfdefault}
   - \usepackage{lineno}
   - \linenumbers
   - \renewcommand\linenumberfont{\normalfont\small}
   - \usepackage{setspace}
   - \spacing{1.5}
   - \pagenumbering{arabic}
   - \usepackage{indentfirst}

bibliography: references.bib
csl: cell-reports.csl
indent: true
editor_options:
  chunk_output_type: console
---
```{r, knitrcommas, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#adds commas to large numbers
library(huxtable)

knitr::knit_hooks$set(inline = function(x) {
  prettyNum(x, big.mark=",")
})

```
# Abstract

# Introduction

A single genome holds the instructions to generate the myriad of cell types found within the adult organism. This is, in part, accomplished by chromatin regulators that can either promote or impede lineage-specific gene expression through DNA and histone modifications[@strahlLanguageCovalentHistone2000; @jenuweinTranslatingHistoneCode2001]. Although many chromatin regulators were initially identified for their roles in shaping cellular and tissue identity[@lewisGeneComplexControlling1978a; @kennisonDosagedependentModifiersPolycomb1988a], recent advancements in next generation sequencing unexpectedly revealed many neurodevelopmental disorders (NDDs) are caused by mutations in chromatin regulators[@gabrieleChromatinBasisNeurodevelopmental2018]. Several studies have suggested this connection between chromatin regulators and neurodevelopment is due to their regulation of brain-specific genes, such as orchestrating transcriptional programs for synaptic maturation[@zhouBrainspecificPhosphorylationMeCP22006] and transitioning between neuronal and glial fates during neural precursor differentiation[@hirabayashiPolycombLimitsNeurogenic2009]. However, loss of some chromatin regulators can also result in the ectopic transcription of tissue-specific genes outside of their target environment[@lewisGeneComplexControlling1978a; @kennisonDosagedependentModifiersPolycomb1988a; @kassisPolycombTrithoraxGroup2017], such as the misexpression of liver-specific genes within adult neurons[@schaeferControlCognitionAdaptive2009]. Very few studies have investigated the misexpression of tissue-specific genes in chromatin-linked NDDs[@schaeferControlCognitionAdaptive2009; @scandagliaLossKdm5cCauses2017] and it is currently unknown if this partial loss of brain identity contributes to neurodevelopmental impairments. 

To elucidate the role of tissue identity in chromatin-linked neurodevelopmental disorders, it is essential to first characterize the types of genes dysregulated and the molecular mechanisms governing their de-repression. In this study, we characterized the aberrant expression of tissue-enriched genes with loss of lysine demethylase 5C (KDM5C). KDM5C, , also known as SMCX or JARID1C, is a chromatin regulator that can repress gene expression through erasure of histone 3 lysine 4 di- and trimethylation (H3K4me2/3)[@iwaseXLinkedMentalRetardation2007], marks enriched at active gene promoters. Pathogenic mutations in _KDM5C_ cause Intellectual Developmental Disorder, X-linked, Syndromic, Claes-Jensen Type (MRXSCJ, OMIM: 300534), whose predominant features include intellectual disability, seizures, aberrant aggression, and autistic behaviors[@claesNovelSyndromicForm2000; @jensenMutationsJARID1CGene2005; @carmignacFurtherDelineationFemale2020]. _Kdm5c_ knockout (-KO) mice recapitulate key MRXSCJ patient phenotypes, including hyperaggression, increased seizure propensity, and learning impairments[@iwaseMouseModelXlinked2016; @scandagliaLossKdm5cCauses2017]. Unexpectedly, RNA sequencing (RNA-seq) of the _Kdm5c_-KO hippocampus revealed ectopic expression of testis genes within the brain[@scandagliaLossKdm5cCauses2017]. It is currently unknown if this dysregulation of brain tissue identity further impairs _Kdm5c_-KO neurodevelopment and if ectopic gene expression within the _Kdm5c_-KO brain is unique to testis genes. 

Intriguingly, some of the ectopic testis transcripts identified within the _Kdm5c_-KO hippocampus have known functions unique to germ cells[@scandagliaLossKdm5cCauses2017], suggesting KDM5C may play a role in demarcating somatic versus germline identity. Distinguishing between the germline (cells that pass on their genetic material, e.g. sperm and eggs) and the soma (cells that perform all other bodily functions) is a key feature of multicellularity and occurs during early embryogenesis. In mammals, chromatin regulators play a key role in decommissioning germline genes in somatic cells as the embryo implants into the uterine wall and transitions from naive to primed pluripotency by placing repressive histone H2A lysine 119 monoubiquitination (H2AK119ub1)[@endohPCGF6PRC1SuppressesPremature2017], histone 3 lysine 9 trimethylation (H3K9me3)[@endohPCGF6PRC1SuppressesPremature2017; @mochizukiRepressionGermlineGenes2021], and DNA CpG methylation[@velascoDnmt3bRecruitmentE2F62010; @hackettPromoterDNAMethylation2012; @mochizukiRepressionGermlineGenes2021] at germline gene promoters. KDM5C may also be involved in this early decommissioning of germline genes, as re-expression of KDM5C in knockout neurons fails to suppress their dysregulation[@scandagliaLossKdm5cCauses2017]. In support of this, KDM5C was very recently shown to repress _Deleted in azoospermia like (Dazl)_, a key regulator of germline development, in mouse embryonic stem cells (ESCs)[@guptaGenomewideScreenReveals2023; @aladhamiSystematicIdentificationFactors2023]. However, KDM5C binding and _Kdm5c_-KO germline gene misexpression has yet to be globally characterized during early embryogenesis. Given that _Dazl_ and other germline-enriched genes can also be expressed in ESCs and at the 2-cell stage, it is unclear if KDM5C has a direct role in the long-term germline gene silencing that occurs in the post-implantation epiblast. Systematically characterizing KDM5C's role in germline gene repression during early embryogenesis will unveil key mechanisms underlying the demarcation between soma and germline identity and while also providing molecular footholds to test the impact of ectopic germline genes on neurodevelopment. 

To illuminate KDM5C's role in tissue identity, here we characterized the aberrant transcription of tissue-enriched genes within the _Kdm5c_-KO brain and epiblast-like stem cells (EpiLCs), an _in vitro_ model of the post-implantation embryo. We observed general dysregulation of tissue-enriched genes in both the _Kdm5c_-KO brain and EpiLCs, including misexpression of genes typically highly enriched within the testis, liver, muscle, and ovary. Both the _Kdm5c_-KO amygdala and hippocampus had significant enrichment of testis genes that are typically uniquely expressed in germ cells. While the _Kdm5c_-KO brain primarily expressed germline genes important for late spermatogenesis, _Kdm5c_-KO EpiLCs aberrantly expressed key drivers of germline identity and meiosis, including _Dazl_ and _Stra8_. KDM5C was highly enriched at germline gene promoters in EpiLCs but only bound to a subset of germline genes expressed in _Kdm5c_-KO cells, indicating germline-enriched mRNAs can be aberrantly transcribed through indirect mechanisms. Finally, we found KDM5C promotes the long-term silencing of germline genes in somatic cells by aiding the placement of DNA methylation in EpiLCs through H3K4me2/3 removal. Thus, we propose KDM5C plays a fundamental role in the development of tissue identity during early embryogenesis, including the establishment of the soma-germline boundary. 

# Results

## Tissue-enriched genes, including testis genes, are aberrantly expressed in the _Kdm5c_-KO brain

- **note: should I compare amygdala and hippocampus? scandaglia only looked at hippocampus**

```{r, DESeq2_params, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the parameters used repeatedly, like the log2fold change cut off
source("../code/utilities/parameters.R")
```

Previous RNA sequencing (RNA-seq) in the adult hippocampus revealed ectopic expression of some testis genes within the _Kdm5c_ knockout (-KO) brain[@scandagliaLossKdm5cCauses2017]. Since tissue-enriched genes have not been systematically characterized in the _Kdm5c_-KO brain, it is currently unclear if this erosion of brain tissue identity is a major consequence of _Kdm5c_ loss and if it is unique to testis-enriched genes. Therefore, we first globally assessed the expression of genes enriched in 17 mouse tissues[@liComprehensiveMouseTranscriptomic2017] in our published mRNA-seq datasets of the adult amygdala and hippocampus in mice with constitutive knockout of _Kdm5c_[@vallianatosMutuallySuppressiveRoles2020].


```{r, tissue_genes, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
#get the number and stats for tissue-enriched gene expression
tissue_res <- read.csv("../data/processed/TissueSpecific_AMYHIP_numberofgenesDEGs.csv", header = TRUE)

#get the value you want to print 
#col is which data column, tiss is which tissue
gettissval <- function(col, tiss){
	val <- subset(tissue_res, tissue == tiss)
	val <- val[,col]
	return(val)
}

```

We found a large proportion of genes that are significantly upregulated within the _Kdm5c_-KO brain (DESeq2[@loveModeratedEstimationFold2014a], log2 fold change > `r l2fcco`, q < 0.1) are typically enriched in non-brain tissues (Amygdala: 35%, Hippocampus: 24%) (Figure 1A-B). The majority of tissue-enriched differentially expressed genes (tissue-enriched DEGs) were testis genes (Figure 1A-C). Even though the testis has the largest total number of tissue-biased genes compared to any other tissue (`r gettissval("count", "Testis")` genes), testis-biased DEGs were significantly enriched for both brain regions (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Testis"), 3)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Testis"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Testis"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Testis"), 3)`, Fisher's Exact Test). 
One example of a testis-enriched DEG is _FK506 binding protein 6 (Fkbp6)_, a known regulator of piRNA expression and meiosis in germ cells[@crackowerEssentialRoleFkbp62003; @xiolRoleFkbp6Chaperone2012] (Figure 1C). 

In addition to the high enrichment of testis genes, we also identified aberrant expression of other tissue-enriched genes within the _Kdm5c_-KO brain. Although the _Kdm5c_-KO mice were male, we observed significant enrichment of ovary-biased DEGs in both the amygdala and hippocampus (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Ovary"), 3)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Ovary"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Ovary"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Ovary"), 3)`, Fisher's Exact) (Figure 1D). Ovary-enriched DEGs included _Zygotic arrest 1 (Zar1)_, which was recently shown to sequester mRNAs in oocytes for meiotic maturation and early zygote development[@chengMammalianOocytesStore2022] (Figure 1D). Although not consistent across brain regions, we also found significant enrichment of DEGS biased towards two non-gonadal tissues - the liver (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Liver"), 3)`, Odds Ratio = `r signif(gettissval("Amygdala_oddsratio", "Liver"), 3)`, Fisher's Exact Test) and the muscle (Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Muscle"), 3)`, Odds Ratio = `r signif(gettissval("Hippocampus_oddsratio", "Muscle"), 3)` Fisher's Exact Test). An example of a liver-biased DEG dysregulated in both the hippocampus and amygdala is _Apolipoprotein C-I (Apoc1)_, which is involved in lipoprotein metabolism (Figure 1E). Testis, ovary, and liver-enriched DEGs showed little to no expression in the developing and adult wild-type brain, yet our mRNA-seq data indicate they are polyadenylated and spliced into mature transcripts (Figure 1C-E). Of note, we did not observe enrichment of brain-enriched genes (Amygdala p = `r signif(gettissval("Amygdala_pvalues", "Brain"), 3)`; Hippocampus p = `r signif(gettissval("Hippocampus_pvalues", "Brain"), 3)`, Fisher's Exact), despite the fact these are brain samples and the brain has the second highest total number of tissue-enriched genes (`r gettissval("count", "Brain")` genes). Together, these results suggest misexpression of tissue-enriched genes within the brain is a major effect of KDM5C loss.


```{r, germgenes, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
germgenes <- read.csv("../data/processed/germGENES20.csv")

#testis gene ontology results
testisGO <- read.csv("../results/GO_Brain_TestisDEGs.csv")

#function to get the GO ID
getGO <- function(x, df, column, cluster){
	df_test <- df
	
	if (cluster == "no") {
		out <- df_test[df_test$Description == x,][column][,1]
	} else {
		out <- df_test[df_test$Description == x & df_test$Cluster == cluster,][column][,1]
	}
	outformat <- ifelse(column == "ID", toString(out), out) #an attempt to remove the commas from GO ID numbers, didn't work
	return(outformat) 
}

```
### Germline genes are misexpressed in the _Kdm5c_-KO brain

The testis contains both germ cells (meiotic cells, e.g. spermatogonia) and somatic cells (non-meiotic, e.g. Leydig cells) that support hormone production and germline functions. Intriugingly, many _Kdm5c_-KO testis and ovary enriched-DEGs have germline-specific functions, suggesting _Kdm5c_-KO cells fail to distinguish between the soma and germline. To test if this holds true for all _Kdm5c_-KO testis-biased DEGs, we first assed their known functions through gene onotology analysis. We found _Kdm5c_-KO testis-enriched DEGs high enrichment of germline-relevant ontologies, including spermatid development (GO: 0007286, p.adjust = `r signif(getGO("spermatid development", testisGO, 'p.adjust', "no"), 3)`) and sperm axoneme assembly (GO: 0007288, p.adjust = `r signif(getGO("sperm axoneme assembly", testisGO, 'p.adjust', "no"), 3)`) (Figure 2A). 

To further validate if these testis DEGs are truly germline genes, we then evaluated their expression in somatic versus germ cells within the testis. We first compared their expression in the testis with germ cell depletion[@muellerIndependentSpecializationHuman2013], which was accomplished by heterozygous _W_ and _Wv_ mutations in the enzymatic domain of _c-Kit_ (Kit^W/Wv^) that prevent the maturation of germ cells[@handelSertoliCellDifferentiation1979]. Almost all _Kdm5c_-KO testis-enriched DEGs lost expression with germ cell depletion (Figure 2B). The only testis-enriched DEG that did not show considerable downregulation with germline depletion was _FK506 binding protein 6 (Fkbp6)_, the aforementioned testis gene that regulates piRNA expression and meiosis in germ cells[@crackowerEssentialRoleFkbp62003; @xiolRoleFkbp6Chaperone2012]. We then assessed testis-enriched DEG expression in a published single cell RNA-seq dataset that identified cell type-specific markers within the testis[@greenComprehensiveRoadmapMurine2018]. We found that while some testis-enriched DEGs were classified as specific markers for different germ cell types (e.g. spermatogonia, spermatocytes, round spermatids, and elongating spermatids) none marked somatic cells (Figure 2C). Together, these data suggest the somatic _Kdm5c_-KO brain aberrantly expresses germline genes.

We then wanted to more deeply characterize germline gene misexpression with _Kdm5c_ loss, but lacked a comprehensive list of mouse germline-enriched genes. To facilitate downstream analyses, we generated a curated list of germline-enriched genes using currently available RNA-seq datasets in Kit^W/Wv^ mice. Wild-type and Kit^W/Wv^ datasets included males and females at embryonic day 12, 14, and 16[@sohGeneRegulatoryProgram2015], as well as adult male testes[@muellerIndependentSpecializationHuman2013]. We defined genes as germline-enriched if their expression met the following criteria: 1) their expression is greater than 1 FPKM in wild-type gonads 2) their expression in any adult wild-type, non-gonadal tissue[@liComprehensiveMouseTranscriptomic2017] does not exceed 20% of their maximum expression in the wild-type germline, and 3) their expression in the germ cell-depleted germline, for any sex or time point, does not exceed 20% of their maximum expression in the wild-type germline. These criteria yielded `r nrow(germgenes)` germline-enriched genes (Figure 2D), which was hereafter used as a resource for assessing germline gene misexpression with _Kdm5c_ loss (Supplementary table 1).

## _Kdm5c_-KO epiblast-like cells aberrantly express master regulators of germline identity 

```{r, epilcs, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}

#testis gene ontology results
epigerm <- read.csv("../results/DESeq2/germDEGs/germDEGs_EpiLC_XY5cKO.csv")
GO_EpiBrain <- read.csv("../results/GO_EpiLC_vs_Brain.csv")

epi_restab <- read.csv("../data/processed/restable_EpiLC_XY5cKO.csv", row.names = 1)
epi_restab$ENSEMBL <- rownames(epi_restab)

zscan4d <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000090714',]['padj'][,1]
duxf3 <- epi_restab[epi_restab$ENSEMBL == 'ENSMUSG00000075046',]['padj'][,1]


```

Misexpression of germline genes in the adult brain suggests _Kdm5c_-KO cells fail to demarcate between germline and somatic identity. Germ cells are typically distinguished from somatic cells soon after the embryo implants into the uterine wall[@magnusdottirHowMakePrimordial2014; @gunesdoganPrimordialGermCell2014] when a subset of epiblast stem cells become the primordial germ cells (PGCs) while the remainder differentiate into the ectoderm, mesoderm, and endoderm to form the somatic tissues[@bardotMouseGastrulationCoordination2020]. This developmental time point can be modeled _in vitro_ through differentiation of embryonic stem cells (ESCs) into post-implantation epiblast-like stem cells (EpiLCs) (Figure 3A, top). Previous studies have demonstrated that while some germline-enriched genes are also expressed in embryonic stem cells (ESCs) and in the 2-cell stage[@wellingDAZLRegulatesTet12015; @macfarlanEmbryonicStemCell2012; @suzukiLossMAXResults2016], they are silenced as they differentiate into EpiLCs[@mochizukiRepressionGermlineGenes2021]. Therefore, we assessed if KDM5C was necessary for embryonic silencing of germline genes by evaluating the impact of _Kdm5c_ loss in EpiLCs. 

We first identified _Kdm5c_-KO EpiLC DEGs in of our previously published RNA-seq dataset[@samantaActivationXistEvolutionarily2022] (DESeq2, log2 fold change > `r l2fcco`, q < 0.1) and assessed tissue-enriched gene expression. Similar to the _Kdm5c_-KO brain, we observed general dysregulation of tissue-enriched genes, with the largest number of genes belonging to the brain and testis, although they were not significantly enriched (Figure 3B). Using the curated list of germline genes generated above, we found _Kdm5c_-KO EpiLCs aberrantly expressed `r format(nrow(epigerm))` germline-enriched genes, including the previously characterized hippocampal DEG[@scandagliaLossKdm5cCauses2017] _Cytochrome C, testis-specific (Cyct)_ (Figure 3C). Although we observed aberrant expression of many tissue-enriched genes, we did not observe any significant difference in primed pluripotency genes (Figure 3D) or gross changes in _Kdm5c_-KO cell morpholgy during differentiation (Figure 3E), indicating KDM5C loss does not impair EpiLC formation.

We then compared EpiLC germline DEGs to those expressed in the _Kdm5c_-KO brain to determine if all germline DEGs, like _Cyct_, are constitutively dysregulated or if they can change over the course of development. We found this was primarily not the case, as the majority of germline DEGs expressed in EpiLCs and the brain were unique, with only _Cyct_ shared across all sequencing datasets (Figure 3F). We then compared the known functions of EpiLC and brain germline DEGs and found particularly high enrichment of meiosis-related gene ontologies in EpiLCs (Figure 3G), such as meiotic cell cycle process (GO: 1903046, p.adjust = `r signif(getGO("meiotic cell cycle process", GO_EpiBrain, 'p.adjust', "EpiLC"), 3)`) and meiotic nuclear division (GO: 0140013, p.adjust = `r signif(getGO("meiotic nuclear division", GO_EpiBrain, 'p.adjust', "EpiLC"), 3)`). While there was modest enrichment of meiotic gene ontologies in in both brain regions, the _Kdm5c_-KO hippocampus primarily expressed late-stage spermatogenesis genes, such those involved in the sperm axoneme structure. 

Interestingly, DEGs unique to _Kdm5c_-KO EpiLCs included key drivers of germline identity, such as _Stimulated by retinoic acid 8 (Stra8)_ and _Deleted in azoospermia like (Dazl)_ (Figure 3H). These genes are typically expressed during embryonic germ cell development to commit PGCs to the germline fate, but are also expressed later in life to trigger meiotic gene expression programs[@koubovaRetinoicAcidRegulates2006; @linGermCellIntrinsicExtrinsic2008; @endoRetinoicAcidGerm2019]. Of note, some germline genes, including _Dazl_, are also expressed in the two-cell embryo[@macfarlanEmbryonicStemCell2012; @guptaGenomewideScreenReveals2023]. However, we did not see misexpression of two-cell embryo-specific genes, like _Duxf3 (Dux)_ (q = `r signif(duxf3, 3)`) and _Zscan4d_ (q = `r signif(zscan4d, 3)`), indicating _Kdm5c_-KO in EpiLCs do not revert back to a 2-cell state (Figure 3H). 

We were particularly interested in the aberrant transcription of _Dazl_, since it is essential for germ cell development and promotes the translation of germline mRNAs[@liDAZLMasterTranslational2019]. A significant portion of germline transcripts misexpressed in _Kdm5c_-KO EpiLCs are known binding targets of DAZL, including _Stra8_[@mikedisDAZLMediatesBroad2020] (p = 1.698e-07, Fisher’s Exact Test). This suggests expression of DAZL protein could promote the translation of other aberrant germline transcripts, influencing their ability to impact _Kdm5c_-KO cellular function. We thus tested DAZL protein expression in _Kdm5c_-KO EpiLCs through immunocytochemistry (Figure 3I). We observed about 25% of _Kdm5c_-KO EpiLCs expressed DAZL protein and it was localized to the cytoplasm (p = 0.0015, Welch's t-test), consistent with the pattern of DAZL expression in spermatogonia[@mikedisDAZLMediatesBroad2020]. Altogether these results suggest tissue-specific genes are misexpressed during _Kdm5c_-KO embryogenesis, including key drivers of germline identity that can be translated into protein.

## KDM5C binds to a subset of germline gene promoters during early embryogenesis 

Previous work suggests KDM5C represses germline genes during early development, as re-expression of KDM5C in knockout neuronal cultures fails to suppress hippocampal germline DEGs and KDM5C is not bound to their promoters in neurons[@scandagliaLossKdm5cCauses2017]. There is some evidence KDM5C binds to select germline gene promoters in ESCs[@scandagliaLossKdm5cCauses2017], including _Dazl_[@guptaGenomewideScreenReveals2023; @aladhamiSystematicIdentificationFactors2023]. As KDM5C's binding at germline gene promoters has not been systematically characterized, it is currently unclear if KDM5C is enriched at germline gene promoters, what types of germline genes KDM5C regulates, and if its binding is maintained at any germline genes in neurons. 


```{r, KDM5C_ChIPseq, results='hide', echo = FALSE, warning=FALSE,message=FALSE,error=FALSE}
KDM5C_ChIP_EpiLC_promo <- read.csv("../data/processed/KDM5C_ChIPseq_boundpromoters_EpiLC.csv")

KDM5C_ChIP_PNC_promo <- read.csv("../data/processed/KDM5C_ChIPseq_boundpromoters_PNC.csv")

KDM5C_ChIP_EpiLC <- read.csv("../data/raw/ChIPseq_mm10_EpiLC_WTnoKO_consensus_peaks.bed", sep = "\t")

KDM5C_ChIP_PNC <- read.csv("../data/raw/ChIPseq_mm10_PNC_WTnoKO_consensus_peaks.bed", sep = "\t")

GO_KDM5C_ChIP <- read.csv("../results/ChIPseq_KDM5C_GO_EpiLC_vs_PNC_unique.csv")

```

To further characterize the mechanism behind _Kdm5c_-KO germline gene misexpression, we analyzed KDM5C chromatin immunoprecipitation followed by DNA sequencing (ChIP-seq) datasets in EpiLCs[@samantaActivationXistEvolutionarily2022] and primary neuron cultures (PNCs) from the cortex and hippocampus[@iwaseMouseModelXlinked2016]. EpiLCs had a higher total number of KDM5C peaks than PNCs (EpiLCs: `r nrow(KDM5C_ChIP_EpiLC)`, PNCs: `r nrow(KDM5C_ChIP_PNC)`, MACS2 q < 0.1 and fold enrichment > 1, removal of _Kdm5c_-KO false positive peaks) and KDM5C was primarily localized to gene promoters in both cell types (EpiLCs: `r nrow(KDM5C_ChIP_EpiLC_promo)`, PNCs: `r nrow(KDM5C_ChIP_PNC_promo)` +/- `r promorange`bp from the TSS), although PNCs showed increased localization to non-promoter regions (Figure 4A). 

The majority of promoters bound by KDM5C in PNCs were also bound in EpiLCs (513 shared promoters), however a large portion of gene promoters were only bound by KDM5C in EpiLCs (3677 EpiLC only promoters) (Figure 4B). We then performed gene ontology analysis to compare the known functions of genes with KDM5C bound to their promoter in EpiLCs and PNCs. While there were no ontologies significantly enriched for genes only bound by KDM5C in PNCs, gene ontologies for genes shared between PNCs and EpiLCs were enriched for functions involving nucleic acid turnover, such as deoxyribonucleotide metabolic process (GO:0009262, p.adjust = `r signif(getGO("deoxyribonucleotide metabolic process", GO_KDM5C_ChIP, 'p.adjust', "Shared"), 3)`) (Figure 4C). Germline-specific ontologies were only enriched in promoters unique to EpiLCs, such as meiotic nuclear division (GO: 0007127 p.adjust = `r signif(getGO("meiosis I", GO_KDM5C_ChIP, 'p.adjust', "EpiLC only"), 3)`) and meiotic cell cycle process (GO:1903046, p.adjust = `r signif(getGO("meiotic cell cycle process", GO_KDM5C_ChIP, 'p.adjust', "EpiLC only"), 3)`) (Figure 3C). We then evaluated KDM5C binding around the transcription start site (TSS) of all germline-enriched genes. While KDM5C was bound to a subset of germline gene promoters in EpiLCs, it was not bound to any in PNCs (Figure 4D). Together, this suggests KDM5C is significantly enriched at a subset of germline gene promoters in EpiLCs, including meiotic genes, but does not regulate germline genes in neurons.

We then compared KDM5C binding at RNA-seq DEG promoters to determine if the germline mRNAs expressed in the _Kdm5c_-KO brain and EpiLCs are direct targets of KDM5C (Figure 4E). About one third of EpiLC-specific and brain-specific (hippocampus, amygdala, or both) germline DEGs were bound by KDM5C in EpiLCs (EpiLC only: `r 18/50*100`%, Brain only: `r signif(7/21*100,3)`%). Some notable differences in KDM5C binding for EpiLC-specific DEGs included _Dazl_ and _Stra8_, two key drivers of germline identity discussed above. Although both mRNAs are expressed in _Kdm5c_-KO EpiLCs, KDM5C is only bound to _Dazl_'s promoter and not _Stra8_'s in EpiLCs (Figure 4F). In contrast to the unique DEGs, 3 out of the 4 genes dysregulated in both the brain and EpiLCs are bound by KDM5C around the TSS (Figure 4G). Again, we did not observe any KDM5C binding at germline gene promoters in PNCs, even for brain-specific DEGs (Figure 4H). Altogether, this suggests the majority of germline mRNAs expressed in _Kdm5c_-KO cells are dysregulated independent of direct KDM5C recruitment to their promoters during embryogenesis. 

**notes:**
- do Direct vs indirect DEGs motif analysis

## KDM5C erases H3K4me3 to promote long-term repression of germline genes via DNA methylation

Although KDM5C is generally thought to supress transcription through erasure of histone 3 lysine 4 di- and trimethylation (H3K4me2/3)[@iwaseXLinkedMentalRetardation2007], recent studies in ESCs have suggested KDM5C's repression _Dazl_ is independent of its catalytic activity[@guptaGenomewideScreenReveals2023]. Somatic repression of germline genes is typically established during the transition between naive and primed pluripotency, which modeled _in vitro_ as ESCs to EpiLC differentiation. In ESCs, chromatin regulators place repressive histone modifications at germline gene promoters, including histone 2A 119 monoubiquitination (H2AK119ub1) and histone 3 lysine 9 trimethylation (H3K9me3)[@endohPCGF6PRC1SuppressesPremature2017; @liuPolycombGroupProtein2020; @mochizukiRepressionGermlineGenes2021]. Germline genes are then silenced long-term in EpiLCs by _de novo_ placement of DNA CpG methylation (DNAme)[@mochizukiRepressionGermlineGenes2021]. Although KDM5C's catalytic activity is not required to suppress _Dazl_ in ESCs, KDM5C may promote long-term germline gene silencing via H3K4me3 removal since H3K4me3 can impede DNAme placement[@otaniStructuralBasisRecognition2009; @guoStructuralInsightAutoinhibition2015] and DNAme is lost at select germline gene promoters in the hippocampus[@scandagliaLossKdm5cCauses2017]. Because KDM5C's role in germline gene repression has only been characterized in ESCs and in the mature brain, it is currently unclear to what extent KDM5C is involved during transition between ESCs and EpiLCs and if its catalytic activity is required for long-term silencing.

To elucidate KDM5C's role in germline gene silencing, we first characterized KDM5C's substrates, histone 3 lysine 4 di- and trimethylation (H3K4me2/3) in our previously published ChIP-seq datasets in wild type and _Kdm5c_-KO amygdala[@vallianatosMutuallySuppressiveRoles2020] and EpiLCs[@samantaActivationXistEvolutionarily2022]. In congruence with previous work in the _Kdm5c_-KO hippocampus[@scandagliaLossKdm5cCauses2017], we observed aberrant accumulation of H3K4me3 around the transcription start site (TSS) of germline genes in the _Kdm5c_-KO amygdala (Figure 5A). We additionally found a marked increase in H3K4me2 around the TSS of germline genes in _Kdm5c_-KO EpiLCs (Figure 5B). 

We then evaluated KDM5C's embryonic role in germline gene silencing during ESC to EpiLC differentiation. We first characterized KDM5C's mRNA and protein expression at 0, 24, and 48 hours of differentiation (Figure 5C). While _Kdm5c_ mRNA steadily decreased from 0 to 48 hours of diferentiation (Figure 5D), KDM5C protein initially increased from 0 to 24 hours but then decreased to near knockout levels by 48 hours (Figure 5E). We then perfomed whole genome bislufite sequencing in wild-type and _Kdm5c_-KO ESCs and 96 hour extended EpiLCs (exEpiLCs) to determine KDM5C's role in the initial placement of DNA methylation at germline gene promoters. While wild-type cells accumulated high levels of DNA methylation at germline gene promoters over the course of ESCs to exEpiLC differentiation, DNA methylation was markedly reduced in _Kdm5c_-KO exEpiLCs (Figure 5F). 

- Catalytic activity
- Altogether, this suggests KDM5C is dynamically regulated during ESC to EpiLC differentiation and promotes germline gene silencing via removal of H3K4me2/3 during early embryogenesis.

- **note:** should I test if incerase in H3K4me2 and H3K4me3 was highest in _Kdm5c_-KO cells at genes bound by KDM5C at their promoter in EpiLCs? Don't think it's that impactful






# Discussion

- The demarcation of the germ vs soma is a key feature of multicellularity
- Anything known about tissue-biased gene expression in other H3K4me regulators?
- Our data suggests the germline developmental program is occuring ectopically as _Kdm5c_-KOs progresses through somatic tissue development
- tissue-biased gene expression:
	- However unlike the gonadal-biased DEGs, many liver and muscle-biased DEGs have a known involvement in brain function. For example, the liver-biased DEG is _Apolipoprotein C-I (Apoc1)_, is important for lipoprotein metabolism but has also been shown to influence learning and memory.
- Otx2 is expressed in EpiLCs and is known to repress PGC identity. 
	- It's properly expressed in _Kdm5c_-KO EpiLCs, further supporting they aren't just becoming PGCs https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6485399/ 

- Altogether, these data indicate that while some germline genes are misexpressed due to direct loss of KDM5C binding during emryogenesis, secondary downstream mechanisms can also promote their aberrant transcription.  
	- This shift from meiotic genes to later spermatogenesis genes in the hippocampus suggests the germline developmental program could occur ectopically as _Kdm5c_-KO cells progresses through somatic tissue development. **note: this is strengthened by the ChIP-seq data since KDM5C is not directly bound to many brain/flagellar DEGs.This point might be stronger in the ChIPseq figure** 

- KDM5C is dynamically regulated during the window of embryonic germline gene silencing, our EpiLC ChIP-seq is likely catching the tail end of KDM5C's main involvement. 



- Papers to read/reference:
	- Reconstitution of the Mouse Germ Cell Specification Pathway in Culture by Pluripotent Stem Cells: https://www.cell.com/fulltext/S0092-8674(11)00771-9
	- two cell gene list used by Suzuki et al Max paper is based on 2 cell sequencing: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3395470/




# References
<div id="refs"></div>

\newpage
# Figures and Tables

\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/Brain_tissue_genes.pdf}
  \caption[Tissue-enriched genes are misexpressed in the \textit{Kdm5c}-KO brain]{\textbf{Tissue-enriched genes are misexpressed in the \textit{Kdm5c}-KO brain.} \textbf{A-B.} Expression of tissue-enriched genes (Li et al 2017) in the male \textit{Kdm5c}-KO amygdala (A) and hippocampus (B). Left - MA plot of mRNA-sequencing. Right - Number of tissue-enriched differentially expressed genes (DEGs). * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test \textbf{C.} Left - Average bigwigs of an example aberrantly expressed testis-enriched DEG, \textit{FK506 binding protein 6 (Fkbp6)} in the wild-type (WT) and \textit{Kdm5c}-KO (5CKO) amygdala (red) and hippocampus (teal). Right - Expression of \textit{Cyct} in wild-type tissues from NCBI Gene, with testis highlighted in blue and brain tissues highlighted in red. \textbf{D.} Left - Average bigwigs of an example ovary-enriched DEG, \textit{Zygotic arrest 1 (Zar1)}. Right - Expression of \textit{Zar1} in wild-type tissues from NCBI Gene, with ovary highlighted in teal and brain tissues highlighted in red. \textbf{E.} Left - Average bigwigs of an example liver-enriched DEG, \textit{Apolipoprotein C-I (Apoc1)}. Right - Expression of \textit{Apoc1} in wild-type tissues from NCBI Gene, with liver highlighted in orange and brain tissues highlighted in red.}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/What_are_germline_genes.pdf}
  \caption[Aberrant transcription of germline genes in the \textit{Kdm5c}-KO in the brain]{\textbf{Aberrant transcription of germline genes in the \textit{Kdm5c}-KO in the brain.} \textbf{A.} enrichPlot gene ontology (GO) of \textit{Kdm5c}-KO amygdala and hippocampus testis-enriched DEGs \textbf{B.} Expression of testis DEGs in wild-type (WT) testis versus germ cell-depleted (W/Wv) testis (Mueller et al 2013). Expression is in Fragments Per Kilobase of transcript per Million mapped reads (FPKM). \textbf{C.} Number of testis DEGs that were classified as cell-type specific markers in a single cell RNA-seq dataset of the testis (Green et al 2018). Germline cell types are highlighted in green, somatic cell types in black. \textbf{D.} Sankey diagram of mouse genes filtered for germline enrichment based on their expression in wild-type and germline-depleted mice and in adult mouse non-gonadal tissues (Li et al 2017).}
  \label{figurelabel}
\end{figure}


\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/EpiLCs_too.pdf}
  \caption[\textit{Kdm5c}-KO epiblast-like cells express key drivers of germline identity ]{\textbf{\textit{Kdm5c}-KO epiblast-like cells express key drivers of germline identity A.} Top - Diagram of \textit{in vivo} differentiation of embryonic stem cells (ESCs) of the inner cell mass into epiblast stem cells. Bottom - \textit{in vitro} differentiation of ESCs into epiblast-like cells (EpiLCs). \textbf{B.} Number of tissue-enriched differentially expressed genes (DEGs) in \textit{Kdm5c}-KO EpiLCs. * p<0.05, ** p<0.01, *** p<0.001, Fisher's exact test. \textbf{C.}  Average bigwigs of an example germline gene, \textit{Cyct}, that is dysregulated \textit{Kdm5c}-KO EpiLCs. \textbf{D.} No significant difference in primed pluripotency maker expression in wild-type versus \textit{Kdm5c}-KO EpiLCs. Welch's t-test, expression in transcripts per million (TPM). \textbf{E.} Representative images of wild-type (WT) and \textit{Kdm5c}-KO cells during ESC to EpiLC differentiation. Brightfield images taken at 20X.\textbf{F.} Upset plot displaying the overlap of germline DEGs expressed in \textit{Kdm5c}-KO EpiLCs, amygdala (AMY), and hippocampus (HIP) RNA-seq datasets. \textbf{G.} enrichPlot comparing enriched biological process gene ontologies for \textit{Kdm5c}-KO EpiLC, amygdala, and hippocampus germline DEGs. \textbf{H.} Top left - Example germline identity DEGs unique to EpiLCs. Top right - Example 2-cell genes that are not dysregulated in \textit{Kdm5c}-KO EpiLCs. p-values for Welch's t-test. Bottom - Average bigwigs of \textit{Dazl} and \textit{Stra8} expression in wild-type and \textit{Kdm5c}-KO EpiLCs \textbf{I.} Immunocytochemistry of DAZL in male wild-type (WT) and \textit{Kdm5c}-KO (5CKO) EpiLCs. Percentage of DAZL-positive cells normalized to number of DAPI-positive cells, p-value for Welch's t-test.}
  \label{figurelabel}
\end{figure}

\begin{figure}
  \centering 
  \includegraphics{../submission/compiled_figs/KDM5C_ChIPseq.pdf}
  \caption[KDM5C binds to a subset of germline gene promoters during early embryogenesis]{\textbf{KDM5C binds to a subset of germline gene promoters during early embryogenesis A.} ChIPseeker localization of KDM5C peaks at different genomic regions in EpiLCs (top) and hippocampal and cortex primary neuron cultures (PNCs, bottom). \textbf{B.} Overlap of genes with KDM5C bound to their promoters in EpiLCs (purple) and PNCs (blue). \textbf{C.} Gene ontology (GO) comparision of genes with KDM5C bound to their promoter in EpiLCs and PNCs. Genes were classified as either bound in EpiLCs only (EpiLC only), unique to PNCs (PNC only, no significant ontologies) or bound in both PNCs and EpiLCs (shared). \textbf{D.} Average KDM5C binding around the transcription start site (TSS) of all germline-enriched genes in EpiLCs (left) and PNCs (right). \textbf{E.} KDM5C binding at the promoters of RNA-seq germline DEGs. Genes were classified as either only dysregulated in EpiLCs (EpiLC only), genes dyresgulated in the hippocampus or amygdala but not EpiLCs (brain only), or genes dysregulated in both EpiLCs and the brain (common). (Legend continued on next page.)}
  \label{figurelabel}
\end{figure}
\addtocounter{figure}{-1}
\begin{figure} [t!]
  \caption[]{\textbf{KDM5C binds to a subset of germline gene promoters during early embryogenesis.} (Legend continued.) \textbf{F.} Example KDM5C ChIP-seq bigwigs of DEGs unqiue to EpiLCs. Although both are expressed in \textit{Kdm5c}-KO EpiLCs, KDM5C is only bound to the \textit{Dazl} promoter and not the \textit{Stra8} promoter in EpiLCs. \textbf{G.} Example KDM5C ChIP-seq bigwigs of RNA-seq DEGs common between the brain and EpiLCs. KDM5C is bound to the \textit{D1Pas1} promoter but not the \textit{XXX} promoter in EpiLCs. \textbf{H.} Example KDM5C ChIP-seq bigwigs of RNA-seq DEGs unique to the brain. KDM5C is bound to the \textit{XXX} promoter but not the \textit{Meig1} promoter}
\end{figure}

\newpage
\begin{figure}
  \centering
  \includegraphics{../submission/compiled_figs/KDM5C_Mechanism.pdf}
  \caption[KDM5C’s catalytic activity promotes long-term silencing of germline genes via DNA methylation]{\textbf{KDM5C’s catalytic activity promotes long-term silencing of germline genes via DNA methylation.} \textbf{A.} Left - Bigwigs of representative histone 3 lysine 4 trimethylation (H3K4me3) ChIP-seq peaks at two germline genes in the wild-type (WT) and \textit{Kdm5c}-KO (5CKO) adult amygdala. Right - Average H3K4me3 at the transcripiton start site (TSS) of all germline-enriched genes in wild-type (dark red) and \textit{Kdm5c}-KO (light red) amygdala. \textbf{B.} Left - Bigwigs of representative histone 3 lysine 4 dimethylation (H3K4me2) ChIP-seq peaks at representative germline genes in wild-type and \textit{Kdm5c}-KO EpiLCs. Right - Average H3K4me2 at the TSS of all germline-enriched genes in wild-type (dark purple) and \textit{Kdm5c}-KO (light purple) EpiLCs. \textbf{C.} Diagram of embryonic stem cell (ESC) to epiblast-like cell (EpiLC) differentiation protocol and collection time points for RNA and protein. \textbf{D.} Real time quantitative PCR (RT-qPCR) of \textit{Kdm5c} RNA expression, calculated in comparision to TBP expression (2\textsuperscript{-deltaCT}). * p < 0.05, ** p < 0.01, *** p < 0.001, Welch's t-test. \textbf{E.} KDM5C protein expression normalized to DAXX. Quantified intensity using ImageJ (artificial units - au). Right - representative lanes of Western blot for KDM5C and DAXX. * p < 0.05, ** p < 0.01, *** p < 0.001, Welch's t-test \textbf{F.} XXX {G.} XXX.}
  \label{figurelabel}
\end{figure}


\clearpage
# Notes

## Figure outline:

**Figure 1: Misexpression of tissue-specific genes in the _Kdm5c_-KO brain**
	* MA-plot and bar graphs of tissue-enriched genes
	* Example testis-specific genes (NCBI and bigwigs)
	* An example ovary tissue-specific gene 
	* An example muscle/liver tissue-specific gene (NCBI and bigwigs)


**Figure 2: The male _Kdm5c_-KO brain expresses male and female germline-enriched genes**
	* Gene ontology of testis DEGs in the amygdala and hippocampus - ontologies are germline ontologies
	* Expression of testis DEGs in germline-depleted testis (this is adult testis data)
	* scRNAseq of testis - # of testis DEGs that are germline-specific markers
	* Although far fewer, 5CKO brain also expresses ovary-enriched genes(NCBI and bigwigs of Zar1) 
	* These ovary enriched genes are also germline specific (NCBI/Li tissues are in adult ovary, so it would be best to show they're oocyte-specific in adult ovary. But I don't think there's a published adult female W/Wv dataset. Could try looking at scRNAseq or just do TPM in embryonic W/Wv data since oocytes are developed at this point? Or both?)
	* Defining what is/isn't a germline gene, and which are male/female biased using embryonic W/Wv data

**Figure 3: Kdm5c-KO epiblast-like cells express key drivers regulators of germline identity**
	* A) ESC to EpiLC differentiation Left - Morphology is unchanged, 
	* B) 5CKO EpiLCs express EpiLC differentiation genes similar to WT lvls
	* C) Male EpiLCs express germline genes (example Cyct again)
	* Overlap between brain and EpiLC germline genes - show they're mostly unique
	* GO of Brain and EpiLC germline genes (meiotic enriched) 
	* Bigwigs or TPM of master regulators
		* Show that while some are also 2-cell genes (Dazl), 2-cell specific genes aren't dysregulated (Zscan4). Important point because published KDM5C dazl paper is saying KDM5C is a 2-cell regulator, but as far as I can tell only genes shared between germline and 2-cell are dysregulated.  
	* Staining of Dazl (+ Stra8 if I can get it to work)


**Figure 4: Loss of KDM5C's catalytic activity impairs DNAme placement and long-term silencing of germline genes**
	* Increase in H3K4me3 in Kdm5c-KO amygdala at germline genes
	* Increase in H3K4me2 in EpiLCs at germline genes 
	* Kdm5c binding in EpiLCs vs PNCs to show that germline repression is happening in early embryo
	* Previous studies only looked at ESCs, unknown if catalytic activity is required for long-term repression, especially since DNA methylation is placed later). KDM5C RNA and protein ESC --> EpiLC (increasing then decreasing)
	* RNA expression of germline genes with catalytic dead rescue (Ilakkiya)
	* DNA methylation in WT and 5CKO EpiLCs (Ilakkiya)

**Figure 5: Ectopic, germline-like phenotypes in Kdm5c-KO ESCs/EpiLCs**
	* Sycp3 staining
	* DDX4 staining and repression of retrotransposons
	* Cilia??


Gaps in knowledge addressed: 
* Are other tissue-enriched genes dysregulated, or only testis, germline genes?
* Curating a robust list of male and female germline genes
	* Should talk about 2-cell genes vs germline genes - way to systematically categorize? 
* Mechanism behind long-term germline gene misexpression
	* Recent evidence suggests loss of KDM5C in ESCs express some germline genes
		* Unclear if catalytic activity is required for long-term silencing
		* Unclear if their dysregulation lasts throughout life or  the same between brain or not
	* When in development does it begin? - Recent evidence suggests some germline genes expressed in 5CKO ESCs but unclear if their dysregulation lasts throughout differentiation and if the identity of germline genes are different compared to the brain
* Are there functional consequences to germline gene misexpression?

Introduction:
* Chromatin regulators are important for cellular identity
	* H3K4me1-3 linked to active gene promoters and enhancers
	* Surprisingly, mutations in Chromatin regulators lead to many NDDs (including many H3K4 regulators)
		* Recent studies have shown some chromatin regulators are important for regulating neuron-specific gene expression/chromatin stat_compare_means 
	* However, loss of some chromatin regulators can also lead to ectopic expression of tissue-enriched genes
		* Very few studies have looked at these genes and it's unclear if these genes contribute to NDD impairments. 
	* Necessary to first characterize the mechanism behind their derepression to identify molecular footholds into testing their contribution to neuronal impairments and potential for therapeutic intervention

* Loss of KDM5C can result in the misexpression of genes typically only found in the testis
	* Misexpression of tissue-enriched genes hasn't been systematically characterized - Unclear if these genes are exceptions or if other tissue-specific genes are dysregulated
	* Interestingly, these genes (Cyct, D1pas1) typically function in the germline
	* Germ cells (meiotic cells) are typically distinguished from somatic cells very early on in embyrogenesis and is a key feature of multicellularity
	* Chromatin regulators are very important for decommissioning germline genes and act successively the embryo implants into the uterine wall
		* Most studies have focused on ESCs, which have a similar transcriptome to germ cells / 2-cells
		* recently, KDM5C was shown to repress DAZL in ESCs, independent of its catalytic activity
		* However, DNA methylation is lost in the mature 5CKO brain, DNA methylation is placed later and it's Unclear if it's required for long-term repression (maybe too specific, just trying to go into the fact that the mechanism is partially understood but unclear)
	* Systematic characterization of ectopic germline genes hasn't been done
		* unknown if other germline-enriched genes are dysregulated, including oocyte-specific genes
		* Crucially, it's unknown if misexpression of the germline program leads to functional consequences in 5CKO cells.


**Germline gene repression background:**

Interestingly, some of the ectopic testis transcripts identified in the _Kdm5c_-KO brain are typically expressed in germ cells[@scandagliaLossKdm5cCauses2017]. Unlike somatic cells, germ cells (e.g. sperm and eggs) undergo meiosis and pass on their genetic material to the next generation. The germline and the soma are typically distinguished during early embryogenesis, when germline genes are silenced in epiblast stem cells soon after implantation and only reactivated in a subset to form the germline. Chromatin regulators play a key role in decommissioning germline genes as the embryo transitions from naive to primed pluripotency by placing repressive histone H2A lysine 119 monoubiquitination (H2AK119ub1)[@endohPCGF6PRC1SuppressesPremature2017], histone 3 lysine 9 trimethylation (H3K9me3)[@endohPCGF6PRC1SuppressesPremature2017; @mochizukiRepressionGermlineGenes2021], and DNA CpG methylation[@velascoDnmt3bRecruitmentE2F62010; @hackettPromoterDNAMethylation2012; @mochizukiRepressionGermlineGenes2021] at germline gene promoters. KDM5C may also be involved in this early decommissioning of germline genes, as re-expression of KDM5C in neurons fails to surpress their dysregulation[@scandagliaLossKdm5cCauses2017]. In support of this, KDM5C was very recently shown to repress _Deleted in azoospermia like (Dazl)_, a key regulator of germline development, in mouse embryonic stem cells (ESCs)[@guptaGenomewideScreenReveals2023; @aladhamiSystematicIdentificationFactors2023]. In support of this, two independent screens in mouse embryonic stem cells (ESCs) recently identified KDM5C as a repressor of _Deleted in azoospermia like (Dazl)_, a key regulator of germline development. However, KDM5C's role in embryonic germline gene repression is currently unclear, given that Dazl is also expressed in ESCs and in the 2-cell stage and germline gene misexpression has yet to be globally characterized during _Kdm5c_-KO embryogenesis.  
